<!-- URL: http://circadiomics.ics.uci.edu/datasets -->
<!DOCTYPE html>
<html>
  <head>

    <title>CircadiOmics</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

    <!-- Bootstrap -->
    <link href="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet">
<link rel="stylesheet"
      href="/static/main.css">
      <!-- colors: ["#457AB2","#6A7E97","#8F837D","#B48862","#D98D58","#FF922E"], -->


    <!--[if lt IE 9]>
          <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.min.js"></script>
          <script src="//cdnjs.cloudflare.com/ajax/libs/respond.js/1.4.2/respond.min.js"></script>
    <![endif]-->



    <!-- Bootstrap -->
    <link href="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet">
<link rel="stylesheet"
      href="/static/main.css">
      <!-- colors: ["#457AB2","#6A7E97","#8F837D","#B48862","#D98D58","#FF922E"], -->


  </head>
  <body>
    
<nav class="navbar navbar-default">
  <div class="container-fluid">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a style="font-size: 16px  !important;"  class="navbar-brand" href="/"><font color="#457ab2">C</font>ircadi<font color="orange">O</font>mics</a>
    </div>

    <!-- Collect the nav links, forms, and other content for toggling -->
    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
      <ul class="nav navbar-nav">
        <li><a href="/about">About</a></li>
        <li><a href="/datasets">Datasets</a></li>
        <li><a href="/biocycle">BIO_CYCLE</a></li>
        <li><a href="/biocycle">Metabolic Atlas</a></li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li><a href="/help">Help</a></li>
      </ul>
    </div><!-- /.navbar-collapse -->
  </div><!-- /.container-fluid -->
</nav>


    

<div style="padding-left: 30px; padding-right: 30px;" class="container-fluid">
  <center>
  <h1 style="font-size:70px"><font color="#457ab2">D</font>ata<font color="orange">s</font>ets</h1>
  <br><br>
  <div class="row">
  <div class="col-lg-12" >
  <table align="left" style="margin: 0px">
    <tr>
      <th><div id="piechart4" style="width: 30%; height:20%;"></div></th>
      <th><div id="piechart1" style="width: 30%; height:20%;"></div></th>
      <th><div id="piechart2" style="width: 30%; height:20%;"></div></th>
      <th><div id="piechart3" style="width: 30%; height:20%;"></div></th>
    </tr>
  </table>
</div>
</div>
<br><br>
<div class="row">
<div class="col-lg-12">
<div align="left">
<input style="font-size:18px; max-width=300px" id="datasetFilter" type="text" class="form-control" placeholder="Search by any keyword.." onkeyup="filterRows()">
</div>
<br><br>
<ul class="nav nav-tabs" role="tablist">
  <li class="nav-item active">
    <a id="nav-Transcriptome" style="font-size:20px" class="nav-link active" data-toggle="tab" href="#trans" role="tab"><font color="#457ab2"><b>Transcriptome</b></font></a>
  </li>
  <li class="nav-item">
    <a id="nav-Metabolome" style="font-size:20px" class="nav-link" data-toggle="tab" href="#metabs" role="tab"><font color="black"><b>Metabolome</b></font></a>
  </li>
  <li class="nav-item">
    <a id="nav-Acetylome" style="font-size:20px" class="nav-link" data-toggle="tab" href="#acyls" role="tab"><font color="black"><b>Acetylome</b></font></a>
  </li>
  <li class="nav-item">
    <a id="nav-Proteome" style="font-size:20px" class="nav-link" data-toggle="tab" href="#prots" role="tab"><font color="black"><b>Proteome</b></font></a>
  </li>
</ul>
<div class="tab-content" >
  <div class="tab-pane active" id="trans" role="tabpanel">
    <table id="transcriptome_table" class="table table-striped">
    <thead class="thead-dark">
      <tr>
        <th scope="col">Dataset Description</th>
        <th scope="col">Species</th>
        <!-- <th scope="col">Tissue</th> -->
        <th scope="col">Tissue Class</th>
        <th scope="col">Condition</th>
        <th scope="col">GEO Accession</th>
        <th scope="col">Experiment Type</th>
        <!-- <th scope="col">Animal</th> -->
        <th scope="col">Timepoints</th>
        <!-- <th scope="col">Replicates</th> -->
        <!-- <th scope="col">Details</th> -->
        <th scope="col">Link</th>
      </tr>
    </thead>
    <tbody>
     
      <tr class="dataset_row">
        <td><b>AGE DEPENDENT CIRCADIAN CHANGES ADRENAL AGED:</b> Defining age-dependent changes at the systems level, profiling the circadian transcriptome in the hypothalamus, lung, heart, kidney, skeletal muscle, and adrenal gland in 3 age groups</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Dependent Circadian Changes Adrenal</td> -->
        <td>Glands</td>
        <td>Aged</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201207">GSE201207</a></td>
        <td>Age</td>
        <!-- <td></td> -->
        <td>26, 38, 58, 22, 46, 18, 30, 42, 50, 34, 62, 54</td>
        <!-- <td>2</td> -->
        <!-- <td>Hypothalamus, lung, kidney, skeletal muscle, heart, and adrenal gland tissues from Young (6mo), Aged (18mo), and Old (27mo) male mice were collected every 4h for 48h to characterize the circadian transcriptome changes with age. Two samples per age were collected and pooled for RNAseq analysis.</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2022.111982">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGE DEPENDENT CIRCADIAN CHANGES ADRENAL OLD:</b> Defining age-dependent changes at the systems level, profiling the circadian transcriptome in the hypothalamus, lung, heart, kidney, skeletal muscle, and adrenal gland in 3 age groups</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Dependent Circadian Changes Adrenal</td> -->
        <td>Glands</td>
        <td>Old</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201207">GSE201207</a></td>
        <td>Age</td>
        <!-- <td></td> -->
        <td>26, 38, 58, 22, 46, 18, 30, 42, 50, 34, 62, 54</td>
        <!-- <td>2</td> -->
        <!-- <td>Hypothalamus, lung, kidney, skeletal muscle, heart, and adrenal gland tissues from Young (6mo), Aged (18mo), and Old (27mo) male mice were collected every 4h for 48h to characterize the circadian transcriptome changes with age. Two samples per age were collected and pooled for RNAseq analysis.</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2022.111982">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGE DEPENDENT CIRCADIAN CHANGES ADRENAL YOUNG:</b> Defining age-dependent changes at the systems level, profiling the circadian transcriptome in the hypothalamus, lung, heart, kidney, skeletal muscle, and adrenal gland in 3 age groups</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Dependent Circadian Changes Adrenal</td> -->
        <td>Glands</td>
        <td>Young</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201207">GSE201207</a></td>
        <td>Age</td>
        <!-- <td></td> -->
        <td>26, 38, 58, 22, 46, 18, 30, 42, 50, 34, 62, 54</td>
        <!-- <td>2</td> -->
        <!-- <td>Hypothalamus, lung, kidney, skeletal muscle, heart, and adrenal gland tissues from Young (6mo), Aged (18mo), and Old (27mo) male mice were collected every 4h for 48h to characterize the circadian transcriptome changes with age. Two samples per age were collected and pooled for RNAseq analysis.</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2022.111982">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGE DEPENDENT CIRCADIAN CHANGES HEART AGED:</b> Defining age-dependent changes at the systems level, profiling the circadian transcriptome in the hypothalamus, lung, heart, kidney, skeletal muscle, and adrenal gland in 3 age group</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Dependent Circadian Changes Heart</td> -->
        <td>Heart</td>
        <td>Aged</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201207">GSE201207</a></td>
        <td>Age</td>
        <!-- <td></td> -->
        <td>26, 38, 58, 22, 46, 18, 30, 42, 50, 34, 62, 54</td>
        <!-- <td>2</td> -->
        <!-- <td>Hypothalamus, lung, kidney, skeletal muscle, heart, and adrenal gland tissues from Young (6mo), Aged (18mo), and Old (27mo) male mice were collected every 4h for 48h to characterize the circadian transcriptome changes with age. Two samples per age were collected and pooled for RNAseq analysis.</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2022.111982">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGE DEPENDENT CIRCADIAN CHANGES HEART OLD:</b> Defining age-dependent changes at the systems level, profiling the circadian transcriptome in the hypothalamus, lung, heart, kidney, skeletal muscle, and adrenal gland in 3 age group</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Dependent Circadian Changes Heart</td> -->
        <td>Heart</td>
        <td>Old</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201207">GSE201207</a></td>
        <td>Age</td>
        <!-- <td></td> -->
        <td>26, 38, 58, 22, 46, 18, 30, 42, 50, 34, 62, 54</td>
        <!-- <td>2</td> -->
        <!-- <td>Hypothalamus, lung, kidney, skeletal muscle, heart, and adrenal gland tissues from Young (6mo), Aged (18mo), and Old (27mo) male mice were collected every 4h for 48h to characterize the circadian transcriptome changes with age. Two samples per age were collected and pooled for RNAseq analysis.</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2022.111982">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGE DEPENDENT CIRCADIAN CHANGES HEART YOUNG:</b> Defining age-dependent changes at the systems level, profiling the circadian transcriptome in the hypothalamus, lung, heart, kidney, skeletal muscle, and adrenal gland in 3 age group</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Dependent Circadian Changes Heart</td> -->
        <td>Heart</td>
        <td>Young</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201207">GSE201207</a></td>
        <td>Age</td>
        <!-- <td></td> -->
        <td>26, 38, 58, 22, 46, 18, 30, 42, 50, 34, 62, 54</td>
        <!-- <td>2</td> -->
        <!-- <td>Hypothalamus, lung, kidney, skeletal muscle, heart, and adrenal gland tissues from Young (6mo), Aged (18mo), and Old (27mo) male mice were collected every 4h for 48h to characterize the circadian transcriptome changes with age. Two samples per age were collected and pooled for RNAseq analysis.</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2022.111982">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGE DEPENDENT CIRCADIAN CHANGES HYPOTHALAMUS AGED:</b> Defining age-dependent changes at the systems level, profiling the circadian transcriptome in the hypothalamus, lung, heart, kidney, skeletal muscle, and adrenal gland in 3 age group</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Dependent Circadian Changes Hypothalamus</td> -->
        <td>Brain</td>
        <td>Aged</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201207">GSE201207</a></td>
        <td>Age</td>
        <!-- <td></td> -->
        <td>26, 38, 58, 22, 46, 18, 30, 42, 50, 34, 62, 54</td>
        <!-- <td>2</td> -->
        <!-- <td>Hypothalamus, lung, kidney, skeletal muscle, heart, and adrenal gland tissues from Young (6mo), Aged (18mo), and Old (27mo) male mice were collected every 4h for 48h to characterize the circadian transcriptome changes with age. Two samples per age were collected and pooled for RNAseq analysis.</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2022.111982">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGE DEPENDENT CIRCADIAN CHANGES HYPOTHALAMUS OLD:</b> Defining age-dependent changes at the systems level, profiling the circadian transcriptome in the hypothalamus, lung, heart, kidney, skeletal muscle, and adrenal gland in 3 age group</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Dependent Circadian Changes Hypothalamus</td> -->
        <td>Brain</td>
        <td>Old</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201207">GSE201207</a></td>
        <td>Age</td>
        <!-- <td></td> -->
        <td>26, 38, 58, 22, 46, 18, 30, 42, 50, 34, 62, 54</td>
        <!-- <td>2</td> -->
        <!-- <td>Hypothalamus, lung, kidney, skeletal muscle, heart, and adrenal gland tissues from Young (6mo), Aged (18mo), and Old (27mo) male mice were collected every 4h for 48h to characterize the circadian transcriptome changes with age. Two samples per age were collected and pooled for RNAseq analysis.</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2022.111982">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGE DEPENDENT CIRCADIAN CHANGES HYPOTHALAMUS YOUNG:</b> Defining age-dependent changes at the systems level, profiling the circadian transcriptome in the hypothalamus, lung, heart, kidney, skeletal muscle, and adrenal gland in 3 age group</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Dependent Circadian Changes Hypothalamus</td> -->
        <td>Brain</td>
        <td>Young</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201207">GSE201207</a></td>
        <td>Age</td>
        <!-- <td></td> -->
        <td>26, 38, 58, 22, 46, 18, 30, 42, 50, 34, 62, 54</td>
        <!-- <td>2</td> -->
        <!-- <td>Hypothalamus, lung, kidney, skeletal muscle, heart, and adrenal gland tissues from Young (6mo), Aged (18mo), and Old (27mo) male mice were collected every 4h for 48h to characterize the circadian transcriptome changes with age. Two samples per age were collected and pooled for RNAseq analysis.</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2022.111982">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGE DEPENDENT CIRCADIAN CHANGES KIDNEY AGED:</b> Defining age-dependent changes at the systems level, profiling the circadian transcriptome in the hypothalamus, lung, heart, kidney, skeletal muscle, and adrenal gland in 3 age group</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Dependent Circadian Changes Kidney</td> -->
        <td>Kidney</td>
        <td>Aged</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201207">GSE201207</a></td>
        <td>Age</td>
        <!-- <td></td> -->
        <td>26, 38, 58, 22, 46, 18, 30, 42, 50, 34, 62, 54</td>
        <!-- <td>2</td> -->
        <!-- <td>Hypothalamus, lung, kidney, skeletal muscle, heart, and adrenal gland tissues from Young (6mo), Aged (18mo), and Old (27mo) male mice were collected every 4h for 48h to characterize the circadian transcriptome changes with age. Two samples per age were collected and pooled for RNAseq analysis.</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2022.111982">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGE DEPENDENT CIRCADIAN CHANGES KIDNEY OLD:</b> Defining age-dependent changes at the systems level, profiling the circadian transcriptome in the hypothalamus, lung, heart, kidney, skeletal muscle, and adrenal gland in 3 age group</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Dependent Circadian Changes Kidney</td> -->
        <td>Kidney</td>
        <td>Old</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201207">GSE201207</a></td>
        <td>Age</td>
        <!-- <td></td> -->
        <td>26, 38, 58, 22, 46, 18, 30, 42, 50, 34, 62, 54</td>
        <!-- <td>2</td> -->
        <!-- <td>Hypothalamus, lung, kidney, skeletal muscle, heart, and adrenal gland tissues from Young (6mo), Aged (18mo), and Old (27mo) male mice were collected every 4h for 48h to characterize the circadian transcriptome changes with age. Two samples per age were collected and pooled for RNAseq analysis.</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2022.111982">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGE DEPENDENT CIRCADIAN CHANGES KIDNEY YOUNG:</b> Defining age-dependent changes at the systems level, profiling the circadian transcriptome in the hypothalamus, lung, heart, kidney, skeletal muscle, and adrenal gland in 3 age group</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Dependent Circadian Changes Kidney</td> -->
        <td>Kidney</td>
        <td>Young</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201207">GSE201207</a></td>
        <td>Age</td>
        <!-- <td></td> -->
        <td>26, 38, 58, 22, 46, 18, 30, 42, 50, 34, 62, 54</td>
        <!-- <td>2</td> -->
        <!-- <td>Hypothalamus, lung, kidney, skeletal muscle, heart, and adrenal gland tissues from Young (6mo), Aged (18mo), and Old (27mo) male mice were collected every 4h for 48h to characterize the circadian transcriptome changes with age. Two samples per age were collected and pooled for RNAseq analysis.</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2022.111982">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGE DEPENDENT CIRCADIAN CHANGES LUNG AGED:</b> Defining age-dependent changes at the systems level, profiling the circadian transcriptome in the hypothalamus, lung, heart, kidney, skeletal muscle, and adrenal gland in 3 age group</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Dependent Circadian Changes Lung</td> -->
        <td>Lung</td>
        <td>Aged</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201207">GSE201207</a></td>
        <td>Age</td>
        <!-- <td></td> -->
        <td>26, 38, 58, 22, 46, 18, 30, 42, 50, 34, 62, 54</td>
        <!-- <td>2</td> -->
        <!-- <td>Hypothalamus, lung, kidney, skeletal muscle, heart, and adrenal gland tissues from Young (6mo), Aged (18mo), and Old (27mo) male mice were collected every 4h for 48h to characterize the circadian transcriptome changes with age. Two samples per age were collected and pooled for RNAseq analysis.</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2022.111982">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGE DEPENDENT CIRCADIAN CHANGES LUNG OLD:</b> Defining age-dependent changes at the systems level, profiling the circadian transcriptome in the hypothalamus, lung, heart, kidney, skeletal muscle, and adrenal gland in 3 age group</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Dependent Circadian Changes Lung</td> -->
        <td>Lung</td>
        <td>Old</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201207">GSE201207</a></td>
        <td>Age</td>
        <!-- <td></td> -->
        <td>26, 38, 58, 22, 46, 18, 30, 42, 50, 34, 62, 54</td>
        <!-- <td>2</td> -->
        <!-- <td>Hypothalamus, lung, kidney, skeletal muscle, heart, and adrenal gland tissues from Young (6mo), Aged (18mo), and Old (27mo) male mice were collected every 4h for 48h to characterize the circadian transcriptome changes with age. Two samples per age were collected and pooled for RNAseq analysis.</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2022.111982">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGE DEPENDENT CIRCADIAN CHANGES LUNG YOUNG:</b> Defining age-dependent changes at the systems level, profiling the circadian transcriptome in the hypothalamus, lung, heart, kidney, skeletal muscle, and adrenal gland in 3 age group</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Dependent Circadian Changes Lung</td> -->
        <td>Lung</td>
        <td>Young</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201207">GSE201207</a></td>
        <td>Age</td>
        <!-- <td></td> -->
        <td>26, 38, 58, 22, 46, 18, 30, 42, 50, 34, 62, 54</td>
        <!-- <td>2</td> -->
        <!-- <td>Hypothalamus, lung, kidney, skeletal muscle, heart, and adrenal gland tissues from Young (6mo), Aged (18mo), and Old (27mo) male mice were collected every 4h for 48h to characterize the circadian transcriptome changes with age. Two samples per age were collected and pooled for RNAseq analysis.</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2022.111982">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGE DEPENDENT CIRCADIAN CHANGES MUSCLE AGED:</b> Defining age-dependent changes at the systems level, profiling the circadian transcriptome in the hypothalamus, lung, heart, kidney, skeletal muscle, and adrenal gland in 3 age group</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Dependent Circadian Changes Muscle</td> -->
        <td>Muscle</td>
        <td>Aged</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201207">GSE201207</a></td>
        <td>Age</td>
        <!-- <td></td> -->
        <td>26, 38, 58, 22, 46, 18, 30, 42, 50, 34, 62, 54</td>
        <!-- <td>2</td> -->
        <!-- <td>Hypothalamus, lung, kidney, skeletal muscle, heart, and adrenal gland tissues from Young (6mo), Aged (18mo), and Old (27mo) male mice were collected every 4h for 48h to characterize the circadian transcriptome changes with age. Two samples per age were collected and pooled for RNAseq analysis.</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2022.111982">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGE DEPENDENT CIRCADIAN CHANGES MUSCLE OLD:</b> Defining age-dependent changes at the systems level, profiling the circadian transcriptome in the hypothalamus, lung, heart, kidney, skeletal muscle, and adrenal gland in 3 age group</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Dependent Circadian Changes Muscle</td> -->
        <td>Muscle</td>
        <td>Old</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201207">GSE201207</a></td>
        <td>Age</td>
        <!-- <td></td> -->
        <td>26, 38, 58, 22, 46, 18, 30, 42, 50, 34, 62, 54</td>
        <!-- <td>2</td> -->
        <!-- <td>Hypothalamus, lung, kidney, skeletal muscle, heart, and adrenal gland tissues from Young (6mo), Aged (18mo), and Old (27mo) male mice were collected every 4h for 48h to characterize the circadian transcriptome changes with age. Two samples per age were collected and pooled for RNAseq analysis.</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2022.111982">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGE DEPENDENT CIRCADIAN CHANGES MUSCLE YOUNG:</b> Defining age-dependent changes at the systems level, profiling the circadian transcriptome in the hypothalamus, lung, heart, kidney, skeletal muscle, and adrenal gland in 3 age group</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Dependent Circadian Changes Muscle</td> -->
        <td>Muscle</td>
        <td>Young</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201207">GSE201207</a></td>
        <td>Age</td>
        <!-- <td></td> -->
        <td>26, 38, 58, 22, 46, 18, 30, 42, 50, 34, 62, 54</td>
        <!-- <td>2</td> -->
        <!-- <td>Hypothalamus, lung, kidney, skeletal muscle, heart, and adrenal gland tissues from Young (6mo), Aged (18mo), and Old (27mo) male mice were collected every 4h for 48h to characterize the circadian transcriptome changes with age. Two samples per age were collected and pooled for RNAseq analysis.</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2022.111982">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGING EPIDERMAL ADULT CONTROL-DIET:</b> Adult stem cells undergo circadian reprogramming during ageing, epidermal stem cells, adult animals</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Epidermal Adult</td> -->
        <td>Skin</td>
        <td>Control-Diet</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84580">GSE84580</a></td>
        <td>Diet, Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="http://www.cell.com/cell/fulltext/S0092-8674(17)30871-1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGING EPIDERMAL ADULT RESTRICTED-DIET:</b> Adult stem cells undergo circadian reprogramming during ageing, epidermal stem cells, adult animals</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Epidermal Adult</td> -->
        <td>Skin</td>
        <td>Restricted-Diet</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84580">GSE84580</a></td>
        <td>Diet, Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="http://www.cell.com/cell/fulltext/S0092-8674(17)30871-1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGING EPIDERMAL AGED AGED:</b> Adult stem cells undergo circadian reprogramming during ageing, epidermal stem cells, aged animals</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Epidermal Aged</td> -->
        <td>Skin</td>
        <td>Aged</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84580">GSE84580</a></td>
        <td>Diet, Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="http://www.cell.com/cell/fulltext/S0092-8674(17)30871-1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGING EPIDERMAL AGED YOUNG:</b> Adult stem cells undergo circadian reprogramming during ageing, epidermal stem cells, aged animals</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Epidermal Aged</td> -->
        <td>Skin</td>
        <td>Young</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84580">GSE84580</a></td>
        <td>Diet, Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="http://www.cell.com/cell/fulltext/S0092-8674(17)30871-1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGING EPIDERMAL NORMAL CONTROL-DIET:</b> Adult stem cells undergo circadian reprogramming during ageing, epidermal stem cells, control animals</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Epidermal Normal</td> -->
        <td>Skin</td>
        <td>Control-Diet</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84580">GSE84580</a></td>
        <td>Diet, Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="http://www.cell.com/cell/fulltext/S0092-8674(17)30871-1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGING EPIDERMAL NORMAL HIGH-FAT-DIET:</b> Adult stem cells undergo circadian reprogramming during ageing, epidermal stem cells, control animals</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Epidermal Normal</td> -->
        <td>Skin</td>
        <td>High-Fat-Diet</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84580">GSE84580</a></td>
        <td>Diet, Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="http://www.cell.com/cell/fulltext/S0092-8674(17)30871-1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGING EPIDERMAL NORMAL ND-CONTROL-DIET:</b> Adult stem cells undergo circadian reprogramming during ageing, epidermal stem cells, control animals</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Epidermal Normal</td> -->
        <td>Skin</td>
        <td>Nd-Control-Diet</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84580">GSE84580</a></td>
        <td>Diet, Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="http://www.cell.com/cell/fulltext/S0092-8674(17)30871-1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGING EPIDERMAL NORMAL RESTRICTED-DIET:</b> Adult stem cells undergo circadian reprogramming during ageing, epidermal stem cells, control animals</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Epidermal Normal</td> -->
        <td>Skin</td>
        <td>Restricted-Diet</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84580">GSE84580</a></td>
        <td>Diet, Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="http://www.cell.com/cell/fulltext/S0092-8674(17)30871-1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGING SATELLITE ADULT CONTROL-DIET:</b> Adult stem cells undergo circadian reprogramming during ageing, satellite stem cells, adult animals</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Satellite Adult</td> -->
        <td>Skin</td>
        <td>Control-Diet</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84580">GSE84580</a></td>
        <td>Diet, Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="http://www.cell.com/cell/fulltext/S0092-8674(17)30871-1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGING SATELLITE ADULT RESTRICTED-DIET:</b> Adult stem cells undergo circadian reprogramming during ageing, satellite stem cells, adult animals</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Satellite Adult</td> -->
        <td>Skin</td>
        <td>Restricted-Diet</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84580">GSE84580</a></td>
        <td>Diet, Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="http://www.cell.com/cell/fulltext/S0092-8674(17)30871-1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGING SATELLITE AGED CONTROL-DIET:</b> Aged stem cells undergo circadian reprogramming during ageing, satellite stem cells, aged animals</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Satellite Aged</td> -->
        <td>Skin</td>
        <td>Control-Diet</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84580">GSE84580</a></td>
        <td>Diet, Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="http://www.cell.com/cell/fulltext/S0092-8674(17)30871-1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGING SATELLITE AGED RESTRICTED-DIET:</b> Aged stem cells undergo circadian reprogramming during ageing, satellite stem cells, aged animals</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Satellite Aged</td> -->
        <td>Skin</td>
        <td>Restricted-Diet</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84580">GSE84580</a></td>
        <td>Diet, Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="http://www.cell.com/cell/fulltext/S0092-8674(17)30871-1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGING SATELLITE NORMAL CONTROL-DIET:</b> Adult stem cells undergo circadian reprogramming during ageing, satellite stem cells, control animals</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Satellite Normal</td> -->
        <td>Skin</td>
        <td>Control-Diet</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84580">GSE84580</a></td>
        <td>Diet, Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="http://www.cell.com/cell/fulltext/S0092-8674(17)30871-1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>AGING SATELLITE NORMAL HIGH-FAT-DIET:</b> Adult stem cells undergo circadian reprogramming during ageing, satellite stem cells, control animals</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Satellite Normal</td> -->
        <td>Skin</td>
        <td>High-Fat-Diet</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84580">GSE84580</a></td>
        <td>Diet, Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="http://www.cell.com/cell/fulltext/S0092-8674(17)30871-1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ARABIDOPSIS EXTENDED PHOTOPERIOD FLOWERING CONTROL:</b> An extended 22-h photoperiod shifts circadian phase in seedlings and predicts flowering synchrony across recombinant inbred lines, mapping QTLs for photoperiod 'memory'.</td>
        <td><i>Arabidopsis thaliana</i></td>
        <!-- <td>Extended Photoperiod Flowering</td> -->
        <td>Seedling</td>
        <td>Control</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE286355">GSE286355</a></td>
        <td>Light-Dark</td>
        <!-- <td></td> -->
        <td>56, 54, 42, 50, 61, 62, 63, 64, 49, 66, 68, 69, 52, 24, 25, 26, 27, 21, 22, 23, 46, 47, 44, 45, 28, 29, 40, 41, 1, 3, 2, 5, 4, 6, 9, 78, 39, 77, 76, 38, 73, 71, 59, 58, 11, 10, 13, 12, 15, 14, 17, 16, 19, 18, 31, 30, 51, 36, 35, 34, 55, 74, 48, 57, 65</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1101/2025.02.25.640030">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ARABIDOPSIS EXTENDED PHOTOPERIOD FLOWERING EXTENDEDDAY:</b> An extended 22-h photoperiod shifts circadian phase in seedlings and predicts flowering synchrony across recombinant inbred lines, mapping QTLs for photoperiod 'memory'.</td>
        <td><i>Arabidopsis thaliana</i></td>
        <!-- <td>Extended Photoperiod Flowering</td> -->
        <td>Seedling</td>
        <td>Extendedday</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE286355">GSE286355</a></td>
        <td>Light-Dark</td>
        <!-- <td></td> -->
        <td>56, 54, 42, 50, 61, 62, 63, 64, 49, 66, 68, 69, 52, 24, 25, 26, 27, 21, 22, 23, 46, 47, 44, 45, 28, 29, 40, 41, 1, 3, 2, 5, 4, 6, 9, 78, 39, 77, 76, 38, 73, 71, 59, 58, 11, 10, 13, 12, 15, 14, 17, 16, 19, 18, 31, 30, 51, 36, 35, 34, 55, 74, 48, 57, 65</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1101/2025.02.25.640030">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ARABIDOPSIS PLANT EDWARDS 2007 WT:</b> Study the circadian expression of genes to model the Arabidopsis circadian clock</td>
        <td><i>Arabidopsis thaliana</i></td>
        <!-- <td>Plant</td> -->
        <td>Plant</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE5612">GSE5612</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>26, 38, 58, 46, 54, 30, 42, 50, 34, 62, 74, 66, 70</td>
        <!-- <td>2</td> -->
        <!-- <td>Plants will be transferred to continuous wight light after entrainment to 12h:12h light dark cycles. Analysis done with 26k chips.</td> -->
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE5612">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON ADRENAL CORTEX MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Adrenal Cortex</td> -->
        <td>Glands</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON ADRENAL MEDULLA MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Adrenal Medulla</td> -->
        <td>Glands</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON AMYGDALA MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Amygdala</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON ANTRUM MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Antrum</td> -->
        <td>Digestive</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON AORTA ENDOTHELIUM MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Aorta Endothelium</td> -->
        <td>Heart</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON ARCUATE NUCLEUS MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Arcuate Nucleus</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON ASCENDING COLON MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Ascending Colon</td> -->
        <td>Digestive</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON AXILLARY LYMPHONODES MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Axillary Lymphonodes</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON BLADDER MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Bladder</td> -->
        <td>Bladder</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON BONE MARROW MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Bone Marrow</td> -->
        <td>Bone</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON CECUM MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Cecum</td> -->
        <td>Digestive</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON CEREBELLUM MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Cerebellum</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON CORNEA MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Cornea</td> -->
        <td>Eye</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON DESCENDING COLON MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Descending Colon</td> -->
        <td>Digestive</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON DORSOMEDIAL HYPOTHALAMUS MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Dorsomedial Hypothalamus</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON DUODENUM MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Duodenum</td> -->
        <td>Digestive</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON HABENULA MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Habenula</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON HEART MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Heart</td> -->
        <td>Heart</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON HIPPOCAMPUS MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Hippocampus</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON ILEUM MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Ileum</td> -->
        <td>Digestive</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON IRIS MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Iris</td> -->
        <td>Eye</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON KIDNEY CORTEX MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Kidney Cortex</td> -->
        <td>Kidney</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON KIDNEY MEDULLA MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Kidney Medulla</td> -->
        <td>Kidney</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON LATERAL GLOBUS PALLIDUS MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Lateral Globus Pallidus</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON LATERAL HYPOTHALAMUS MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Lateral Hypothalamus</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON LIVER MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON LUNG MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Lung</td> -->
        <td>Lung</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON MAMMILARY BODIES MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Mammilary Bodies</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON MEDIAL GLOBUS PALLIDUS MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Medial Globus Pallidus</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON MESENTERIC LYMPHONODES MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Mesenteric Lymphonodes</td> -->
        <td>Bone</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON MUSCLE ABDOMINAL MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Muscle Abdominal</td> -->
        <td>Muscle</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON MUSCLE GASTROCNEMIAN MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Muscle Gastrocnemian</td> -->
        <td>Muscle</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON OESOPHAGUS MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Oesophagus</td> -->
        <td>Digestive</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON OLFACTORY BULB MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Olfactory Bulb</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON OMENTAL FAT MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Omental Fat</td> -->
        <td>Adipose</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON OPTIC NERVE HEAD MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Optic Nerve Head</td> -->
        <td>Eye</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON PANCREAS MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Pancreas</td> -->
        <td>Glands</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON PARAVENTRICULAR NUCLEI MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Paraventricular Nuclei</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON PINEAL MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Pineal</td> -->
        <td>Glands</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON PITUITARY MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Pituitary</td> -->
        <td>Glands</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON PONS MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Pons</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON PREFRONTAL CORTEX MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Prefrontal Cortex</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON PREOPTIC AREA MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Preoptic Area</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON PROSTATE MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Prostate</td> -->
        <td>Reproductive</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON PUTAMEN MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Putamen</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON RETINA MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Retina</td> -->
        <td>Eye</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON RETINAL PIGMENT EPITHELIUM MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Retinal Pigment Epithelium</td> -->
        <td>Eye</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON SKIN MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Skin</td> -->
        <td>Skin</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON SMOOTH MUSCLE MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Smooth Muscle</td> -->
        <td>Muscle</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON SPLEEN MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Spleen</td> -->
        <td>Spleen</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON STOMACH FUNDUS MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Stomach Fundus</td> -->
        <td>Digestive</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON SUBSTANTIA NIGRA MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Substantia Nigra</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON SUPRACHIASMATIC NUCLEI MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Suprachiasmatic Nucleus</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON SUPRAOPTIC NUCLEUS MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Supraoptic Nucleus</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON TESTICLES MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Testicles</td> -->
        <td>Reproductive</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON THALAMUS MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Thalamus</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON THYROID MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Thyroid</td> -->
        <td>Glands</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON VENTRO MEDIAL HYPOTHALAMUS MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Ventro Medial Hypothalamus</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON VISUAL CORTEX MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>Visual Cortex</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON WHITE ADIPOSE MESENTERIC MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>White Adipose Mesenteric</td> -->
        <td>Adipose</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON WHITE ADIPOSE PERICARDIAL MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>White Adipose Pericardial</td> -->
        <td>Adipose</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON WHITE ADIPOSE PERIRENAL MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>White Adipose Perirenal</td> -->
        <td>Adipose</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON WHITE ADIPOSE RETROPERITONEAL MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>White Adipose Retroperitoneal</td> -->
        <td>Adipose</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>BABOON WHITE ADIPOSE SUBCUTANEOUS MURE 2018 WT:</b> Diurnal transcriptome of 64 tissues sampled every 2 hours over 24 hours.</td>
        <td><i>Papio anubis</i></td>
        <!-- <td>White Adipose Subcutaneous</td> -->
        <td>Adipose</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98965">GSE98965</a></td>
        <td>Control</td>
        <!-- <td>Young Male</td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Animals housed individually in semi-natural condition under 12 hours light:12 hours dark. Tissue collected every 2 hours across the 24 hour day.</td> -->
        <td><a href="http://science.sciencemag.org/content/early/2018/02/07/science.aao0318">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>C. ELEGANS 2024 NHR-23 KO:</b> C. elegans were cultivated under temperature cycles. Then, depleted NHR-23, and sampled worms under constant(CC) condition every 2 h.</td>
        <td><i>Caenorhabditis elegans</i></td>
        <!-- <td>Elegans</td> -->
        <td>Whole Body</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE233891">GSE233891</a></td>
        <td>Knock-Out</td>
        <!-- <td>JDW259</td> -->
        <td>11, 13, 15, 21, 17, 23, 19, 1, 3, 5, 7, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Animals were transfered to NGM plates with/without 0.4 mM K-NAA 12h prior to sampling.</td> -->
        <td><a href="https://www.nature.com/articles/s42003-024-05894-3">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>C.ELEGANS DEVELOPMENT ON FOOD WT:</b> Synchronized L1 stage larvae were placed on food at 25C, and samples collected hourly over a 16 hr period that covered development from L3 to the young adult stage.</td>
        <td><i>Caenorhabditis elegans</i></td>
        <!-- <td>Development On Food</td> -->
        <td>Whole Body</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE52910">GSE52910</a></td>
        <td>Control</td>
        <!-- <td>N2</td> -->
        <td>11, 10, 13, 12, 15, 14, 16, 1, 3, 2, 5, 4, 7, 6, 9, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>total RNA isolation was performed using Tri-reagent (MRC) Library preparation was performed using the Truseq stranded mRNA sample preparation kit (Illumina) according to the manufacturer's protocol</td> -->
        <td><a href="https://www.sciencedirect.com/science/article/pii/S1097276513009039?via%3Dihub">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>CANCER FOXP1 MUSCLE CACHEXIA KPC-FOXP1-KO:</b> FoxP1 up-regulation in pancreatic-cancer cachexia (KPC: PDAC tumour-bearing mice; Sham: non-tumour controls) disrupts skeletal-muscle circadian transcription, whereas muscle-specific FoxP1 knockout preserves rhythmic genes and mitigates wasting.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Foxp1 Muscle Cachexia</td> -->
        <td>Muscle</td>
        <td>Kpc-Foxp1-KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE273878">GSE273878</a></td>
        <td>Disease, Knock-Out</td>
        <!-- <td></td> -->
        <td>26, 38, 22, 18, 30, 34</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2025.115689">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>CANCER FOXP1 MUSCLE CACHEXIA KPC-WT:</b> FoxP1 up-regulation in pancreatic-cancer cachexia (KPC: PDAC tumour-bearing mice; Sham: non-tumour controls) disrupts skeletal-muscle circadian transcription, whereas muscle-specific FoxP1 knockout preserves rhythmic genes and mitigates wasting.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Foxp1 Muscle Cachexia</td> -->
        <td>Muscle</td>
        <td>Kpc-Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE273878">GSE273878</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>26, 38, 22, 18, 30, 34</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2025.115689">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>CDKL5 IMPACT IN HIPPOCAMPUS KO:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Impact In Hippocampus</td> -->
        <td>Brain</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>CDKL5 IMPACT IN HIPPOCAMPUS WT:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Impact In Hippocampus</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>CDKL5 IMPACT IN SCN KO:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Impact In Scn</td> -->
        <td>Brain</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>CDKL5 IMPACT IN SCN WT:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Impact In Scn</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>CHICKEN ERYTHROIDPROGENITORS DAMIOLA 2004 WT:</b> Global gene expression profile of immature avian erythrocytic progenitor cells, comparing self-renewal vs differentiation states (SAGE analysis) to identify factors involved in the switch from progenitor self-renewal to differentiation.</td>
        <td><i>Gallus gallus</i></td>
        <!-- <td>Erythroidprogenitors</td> -->
        <td>Cells</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE2026">GSE2026</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>56, 54, 42, 50, 60, 62, 64, 66, 68, 52, 24, 26, 20, 22, 46, 44, 48, 28, 40, 38, 72, 70, 58, 12, 14, 16, 18, 30, 36, 34, 32</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1038/sj.onc.1208061">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>DROSOPHILA DPPIL4 CIRCADIAN CONTROL:</b> RNAi knock-down of peptidyl-prolyl isomerase-like 4 lengthens free-running period and dampens PERIOD oscillations in fly clock neurons.</td>
        <td><i>Drosophila melanogaster</i></td>
        <!-- <td>Dppil4 Circadian</td> -->
        <td>Brain</td>
        <td>Control</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE292592">GSE292592</a></td>
        <td>Knock-Down</td>
        <!-- <td></td> -->
        <td>8, 16</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1016/j.isci.2025.109245">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>DROSOPHILA DPPIL4 CIRCADIAN KO:</b> RNAi knock-down of peptidyl-prolyl isomerase-like 4 lengthens free-running period and dampens PERIOD oscillations in fly clock neurons.</td>
        <td><i>Drosophila melanogaster</i></td>
        <!-- <td>Dppil4 Circadian</td> -->
        <td>Brain</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE292592">GSE292592</a></td>
        <td>Knock-Down</td>
        <!-- <td></td> -->
        <td>8, 16</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1016/j.isci.2025.109245">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>DROSOPHILA KADENER 2015 CBTOE:</b> Flies were entrained in LD (light: dark) condition for 3-4 days and harvested at six time points</td>
        <td><i>Drosophila melanogaster</i></td>
        <!-- <td></td> -->
        <td>Liver</td>
        <td>Cbtoe</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Light-Dark</td>
        <!-- <td></td> -->
        <td>11, 15, 23, 19, 3, 7</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/25916830/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>DROSOPHILA KADENER 2015 CBTOE-CONTROL:</b> Flies were entrained in LD (light: dark) condition for 3-4 days and harvested at six time points</td>
        <td><i>Drosophila melanogaster</i></td>
        <!-- <td></td> -->
        <td>Liver</td>
        <td>Cbtoe-Control</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Light-Dark</td>
        <!-- <td></td> -->
        <td>11, 15, 23, 19, 3, 7</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/25916830/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>DROSOPHILA KADENER 2015 CBTRNAI:</b> Flies were entrained in LD (light: dark) condition for 3-4 days and harvested at six time points</td>
        <td><i>Drosophila melanogaster</i></td>
        <!-- <td></td> -->
        <td>Liver</td>
        <td>Cbtrnai</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Light-Dark</td>
        <!-- <td></td> -->
        <td>11, 15, 23, 19, 3, 7</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/25916830/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>DROSOPHILA KADENER 2015 CBTRNAI-CONTROL:</b> Flies were entrained in LD (light: dark) condition for 3-4 days and harvested at six time points</td>
        <td><i>Drosophila melanogaster</i></td>
        <!-- <td></td> -->
        <td>Liver</td>
        <td>Cbtrnai-Control</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Light-Dark</td>
        <!-- <td></td> -->
        <td>11, 15, 23, 19, 3, 7</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/25916830/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>DROSOPHILA TEMPERATURE KADENER 2019 18C:</b> RNA seq from fly heads at 3 temperatures (18, 25, 29C) and circadian timepoints (zt3, zt7, zt11, zt15, zt18, zt23)</td>
        <td><i>Drosophila melanogaster</i></td>
        <!-- <td>Temperature</td> -->
        <td>Brain</td>
        <td>18c</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Temperature</td>
        <!-- <td></td> -->
        <td>11, 15, 23, 19, 3, 7</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/31702556/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>DROSOPHILA TEMPERATURE KADENER 2019 25C:</b> RNA seq from fly heads at 3 temperatures (18, 25, 29C) and circadian timepoints (zt3, zt7, zt11, zt15, zt18, zt23)</td>
        <td><i>Drosophila melanogaster</i></td>
        <!-- <td>Temperature</td> -->
        <td>Brain</td>
        <td>25c</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Temperature</td>
        <!-- <td></td> -->
        <td>11, 15, 23, 19, 3, 7</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/31702556/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>DROSOPHILA TEMPERATURE KADENER 2019 29C:</b> RNA seq from fly heads at 3 temperatures (18, 25, 29C) and circadian timepoints (zt3, zt7, zt11, zt15, zt18, zt23)</td>
        <td><i>Drosophila melanogaster</i></td>
        <!-- <td>Temperature</td> -->
        <td>Brain</td>
        <td>29c</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Temperature</td>
        <!-- <td></td> -->
        <td>11, 15, 23, 19, 3, 7</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/31702556/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>DROSOPHILA TIM PROMOTER/ENHANCER TIM IN24:</b> Effects of regulatory element deletions on circadian gene expression</td>
        <td><i>Drosophila melanogaster</i></td>
        <!-- <td>Tim Promoter/enhancer Tim</td> -->
        <td>Brain</td>
        <td>In24</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE259245">GSE259245</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>3</td> -->
        <!-- <td>Male and female flies were entrained in 12:12 LD conditions for 4 days at 25C, then collected at ZT2, ZT6, ZT10, ZT14, ZT18 and ZT22.</td> -->
        <td><a href="https://www.pnas.org/doi/10.1073/pnas.2321338121">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>DROSOPHILA TIM PROMOTER/ENHANCER TIM UP10:</b> Effects of regulatory element deletions on circadian gene expression</td>
        <td><i>Drosophila melanogaster</i></td>
        <!-- <td>Tim Promoter/enhancer Tim</td> -->
        <td>Brain</td>
        <td>Up10</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE259245">GSE259245</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>3</td> -->
        <!-- <td>Male and female flies were entrained in 12:12 LD conditions for 4 days at 25C, then collected at ZT2, ZT6, ZT10, ZT14, ZT18 and ZT22.</td> -->
        <td><a href="https://www.pnas.org/doi/10.1073/pnas.2321338121">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>DROSOPHILA TIM PROMOTER/ENHANCER TIM UP122:</b> Effects of regulatory element deletions on circadian gene expression</td>
        <td><i>Drosophila melanogaster</i></td>
        <!-- <td>Tim Promoter/enhancer Tim</td> -->
        <td>Brain</td>
        <td>Up122</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE259245">GSE259245</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>3</td> -->
        <!-- <td>Male and female flies were entrained in 12:12 LD conditions for 4 days at 25C, then collected at ZT2, ZT6, ZT10, ZT14, ZT18 and ZT22.</td> -->
        <td><a href="https://www.pnas.org/doi/10.1073/pnas.2321338121">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>DROSOPHILA TIM PROMOTER/ENHANCER TIM UP126:</b> Effects of regulatory element deletions on circadian gene expression</td>
        <td><i>Drosophila melanogaster</i></td>
        <!-- <td>Tim Promoter/enhancer Tim</td> -->
        <td>Brain</td>
        <td>Up126</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE259245">GSE259245</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>3</td> -->
        <!-- <td>Male and female flies were entrained in 12:12 LD conditions for 4 days at 25C, then collected at ZT2, ZT6, ZT10, ZT14, ZT18 and ZT22.</td> -->
        <td><a href="https://www.pnas.org/doi/10.1073/pnas.2321338121">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>DROSOPHILA TIM PROMOTER/ENHANCER WT:</b> Effects of regulatory element deletions on circadian gene expression</td>
        <td><i>Drosophila melanogaster</i></td>
        <!-- <td>Tim Promoter/enhancer</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE259245">GSE259245</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>3</td> -->
        <!-- <td>Male and female flies were entrained in 12:12 LD conditions for 4 days at 25C, then collected at ZT2, ZT6, ZT10, ZT14, ZT18 and ZT22.</td> -->
        <td><a href="https://www.pnas.org/doi/10.1073/pnas.2321338121">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>EMBRYONIC HEART E10:</b> Involvement of posttranscriptional regulation of Clock in the emergence of circadian clock oscillation during mouse development, embryonic or young mouse heart</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Heart</td> -->
        <td>Heart</td>
        <td>E10</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 32, 44, 28, 36, 40, 0, 4, 8</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="http://www.pnas.org/content/114/36/E7479">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>EMBRYONIC HEART E17:</b> Involvement of posttranscriptional regulation of Clock in the emergence of circadian clock oscillation during mouse development, embryonic or young mouse heart</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Heart</td> -->
        <td>Heart</td>
        <td>E17</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 32, 44, 28, 36, 40, 0, 4, 8</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="http://www.pnas.org/content/114/36/E7479">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>EMBRYONIC HEART YOUNG:</b> Involvement of posttranscriptional regulation of Clock in the emergence of circadian clock oscillation during mouse development, embryonic or young mouse heart</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Heart</td> -->
        <td>Heart</td>
        <td>Young</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 32, 44, 28, 36, 40, 0, 4, 8</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="http://www.pnas.org/content/114/36/E7479">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL ARCUATE NUCLEUS HIGH-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Arcuate Nucleus</td> -->
        <td>Brain</td>
        <td>High-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL ARCUATE NUCLEUS LOW-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Arcuate Nucleus</td> -->
        <td>Brain</td>
        <td>Low-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL BRAINSTEM HIGH-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Brainstem</td> -->
        <td>Brain</td>
        <td>High-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL BRAINSTEM LOW-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Brainstem</td> -->
        <td>Brain</td>
        <td>Low-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL BROWN ADIPOSE HIGH-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Brown Adipose</td> -->
        <td>Arcuate nucleus</td>
        <td>High-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL BROWN ADIPOSE LOW-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Brown Adipose</td> -->
        <td>Arcuate nucleus</td>
        <td>Low-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL CEREBELLUM HIGH-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Cerebellum</td> -->
        <td>Brain</td>
        <td>High-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL CEREBELLUM LOW-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Cerebellum</td> -->
        <td>Brain</td>
        <td>Low-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL CORTEX HIGH-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Cortex</td> -->
        <td>Brain</td>
        <td>High-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL CORTEX LOW-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Cortex</td> -->
        <td>Brain</td>
        <td>Low-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL DORSOMEDIAL HYPOTHALAMUS HIGH-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Dorsomedial Hypothalamus</td> -->
        <td>Brain</td>
        <td>High-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL DORSOMEDIAL HYPOTHALAMUS LOW-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Dorsomedial Hypothalamus</td> -->
        <td>Brain</td>
        <td>Low-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL HABENULA HIGH-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Habenula</td> -->
        <td>Brain</td>
        <td>High-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL HABENULA LOW-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Habenula</td> -->
        <td>Brain</td>
        <td>Low-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL HIPPOCAMPUS HIGH-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Hippocampus</td> -->
        <td>Brain</td>
        <td>High-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL HIPPOCAMPUS LOW-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Hippocampus</td> -->
        <td>Brain</td>
        <td>Low-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL LATERAL HYPOTHALAMUS CAUDAL HIGH-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Lateral Hypothalamus Caudal</td> -->
        <td>Brain</td>
        <td>High-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL LATERAL HYPOTHALAMUS CAUDAL LOW-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Lateral Hypothalamus Caudal</td> -->
        <td>Brain</td>
        <td>Low-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL LATERAL HYPOTHALAMUS ROSTRAL HIGH-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Lateral Hypothalamus Rostral</td> -->
        <td>Brain</td>
        <td>High-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL LATERAL HYPOTHALAMUS ROSTRAL LOW-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Lateral Hypothalamus Rostral</td> -->
        <td>Brain</td>
        <td>Low-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL LIVER HIGH-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Liver</td> -->
        <td>Liver</td>
        <td>High-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL LIVER LOW-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Liver</td> -->
        <td>Liver</td>
        <td>Low-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL OLFACTORY BULB HIGH-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Olfactory Bulb</td> -->
        <td>Brain</td>
        <td>High-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL OLFACTORY BULB LOW-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Olfactory Bulb</td> -->
        <td>Brain</td>
        <td>Low-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL PARAVENTRICULAR NUCLEI HIGH-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Paraventricular Nuclei</td> -->
        <td>Brain</td>
        <td>High-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL PARAVENTRICULAR NUCLEI LOW-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Paraventricular Nuclei</td> -->
        <td>Brain</td>
        <td>Low-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL PARAVENTRICULAR NUCLEUS THALAMUS HIGH-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Paraventricular Nucleus Thalamus</td> -->
        <td>Brain</td>
        <td>High-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL PARAVENTRICULAR NUCLEUS THALAMUS LOW-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Paraventricular Nucleus Thalamus</td> -->
        <td>Brain</td>
        <td>Low-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL PERIVENTRICULAR ZONE HIGH-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Periventricular Zone</td> -->
        <td>Brain</td>
        <td>High-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL PERIVENTRICULAR ZONE LOW-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Periventricular Zone</td> -->
        <td>Brain</td>
        <td>Low-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL PREFRONTAL CORTEX HIGH-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Prefrontal Cortex</td> -->
        <td>Brain</td>
        <td>High-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL PREFRONTAL CORTEX LOW-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Prefrontal Cortex</td> -->
        <td>Brain</td>
        <td>Low-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL PREOPTIC AREA HIGH-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Preoptic Area</td> -->
        <td>Brain</td>
        <td>High-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL PREOPTIC AREA LOW-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Preoptic Area</td> -->
        <td>Brain</td>
        <td>Low-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL QUADRICEPS HIGH-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Quadriceps</td> -->
        <td>Muscle</td>
        <td>High-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL QUADRICEPS LOW-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Quadriceps</td> -->
        <td>Muscle</td>
        <td>Low-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL SUPRACHIASMATIC NUCLEUS HIGH-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Suprachiasmatic Nucleus</td> -->
        <td>Brain</td>
        <td>High-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL SUPRACHIASMATIC NUCLEUS LOW-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Suprachiasmatic Nucleus</td> -->
        <td>Brain</td>
        <td>Low-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL TESTIS HIGH-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Testis</td> -->
        <td>Reproductive</td>
        <td>High-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL TESTIS LOW-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Testis</td> -->
        <td>Reproductive</td>
        <td>Low-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL VENTROMEDIAL HYPOTHALAMUS HIGH-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Ventromedial Hypothalamus</td> -->
        <td>Brain</td>
        <td>High-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>ENERGY BALANCE DIURNAL NOCTURNAL VENTROMEDIAL HYPOTHALAMUS LOW-WORKLOAD:</b> Regulating wheel-running activity with food rewards, Switching between nocturnality and diurnality, and revealing distinct rhythmic gene expression in various tissues and brain regions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Balance Diurnal Nocturnal Ventromedial Hypothalamus</td> -->
        <td>Brain</td>
        <td>Low-Workload</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228967">GSE228967</a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA was extracted from neural and peripheral tissues using TRIzol reagent according to the manufacturer's instructions. 300ng of total RNA was used for the construction of sequencing libraries</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.113951">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>GUT KO TOMOKI 2021 KO:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Ko</td> -->
        <td>Digestive</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>GUT KO TOMOKI 2021 WT:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Ko</td> -->
        <td>Digestive</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>GUT MICROBIOTA AFFECT IN DUODENUM MALE GERM-FREE:</b> Rolee of gut microbiota in daily rhythms of gene expression and physiology</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Microbiota Affect In Duodenum Male</td> -->
        <td>Digestive</td>
        <td>Germ-Free</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE114399">GSE114399</a></td>
        <td>Microbiome</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA-Seq of conventional raised (convR) and Germ-free (GF) male under ad libitum feeding regime. Temporal mRNA profiles of duodenum and ileum were assessed every 4 hours during the diurnal cycle (Zeitgerber time (ZT) 2-22).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2018.09.023">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>GUT MICROBIOTA AFFECT IN DUODENUM MALE WT:</b> Rolee of gut microbiota in daily rhythms of gene expression and physiology</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Microbiota Affect In Duodenum Male</td> -->
        <td>Digestive</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE114399">GSE114399</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA-Seq of conventional raised (convR) and Germ-free (GF) male under ad libitum feeding regime. Temporal mRNA profiles of duodenum and ileum were assessed every 4 hours during the diurnal cycle (Zeitgerber time (ZT) 2-22).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2018.09.023">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>GUT MICROBIOTA AFFECT IN ILEUM MALE GERM-FREE:</b> Role of gut microbiota in daily rhythms of gene expression and physiology</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Microbiota Affect In Ileum Male</td> -->
        <td>Digestive</td>
        <td>Germ-Free</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE114399">GSE114399</a></td>
        <td>Microbiome</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA-Seq of conventional raised (convR) and Germ-free (GF) male under ad libitum feeding regime. Temporal mRNA profiles of duodenum and ileum were assessed every 4 hours during the diurnal cycle (Zeitgerber time (ZT) 2-22).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2018.09.023">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>GUT MICROBIOTA AFFECT IN ILEUM MALE WT:</b> Role of gut microbiota in daily rhythms of gene expression and physiology</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Microbiota Affect In Ileum Male</td> -->
        <td>Digestive</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE114399">GSE114399</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA-Seq of conventional raised (convR) and Germ-free (GF) male under ad libitum feeding regime. Temporal mRNA profiles of duodenum and ileum were assessed every 4 hours during the diurnal cycle (Zeitgerber time (ZT) 2-22).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2018.09.023">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>GUT MICROBIOTA AFFECT IN LIVER FEMALE GERM-FREE:</b> Role of gut microbiota in daily rhythms of gene expression and physiology</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Microbiota Affect In Liver Female</td> -->
        <td>Liver</td>
        <td>Germ-Free</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE114400">GSE114400</a></td>
        <td>Microbiome</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA-Seq of conventional raised (convR) and Germ-free (GF) female and male under ad libitum feeding regime. Temporal mRNA profiles of liver were assessed every 4 hours during the diurnal cycle (Zeitgerber time (ZT) 2-22).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2018.09.023">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>GUT MICROBIOTA AFFECT IN LIVER FEMALE WT:</b> Role of gut microbiota in daily rhythms of gene expression and physiology</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Microbiota Affect In Liver Female</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE114400">GSE114400</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA-Seq of conventional raised (convR) and Germ-free (GF) female and male under ad libitum feeding regime. Temporal mRNA profiles of liver were assessed every 4 hours during the diurnal cycle (Zeitgerber time (ZT) 2-22).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2018.09.023">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>GUT MICROBIOTA AFFECT IN LIVER MALE GERM-FREE:</b> Role of gut microbiota in daily rhythms of gene expression and physiology</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Microbiota Affect In Liver Male</td> -->
        <td>Liver</td>
        <td>Germ-Free</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE114400">GSE114400</a></td>
        <td>Microbiome</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA-Seq of conventional raised (convR) and Germ-free (GF) female and male under ad libitum feeding regime. Temporal mRNA profiles of liver were assessed every 4 hours during the diurnal cycle (Zeitgerber time (ZT) 2-22).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2018.09.023">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>GUT MICROBIOTA AFFECT IN LIVER MALE WT:</b> Role of gut microbiota in daily rhythms of gene expression and physiology</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Microbiota Affect In Liver Male</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE114400">GSE114400</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA-Seq of conventional raised (convR) and Germ-free (GF) female and male under ad libitum feeding regime. Temporal mRNA profiles of liver were assessed every 4 hours during the diurnal cycle (Zeitgerber time (ZT) 2-22).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2018.09.023">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>GUT MICROBIOTA AFFECT IN MALE CRY KO:</b> Role of gut microbiota in daily rhythms of gene expression and physiology</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Microbiota Affect In Male Cry</td> -->
        <td>Liver</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE114402">GSE114402</a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA-Seq of night fed Cry1/2 DKO male mice. Temporal mRNA profiles of liver were assessed every 4 hours during a diurnal cycle (Zeitgerber time (ZT) 2-22).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2018.09.023">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>GUT MICROBIOTA AFFECT IN WHITE ADIPOSE MALE GERM-FREE:</b> Role of gut microbiota in daily rhythms of gene expression and physiology</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Microbiota Affect In White Adipose Male</td> -->
        <td>Adipose</td>
        <td>Germ-Free</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE114401">GSE114401</a></td>
        <td>Microbiome</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA-Seq of conventional raised (convR) and Germ-free (GF) male under ad libitum feeding regime. Temporal mRNA profiles of WAT were assessed every 4 hours during the diurnal cycle (Zeitgerber time (ZT) 2-22).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2018.09.023">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>GUT MICROBIOTA AFFECT IN WHITE ADIPOSE MALE WT:</b> Role of gut microbiota in daily rhythms of gene expression and physiology</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Microbiota Affect In White Adipose Male</td> -->
        <td>Adipose</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE114401">GSE114401</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>3</td> -->
        <!-- <td>Total RNA-Seq of conventional raised (convR) and Germ-free (GF) male under ad libitum feeding regime. Temporal mRNA profiles of WAT were assessed every 4 hours during the diurnal cycle (Zeitgerber time (ZT) 2-22).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2018.09.023">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>GUT RE TOMOKI 2021 KO:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Re</td> -->
        <td>Digestive</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>GUT RE TOMOKI 2021 RE:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Re</td> -->
        <td>Digestive</td>
        <td>Re</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>GUT RE TOMOKI 2021 WT:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Re</td> -->
        <td>Digestive</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMAN BREAST GUTIERREZ 2016 CANCER:</b> Gene expression analysis of cancerous breast cell lines determine the degree to which the circadian clock is damaged.</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>Breast</td> -->
        <td>Cells</td>
        <td>Cancer</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE76370">GSE76370</a></td>
        <td>Cancer</td>
        <!-- <td>MCF-7</td> -->
        <td>24, 12, 20, 16, 28, 0, 4, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Serum shock to synchronize the circadian clocks and observe the time course of gene expression using cDNA microarrays</td> -->
        <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27010605">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMAN FIBROBLAST LIU 2017 MYOGENIC-REPROGRAMMING:</b> Human Fibroblast Reprogrammed to Myogenic Lineage via MyoD1</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>Fibroblast</td> -->
        <td>Cells</td>
        <td>Myogenic-Reprogramming</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54652">GSE54652</a></td>
        <td>Control</td>
        <!-- <td>Cornal Epithelial</td> -->
        <td>24, 16, 32, 40, 0, 8</td>
        <!-- <td>4</td> -->
        <!-- <td>Global bifurcation event delineated the transition into a myogenic cell identity 32 hours after exogenous MYOD1 activation.</td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMAN PANCREATIC DUCTAL ADENOCARCINOMA ASPC1:</b> Explored the presence of the circadian transcriptome in PDA using patient-derived organoids (PDOs) and validated these findings by comparing PDA data from TCGA with non-cancerous healthy pancreas data from GTeX</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>Pancreatic Ductal Adenocarcinoma</td> -->
        <td>Cells</td>
        <td>Aspc1</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE262627">GSE262627</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 0, 4, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Pancreas cancer tissue was collected from untreated patients undergoing tissue biopsy at Rush University Medical Center. Immediately following the tissue acquisition, the samples were processed as previously described. Briefly, after digestion with collagenase-II, dispase-II, and DNase-I, biopsy tissues were washed several times and then plated in 50 micoL Matrigel domes on a 24-welled plate supplemented with 500 microL organoid medium.</td> -->
        <td><a href="https://doi.org/10.1172/jci.insight.177697">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMAN PANCREATIC DUCTAL ADENOCARCINOMA CAPAN1:</b> Explored the presence of the circadian transcriptome in PDA using patient-derived organoids (PDOs) and validated these findings by comparing PDA data from TCGA with non-cancerous healthy pancreas data from GTeX</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>Pancreatic Ductal Adenocarcinoma</td> -->
        <td>Cells</td>
        <td>Capan1</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE262627">GSE262627</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 0, 4, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Pancreas cancer tissue was collected from untreated patients undergoing tissue biopsy at Rush University Medical Center. Immediately following the tissue acquisition, the samples were processed as previously described. Briefly, after digestion with collagenase-II, dispase-II, and DNase-I, biopsy tissues were washed several times and then plated in 50 micoL Matrigel domes on a 24-welled plate supplemented with 500 microL organoid medium.</td> -->
        <td><a href="https://doi.org/10.1172/jci.insight.177697">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMAN PANCREATIC DUCTAL ADENOCARCINOMA MIAPACA2:</b> Explored the presence of the circadian transcriptome in PDA using patient-derived organoids (PDOs) and validated these findings by comparing PDA data from TCGA with non-cancerous healthy pancreas data from GTeX</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>Pancreatic Ductal Adenocarcinoma</td> -->
        <td>Cells</td>
        <td>Miapaca2</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE262627">GSE262627</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 0, 4, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Pancreas cancer tissue was collected from untreated patients undergoing tissue biopsy at Rush University Medical Center. Immediately following the tissue acquisition, the samples were processed as previously described. Briefly, after digestion with collagenase-II, dispase-II, and DNase-I, biopsy tissues were washed several times and then plated in 50 micoL Matrigel domes on a 24-welled plate supplemented with 500 microL organoid medium.</td> -->
        <td><a href="https://doi.org/10.1172/jci.insight.177697">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMAN PANCREATIC DUCTAL ADENOCARCINOMA ORGANOID:</b> Explored the presence of the circadian transcriptome in PDA using patient-derived organoids (PDOs) and validated these findings by comparing PDA data from TCGA with non-cancerous healthy pancreas data from GTeX</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>Pancreatic Ductal Adenocarcinoma</td> -->
        <td>Organoid</td>
        <td>Organoid</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE262627">GSE262627</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 0, 4, 8</td>
        <!-- <td>7</td> -->
        <!-- <td>Pancreas cancer tissue was collected from untreated patients undergoing tissue biopsy at Rush University Medical Center. Immediately following the tissue acquisition, the samples were processed as previously described. Briefly, after digestion with collagenase-II, dispase-II, and DNase-I, biopsy tissues were washed several times and then plated in 50 microL Matrigel domes on a 24-welled plate supplemented with 500 microL organoid medium.</td> -->
        <td><a href="https://doi.org/10.1172/jci.insight.177697">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMAN PANCREATIC DUCTAL ADENOCARCINOMA PANC1:</b> Explored the presence of the circadian transcriptome in PDA using patient-derived organoids (PDOs) and validated these findings by comparing PDA data from TCGA with non-cancerous healthy pancreas data from GTeX</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>Pancreatic Ductal Adenocarcinoma</td> -->
        <td>Cells</td>
        <td>Panc1</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE262627">GSE262627</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 0, 4, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Pancreas cancer tissue was collected from untreated patients undergoing tissue biopsy at Rush University Medical Center. Immediately following the tissue acquisition, the samples were processed as previously described. Briefly, after digestion with collagenase-II, dispase-II, and DNase-I, biopsy tissues were washed several times and then plated in 50 micoL Matrigel domes on a 24-welled plate supplemented with 500 microL organoid medium.</td> -->
        <td><a href="https://doi.org/10.1172/jci.insight.177697">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMAN SERUM NA 2017 TBI-PATIENT-1:</b> Patients with inconsistent chrono-types diagnosed with TBI</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>Serum</td> -->
        <td>Blood</td>
        <td>Traumatic Brain Injury</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Disease</td>
        <!-- <td></td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMAN SERUM NA 2017 TBI-PATIENT-10:</b> Patients with inconsistent chrono-types diagnosed with TBI</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>Serum</td> -->
        <td>Blood</td>
        <td>Traumatic Brain Injury</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Disease</td>
        <!-- <td></td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMAN SERUM NA 2017 TBI-PATIENT-11:</b> Patients with inconsistent chrono-types diagnosed with TBI</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>Serum</td> -->
        <td>Blood</td>
        <td>Traumatic Brain Injury</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Disease</td>
        <!-- <td></td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMAN SERUM NA 2017 TBI-PATIENT-12:</b> Patients with inconsistent chrono-types diagnosed with TBI</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>Serum</td> -->
        <td>Blood</td>
        <td>Traumatic Brain Injury</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Disease</td>
        <!-- <td></td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMAN SERUM NA 2017 TBI-PATIENT-2:</b> Patients with inconsistent chrono-types diagnosed with TBI</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>Serum</td> -->
        <td>Blood</td>
        <td>Traumatic Brain Injury</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Disease</td>
        <!-- <td></td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMAN SERUM NA 2017 TBI-PATIENT-3:</b> Patients with inconsistent chrono-types diagnosed with TBI</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>Serum</td> -->
        <td>Blood</td>
        <td>Traumatic Brain Injury</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Disease</td>
        <!-- <td></td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMAN SERUM NA 2017 TBI-PATIENT-4:</b> Patients with inconsistent chrono-types diagnosed with TBI</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>Serum</td> -->
        <td>Blood</td>
        <td>Traumatic Brain Injury</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Disease</td>
        <!-- <td></td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMAN SERUM NA 2017 TBI-PATIENT-5:</b> Patients with inconsistent chrono-types diagnosed with TBI</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>Serum</td> -->
        <td>Blood</td>
        <td>Traumatic Brain Injury</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Disease</td>
        <!-- <td></td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMAN SERUM NA 2017 TBI-PATIENT-6:</b> Patients with inconsistent chrono-types diagnosed with TBI</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>Serum</td> -->
        <td>Blood</td>
        <td>Traumatic Brain Injury</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Disease</td>
        <!-- <td></td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMAN SERUM NA 2017 TBI-PATIENT-7:</b> Patients with inconsistent chrono-types diagnosed with TBI</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>Serum</td> -->
        <td>Blood</td>
        <td>Traumatic Brain Injury</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Disease</td>
        <!-- <td></td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMAN SERUM NA 2017 TBI-PATIENT-8:</b> Patients with inconsistent chrono-types diagnosed with TBI</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>Serum</td> -->
        <td>Blood</td>
        <td>Traumatic Brain Injury</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Disease</td>
        <!-- <td></td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMAN SERUM NA 2017 TBI-PATIENT-9:</b> Patients with inconsistent chrono-types diagnosed with TBI</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>Serum</td> -->
        <td>Blood</td>
        <td>Traumatic Brain Injury</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Disease</td>
        <!-- <td></td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMAN U2 OS HUGHES 2009 OSTEOSARCOMA:</b> Comparison of oscillating transcription from mouse liver, NIH3T3, and U2OS cells showing 12-hour oscillatory transcripts.</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>U2 Os</td> -->
        <td>Cells</td>
        <td>Osteosarcoma</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE11923">GSE11923</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>42, 48, 43, 24, 25, 26, 27, 20, 21, 22, 23, 46, 47, 44, 45, 28, 29, 40, 41, 1, 3, 2, 5, 4, 7, 6, 9, 8, 39, 38, 11, 10, 13, 12, 15, 14, 17, 16, 19, 18, 31, 30, 37, 36, 35, 34, 33, 32</td>
        <!-- <td>2</td> -->
        <!-- <td>U2OS cells were grown to confluence and schocked with 0.1 microM dexamethasone (Sigma).</td> -->
        <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19343201">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMANIZED FU 2024 HCC HUMAN CJ-HCC:</b> Human hepatocellular carcinoma (HCC) xenograft transcriptomes (from human hepatocytes) in chimeric mice under normal vs circadian-disrupted conditions, showing that chronic circadian disruption (CJ) shifts tumor gene expression toward more aggressive, poor-prognosis molecular profiles.</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>Fu 2024</td> -->
        <td>Tumor</td>
        <td>Cj-Hcc</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE205881">GSE205881</a></td>
        <td>Light-Dark, Disease</td>
        <!-- <td></td> -->
        <td>10, 2, 18</td>
        <!-- <td>7</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1016/j.jhep.2023.10.018">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMANIZED FU 2024 HCC HUMAN CONTROL-HCC:</b> Human hepatocellular carcinoma (HCC) xenograft transcriptomes (from human hepatocytes) in chimeric mice under normal vs circadian-disrupted conditions, showing that chronic circadian disruption (CJ) shifts tumor gene expression toward more aggressive, poor-prognosis molecular profiles.</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>Fu 2024</td> -->
        <td>Tumor</td>
        <td>Control-Hcc</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE205881">GSE205881</a></td>
        <td>Light-Dark, Disease</td>
        <!-- <td></td> -->
        <td>10, 2, 18</td>
        <!-- <td>7</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1016/j.jhep.2023.10.018">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMANIZED FU 2024 HCC MOUSE CJ-HCC:</b> Mouse stromal cell transcriptomes from liver tumors in humanized mice under normal vs circadian-disrupted conditions, indicating that circadian misalignment alters the host tumor microenvironment gene expression and immune/metabolic pathways in HCC.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Fu 2024</td> -->
        <td>Tumor</td>
        <td>Cj-Hcc</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE205881">GSE205881</a></td>
        <td>Light-Dark, Disease</td>
        <!-- <td></td> -->
        <td>10, 2, 18</td>
        <!-- <td>7</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1016/j.jhep.2023.10.018">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMANIZED FU 2024 HCC MOUSE CONTROL-HCC:</b> Mouse stromal cell transcriptomes from liver tumors in humanized mice under normal vs circadian-disrupted conditions, indicating that circadian misalignment alters the host tumor microenvironment gene expression and immune/metabolic pathways in HCC.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Fu 2024</td> -->
        <td>Tumor</td>
        <td>Control-Hcc</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE205881">GSE205881</a></td>
        <td>Light-Dark, Disease</td>
        <!-- <td></td> -->
        <td>10, 2, 18</td>
        <!-- <td>7</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1016/j.jhep.2023.10.018">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMANIZED FU 2024 LIVER HUMAN CONTROL-LIVER:</b> Human hepatocyte transcriptome in chimeric humanized mouse livers comparing control vs NASH diet conditions and circadian disruption (chronic jet lag), showing that circadian misalignment (CJ) reprograms rhythmic gene expression in NASH-affected human hepatocytes and exacerbates pro-tumorigenic transcriptomic profiles.</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>Fu 2024</td> -->
        <td>Liver</td>
        <td>Control-Liver</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE205881">GSE205881</a></td>
        <td>Light-Dark, Disease</td>
        <!-- <td></td> -->
        <td>10, 2, 18</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1016/j.jhep.2023.10.018">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMANIZED FU 2024 LIVER HUMAN NASH-CJ-LIVER:</b> Human hepatocyte transcriptome in chimeric humanized mouse livers comparing control vs NASH diet conditions and circadian disruption (chronic jet lag), showing that circadian misalignment (CJ) reprograms rhythmic gene expression in NASH-affected human hepatocytes and exacerbates pro-tumorigenic transcriptomic profiles.</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>Fu 2024</td> -->
        <td>Liver</td>
        <td>Nash-Cj-Liver</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE205881">GSE205881</a></td>
        <td>Light-Dark, Disease</td>
        <!-- <td></td> -->
        <td>10, 2, 18</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1016/j.jhep.2023.10.018">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMANIZED FU 2024 LIVER HUMAN NASH-CONTROL-LIVER:</b> Human hepatocyte transcriptome in chimeric humanized mouse livers comparing control vs NASH diet conditions and circadian disruption (chronic jet lag), showing that circadian misalignment (CJ) reprograms rhythmic gene expression in NASH-affected human hepatocytes and exacerbates pro-tumorigenic transcriptomic profiles.</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>Fu 2024</td> -->
        <td>Liver</td>
        <td>Nash-Control-Liver</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE205881">GSE205881</a></td>
        <td>Light-Dark, Disease</td>
        <!-- <td></td> -->
        <td>10, 2, 18</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1016/j.jhep.2023.10.018">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMANIZED FU 2024 LIVER MOUSE CONTROL-LIVER:</b> Mouse (host) liver transcriptome in humanized chimeric mice under energy-rich control vs NASH diet, with and without chronic circadian disruption (jet lag), indicating that host mouse gene expression also shifts with circadian misalignment and metabolic disease, paralleling changes in human hepatocyte rhythms.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Fu 2024</td> -->
        <td>Liver</td>
        <td>Control-Liver</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE205881">GSE205881</a></td>
        <td>Light-Dark, Disease</td>
        <!-- <td></td> -->
        <td>10, 2, 18</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1016/j.jhep.2023.10.018">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMANIZED FU 2024 LIVER MOUSE NASH-CJ-LIVER:</b> Mouse (host) liver transcriptome in humanized chimeric mice under energy-rich control vs NASH diet, with and without chronic circadian disruption (jet lag), indicating that host mouse gene expression also shifts with circadian misalignment and metabolic disease, paralleling changes in human hepatocyte rhythms.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Fu 2024</td> -->
        <td>Liver</td>
        <td>Nash-Cj-Liver</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE205881">GSE205881</a></td>
        <td>Light-Dark, Disease</td>
        <!-- <td></td> -->
        <td>10, 2, 18</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1016/j.jhep.2023.10.018">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HUMANIZED FU 2024 LIVER MOUSE NASH-CONTROL-LIVER:</b> Mouse (host) liver transcriptome in humanized chimeric mice under energy-rich control vs NASH diet, with and without chronic circadian disruption (jet lag), indicating that host mouse gene expression also shifts with circadian misalignment and metabolic disease, paralleling changes in human hepatocyte rhythms.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Fu 2024</td> -->
        <td>Liver</td>
        <td>Nash-Control-Liver</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE205881">GSE205881</a></td>
        <td>Light-Dark, Disease</td>
        <!-- <td></td> -->
        <td>10, 2, 18</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1016/j.jhep.2023.10.018">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HYPOXIA AND OBSTRUCTIVE SLEEP APNEA CER HYPOXIC:</b> Effect of intermittent hypoxia on 24-hour expression in lung, heart, liver, kidney, muscle, and cerebellum.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>And Obstructive Sleep Apnea Cer</td> -->
        <td>Brain</td>
        <td>Hypoxic</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214530">GSE214530</a></td>
        <td>Oxygen-Level</td>
        <!-- <td></td> -->
        <td>12, 15, 21, 18, 0, 3, 6, 9</td>
        <!-- <td>4</td> -->
        <!-- <td>The RNA from all tissues was extracted using a Trizol method In accordance with the manufacturer's protocol, 1 micro-gram of total RNA was used to construct the library using NEBNext UltraTM II Directional RNA Library Prep Kit for Illumina</td> -->
        <td><a href="https://doi.org/10.1371/journal.pbio.3002139">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HYPOXIA AND OBSTRUCTIVE SLEEP APNEA CER NORMOXIC:</b> Effect of intermittent hypoxia on 24-hour expression in lung, heart, liver, kidney, muscle, and cerebellum.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>And Obstructive Sleep Apnea Cer</td> -->
        <td>Brain</td>
        <td>Normoxic</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214530">GSE214530</a></td>
        <td>Oxygen-Level</td>
        <!-- <td></td> -->
        <td>12, 15, 21, 18, 0, 3, 6, 9</td>
        <!-- <td>4</td> -->
        <!-- <td>The RNA from all tissues was extracted using a Trizol method In accordance with the manufacturer's protocol, 1 micro-gram of total RNA was used to construct the library using NEBNext UltraTM II Directional RNA Library Prep Kit for Illumina</td> -->
        <td><a href="https://doi.org/10.1371/journal.pbio.3002139">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HYPOXIA AND OBSTRUCTIVE SLEEP APNEA HEART HYPOXIC:</b> Effect of intermittent hypoxia on 24-hour expression in lung, heart, liver, kidney, muscle, and cerebellum.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>And Obstructive Sleep Apnea Heart</td> -->
        <td>Heart</td>
        <td>Hypoxic</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214530">GSE214530</a></td>
        <td>Oxygen-Level</td>
        <!-- <td></td> -->
        <td>12, 15, 21, 18, 0, 3, 6, 9</td>
        <!-- <td>4</td> -->
        <!-- <td>The RNA from all tissues was extracted using a Trizol method In accordance with the manufacturer's protocol, 1 micro-gram of total RNA was used to construct the library using NEBNext UltraTM II Directional RNA Library Prep Kit for Illumina</td> -->
        <td><a href="https://doi.org/10.1371/journal.pbio.3002139">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HYPOXIA AND OBSTRUCTIVE SLEEP APNEA HEART NORMOXIC:</b> Effect of intermittent hypoxia on 24-hour expression in lung, heart, liver, kidney, muscle, and cerebellum.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>And Obstructive Sleep Apnea Heart</td> -->
        <td>Heart</td>
        <td>Normoxic</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214530">GSE214530</a></td>
        <td>Oxygen-Level</td>
        <!-- <td></td> -->
        <td>12, 15, 21, 18, 0, 3, 6, 9</td>
        <!-- <td>4</td> -->
        <!-- <td>The RNA from all tissues was extracted using a Trizol method In accordance with the manufacturer's protocol, 1 micro-gram of total RNA was used to construct the library using NEBNext UltraTM II Directional RNA Library Prep Kit for Illumina</td> -->
        <td><a href="https://doi.org/10.1371/journal.pbio.3002139">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HYPOXIA AND OBSTRUCTIVE SLEEP APNEA KIDNEY HYPOXIC:</b> Effect of intermittent hypoxia on 24-hour expression in lung, heart, liver, kidney, muscle, and cerebellum.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>And Obstructive Sleep Apnea Kidney</td> -->
        <td>Kidney</td>
        <td>Hypoxic</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214530">GSE214530</a></td>
        <td>Oxygen-Level</td>
        <!-- <td></td> -->
        <td>12, 15, 21, 18, 0, 3, 6, 9</td>
        <!-- <td>4</td> -->
        <!-- <td>The RNA from all tissues was extracted using a Trizol method In accordance with the manufacturer's protocol, 1 micro-gram of total RNA was used to construct the library using NEBNext UltraTM II Directional RNA Library Prep Kit for Illumina</td> -->
        <td><a href="https://doi.org/10.1371/journal.pbio.3002139">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HYPOXIA AND OBSTRUCTIVE SLEEP APNEA KIDNEY NORMOXIC:</b> Effect of intermittent hypoxia on 24-hour expression in lung, heart, liver, kidney, muscle, and cerebellum.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>And Obstructive Sleep Apnea Kidney</td> -->
        <td>Kidney</td>
        <td>Normoxic</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214530">GSE214530</a></td>
        <td>Oxygen-Level</td>
        <!-- <td></td> -->
        <td>12, 15, 21, 18, 0, 3, 6, 9</td>
        <!-- <td>4</td> -->
        <!-- <td>The RNA from all tissues was extracted using a Trizol method In accordance with the manufacturer's protocol, 1 micro-gram of total RNA was used to construct the library using NEBNext UltraTM II Directional RNA Library Prep Kit for Illumina</td> -->
        <td><a href="https://doi.org/10.1371/journal.pbio.3002139">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HYPOXIA AND OBSTRUCTIVE SLEEP APNEA LIVER HYPOXIC:</b> Effect of intermittent hypoxia on 24-hour expression in lung, heart, liver, kidney, muscle, and cerebellum.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>And Obstructive Sleep Apnea Liver</td> -->
        <td>Liver</td>
        <td>Hypoxic</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214530">GSE214530</a></td>
        <td>Oxygen-Level</td>
        <!-- <td></td> -->
        <td>12, 15, 21, 18, 0, 3, 6, 9</td>
        <!-- <td>4</td> -->
        <!-- <td>The RNA from all tissues was extracted using a Trizol method In accordance with the manufacturer's protocol, 1 micro-gram of total RNA was used to construct the library using NEBNext UltraTM II Directional RNA Library Prep Kit for Illumina</td> -->
        <td><a href="https://doi.org/10.1371/journal.pbio.3002139">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HYPOXIA AND OBSTRUCTIVE SLEEP APNEA LIVER NORMOXIC:</b> Effect of intermittent hypoxia on 24-hour expression in lung, heart, liver, kidney, muscle, and cerebellum.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>And Obstructive Sleep Apnea Liver</td> -->
        <td>Liver</td>
        <td>Normoxic</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214530">GSE214530</a></td>
        <td>Oxygen-Level</td>
        <!-- <td></td> -->
        <td>12, 15, 21, 18, 0, 3, 6, 9</td>
        <!-- <td>4</td> -->
        <!-- <td>The RNA from all tissues was extracted using a Trizol method In accordance with the manufacturer's protocol, 1 micro-gram of total RNA was used to construct the library using NEBNext UltraTM II Directional RNA Library Prep Kit for Illumina</td> -->
        <td><a href="https://doi.org/10.1371/journal.pbio.3002139">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HYPOXIA AND OBSTRUCTIVE SLEEP APNEA LUNG HYPOXIC:</b> Effect of intermittent hypoxia on 24-hour expression in lung, heart, liver, kidney, muscle, and cerebellum.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>And Obstructive Sleep Apnea Lung</td> -->
        <td>Lung</td>
        <td>Hypoxic</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214530">GSE214530</a></td>
        <td>Oxygen-Level</td>
        <!-- <td></td> -->
        <td>12, 15, 21, 18, 0, 3, 6, 9</td>
        <!-- <td>4</td> -->
        <!-- <td>The RNA from all tissues was extracted using a Trizol method In accordance with the manufacturer's protocol, 1 micro-gram of total RNA was used to construct the library using NEBNext UltraTM II Directional RNA Library Prep Kit for Illumina</td> -->
        <td><a href="https://doi.org/10.1371/journal.pbio.3002139">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HYPOXIA AND OBSTRUCTIVE SLEEP APNEA LUNG NORMOXIC:</b> Effect of intermittent hypoxia on 24-hour expression in lung, heart, liver, kidney, muscle, and cerebellum.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>And Obstructive Sleep Apnea Lung</td> -->
        <td>Lung</td>
        <td>Normoxic</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214530">GSE214530</a></td>
        <td>Oxygen-Level</td>
        <!-- <td></td> -->
        <td>12, 15, 21, 18, 0, 3, 6, 9</td>
        <!-- <td>4</td> -->
        <!-- <td>The RNA from all tissues was extracted using a Trizol method In accordance with the manufacturer's protocol, 1 micro-gram of total RNA was used to construct the library using NEBNext UltraTM II Directional RNA Library Prep Kit for Illumina</td> -->
        <td><a href="https://doi.org/10.1371/journal.pbio.3002139">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HYPOXIA AND OBSTRUCTIVE SLEEP APNEA MUSCLE HYPOXIC:</b> Effect of intermittent hypoxia on 24-hour expression in lung, heart, liver, kidney, muscle, and cerebellum.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>And Obstructive Sleep Apnea Muscle</td> -->
        <td>Muscle</td>
        <td>Hypoxic</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214530">GSE214530</a></td>
        <td>Oxygen-Level</td>
        <!-- <td></td> -->
        <td>12, 15, 21, 18, 0, 3, 6, 9</td>
        <!-- <td>4</td> -->
        <!-- <td>The RNA from all tissues was extracted using a Trizol method In accordance with the manufacturer's protocol, 1 micro-gram of total RNA was used to construct the library using NEBNext UltraTM II Directional RNA Library Prep Kit for Illumina</td> -->
        <td><a href="https://doi.org/10.1371/journal.pbio.3002139">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>HYPOXIA AND OBSTRUCTIVE SLEEP APNEA MUSCLE NORMOXIC:</b> Effect of intermittent hypoxia on 24-hour expression in lung, heart, liver, kidney, muscle, and cerebellum.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>And Obstructive Sleep Apnea Muscle</td> -->
        <td>Muscle</td>
        <td>Normoxic</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214530">GSE214530</a></td>
        <td>Oxygen-Level</td>
        <!-- <td></td> -->
        <td>12, 15, 21, 18, 0, 3, 6, 9</td>
        <!-- <td>4</td> -->
        <!-- <td>The RNA from all tissues was extracted using a Trizol method In accordance with the manufacturer's protocol, 1 micro-gram of total RNA was used to construct the library using NEBNext UltraTM II Directional RNA Library Prep Kit for Illumina</td> -->
        <td><a href="https://doi.org/10.1371/journal.pbio.3002139">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>IL10 CIRCADIAN INFLUENZA IMMUNE FEMALE-AB:</b> Mice were treated with an anti-IL-10 receptor blocking antibody (Ab) or an isotype-matched IgG control before influenza infection. Blocking IL-10 signalling (Ab groups) abolishes the usual time-of-day survival advantage and triggers exaggerated lung immune responses, independent of infection time.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Circadian Influenza Immune</td> -->
        <td>Lung</td>
        <td>Female-Ab</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE287759">GSE287759</a></td>
        <td>Sex, Disease</td>
        <!-- <td></td> -->
        <td>11, 23</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1101/2025.03.03.641134">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>IL10 CIRCADIAN INFLUENZA IMMUNE FEMALE-IGG:</b> Mice were treated with an anti-IL-10 receptor blocking antibody (Ab) or an isotype-matched IgG control before influenza infection. Blocking IL-10 signalling (Ab groups) abolishes the usual time-of-day survival advantage and triggers exaggerated lung immune responses, independent of infection time.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Circadian Influenza Immune</td> -->
        <td>Lung</td>
        <td>Female-Igg</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE287759">GSE287759</a></td>
        <td>Sex, Disease</td>
        <!-- <td></td> -->
        <td>11, 23</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1101/2025.03.03.641134">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>IL10 CIRCADIAN INFLUENZA IMMUNE MALE-AB:</b> Mice were treated with an anti-IL-10 receptor blocking antibody (Ab) or an isotype-matched IgG control before influenza infection. Blocking IL-10 signalling (Ab groups) abolishes the usual time-of-day survival advantage and triggers exaggerated lung immune responses, independent of infection time.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Circadian Influenza Immune</td> -->
        <td>Lung</td>
        <td>Male-Ab</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE287759">GSE287759</a></td>
        <td>Sex, Disease</td>
        <!-- <td></td> -->
        <td>11, 23</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1101/2025.03.03.641134">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>IL10 CIRCADIAN INFLUENZA IMMUNE MALE-IGG:</b> Mice were treated with an anti-IL-10 receptor blocking antibody (Ab) or an isotype-matched IgG control before influenza infection. Blocking IL-10 signalling (Ab groups) abolishes the usual time-of-day survival advantage and triggers exaggerated lung immune responses, independent of infection time.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Circadian Influenza Immune</td> -->
        <td>Lung</td>
        <td>Male-Igg</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE287759">GSE287759</a></td>
        <td>Sex, Disease</td>
        <!-- <td></td> -->
        <td>11, 23</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1101/2025.03.03.641134">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>INSULIN AND LEPTIN OSCILLATIONS AD-LIB:</b> Effect of timed-feeding (Ad-Lib or twice-a-night) on causing biphasic oscillations of circulating insulin and leptin.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>And Leptin Oscillations</td> -->
        <td>iWAT</td>
        <td>Ad-Lib</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE264173">GSE264173</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>5</td> -->
        <!-- <td>Total RNA was isolated using the Trizol Reagent and further purified with the RNeasy Plus kit after genomic DNA elimination. Messenger RNA was extracted, fragmented, and converted into cDNA, followed by library preparation with either directional or non-directional protocols, then quantified and checked for size distribution before being pooled and sequenced on Illumina platforms.</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.114390">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>INSULIN AND LEPTIN OSCILLATIONS TWICE-A-NIGHT:</b> Effect of timed-feeding (Ad-Lib or twice-a-night) on causing biphasic oscillations of circulating insulin and leptin.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>And Leptin Oscillations</td> -->
        <td>iWAT</td>
        <td>Twice-A-Night</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE264173">GSE264173</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>5</td> -->
        <!-- <td>Total RNA was isolated using the Trizol Reagent and further purified with the RNeasy Plus kit after genomic DNA elimination. Messenger RNA was extracted, fragmented, and converted into cDNA, followed by library preparation with either directional or non-directional protocols, then quantified and checked for size distribution before being pooled and sequenced on Illumina platforms.</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.114390">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>INTESTINAL BMAL1 KO COLITIS WT:</b> Intestinal-epithelium Bmal1 knockout remodels apoptotic and barrier pathways and lessens DSS-induced colitis severity, linking the local clock to gut inflammation.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Bmal1 Ko Colitis</td> -->
        <td>Digestive</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE271174">GSE271174</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>0, 12, 18, 6</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1038/s41467-024-55487-1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>KETONE BODIES AND KETOGENESIS FEMALE AD-LIBITUM:</b> Sex difference in liver transcriptome of mice on calorie restriction (CR) diet versus ad libitum (AL) fed mice</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Bodies And Ketogenesis Female</td> -->
        <td>Liver</td>
        <td>Ad-Libitum</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE216416">GSE216416</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>4</td> -->
        <!-- <td>After wean, mice were maintained on 5008 LabDiet (proteins 26.5%, fat 16.9%, carbohydrates 56.5%) with ad-libitum access to food and water</td> -->
        <td><a href="https://doi.org/10.1016/j.isci.2024.109483">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>KETONE BODIES AND KETOGENESIS FEMALE CALORIE-RESTRICTION:</b> Sex difference in liver transcriptome of mice on calorie restriction (CR) diet versus ad libitum (AL) fed mice</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Bodies And Ketogenesis Female</td> -->
        <td>Liver</td>
        <td>Calorie-Restriction</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE216416">GSE216416</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>4</td> -->
        <!-- <td>After wean, mice were maintained on 5008 LabDiet (proteins 26.5%, fat 16.9%, carbohydrates 56.5%) with ad-libitum access to food and water</td> -->
        <td><a href="https://doi.org/10.1016/j.isci.2024.109483">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>KETONE BODIES AND KETOGENESIS MALE AD-LIBITUM:</b> Liver transcriptome of mice on calorie restriction (CR) diet versus ad libitum (AL) fed mice</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Bodies And Ketogenesis Male</td> -->
        <td>Liver</td>
        <td>Ad-Libitum</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE211975">GSE211975</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>4</td> -->
        <!-- <td>Mice were maintained on 12 h light: 12 h dark cycle, the light was on at 7AM (ZT0) and the light was off at 7PM (ZT12). Animals were fed 5008 LabDiet (proteins 26.5%, fat 16.9%, carbohydrates 56.5%). CR mice received 70 percent of food once per day at ZT14 (2 hours after the light was turned off). Liver tissues were collected across the day and immediately frozen and stored at -80C. At ZT14, tissues were collected before feeding.</td> -->
        <td><a href="https://doi.org/10.1073/pnas.2205755119">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>KETONE BODIES AND KETOGENESIS MALE CALORIE-RESTRICTION:</b> Liver transcriptome of mice on calorie restriction (CR) diet versus ad libitum (AL) fed mice</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Bodies And Ketogenesis Male</td> -->
        <td>Liver</td>
        <td>Calorie-Restriction</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE211975">GSE211975</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>4</td> -->
        <!-- <td>Mice were maintained on 12 h light: 12 h dark cycle, the light was on at 7AM (ZT0) and the light was off at 7PM (ZT12). Animals were fed 5008 LabDiet (proteins 26.5%, fat 16.9%, carbohydrates 56.5%). CR mice received 70 percent of food once per day at ZT14 (2 hours after the light was turned off). Liver tissues were collected across the day and immediately frozen and stored at -80C. At ZT14, tissues were collected before feeding.</td> -->
        <td><a href="https://doi.org/10.1073/pnas.2205755119">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>KIDNEY CCRCC BMAL1 HIF2A CONTROL:</b> Clear-cell renal-carcinoma xenografts were treated either with vehicle (Control) or PT2399, a selective HIF-2a antagonist (HIF2a_Inhibitor). Tumours with high BMAL1 expression show stronger transcriptional re-programming of hypoxia targets and greater growth inhibition after PT2399 therapy, implicating BMAL1 as a biomarker of HIF-2a-dependency.</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>Ccrcc Bmal1 Hif2a</td> -->
        <td>Kidney</td>
        <td>Control</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE290779">GSE290779</a></td>
        <td>Drug</td>
        <!-- <td></td> -->
        <td>0, 12</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1101/2024.06.07.597806">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>KIDNEY CCRCC BMAL1 HIF2A HIF2A-INHIBITOR:</b> Clear-cell renal-carcinoma xenografts were treated either with vehicle (Control) or PT2399, a selective HIF-2a antagonist (HIF2a_Inhibitor). Tumours with high BMAL1 expression show stronger transcriptional re-programming of hypoxia targets and greater growth inhibition after PT2399 therapy, implicating BMAL1 as a biomarker of HIF-2a-dependency.</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>Ccrcc Bmal1 Hif2a</td> -->
        <td>Kidney</td>
        <td>Hif2a-Inhibitor</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE290779">GSE290779</a></td>
        <td>Drug</td>
        <!-- <td></td> -->
        <td>0, 12</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1101/2024.06.07.597806">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>LAWRENCE MICROGLIA DIURNAL BBB BREAKDOWN CONTROL:</b> Evening vs morning systemic lipopolysaccharide (LPS) challenge causes greater neuroinflammation and blood-brain barrier (BBB) breakdown, highlighting microglia-dependent time-of-day differences in  BBB disruption.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Microglia Diurnal Bbb Breakdown</td> -->
        <td>Brain</td>
        <td>Control</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE263794">GSE263794</a></td>
        <td>Inflammation</td>
        <!-- <td></td> -->
        <td>1, 13</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1172/jci.insight.180081">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>LAWRENCE MICROGLIA DIURNAL BBB BREAKDOWN LPS:</b> Evening vs morning systemic lipopolysaccharide (LPS) challenge causes greater neuroinflammation and blood-brain barrier (BBB) breakdown, highlighting microglia-dependent time-of-day differences in  BBB disruption.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Microglia Diurnal Bbb Breakdown</td> -->
        <td>Brain</td>
        <td>Lps</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE263794">GSE263794</a></td>
        <td>Inflammation</td>
        <!-- <td></td> -->
        <td>1, 13</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1172/jci.insight.180081">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>LIGHTCUE DISRUPTION INFLUENZA LD-FEMALE:</b> Normal light-dark cues confer circadian protection against influenza; light disruption worsens disease unless feeding-time rhythms remain intact.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Disruption Influenza</td> -->
        <td>Lung</td>
        <td>Ld-Female</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE288858">GSE288858</a></td>
        <td>Light-Dark, Disease</td>
        <!-- <td></td> -->
        <td>11, 23</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1101/2025.03.03.641279">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>LIVER BMAL1 KNOCKOUT SCIDATA 2025 FLOX:</b> Temporal RNA-seq at CT2 & CT14 compares wild-type, flox/flox controls, whole-body Bmal1-/- (Global_KO) and hepatocyte-specific Bmal1-/- (Liver_KO), showing broad loss or re-programming of liver rhythmic genes.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Bmal1 Knockout</td> -->
        <td>Liver</td>
        <td>Flox</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE284601">GSE284601</a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>2, 14</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1038/s41597-025-04545-4">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>LIVER BMAL1 KNOCKOUT SCIDATA 2025 GLOBAL-KO:</b> Temporal RNA-seq at CT2 & CT14 compares wild-type, flox/flox controls, whole-body Bmal1-/- (Global_KO) and hepatocyte-specific Bmal1-/- (Liver_KO), showing broad loss or re-programming of liver rhythmic genes.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Bmal1 Knockout</td> -->
        <td>Liver</td>
        <td>Global-KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE284601">GSE284601</a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>2, 14</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1038/s41597-025-04545-4">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>LIVER BMAL1 KNOCKOUT SCIDATA 2025 LIVER-KO:</b> Temporal RNA-seq at CT2 & CT14 compares wild-type, flox/flox controls, whole-body Bmal1-/- (Global_KO) and hepatocyte-specific Bmal1-/- (Liver_KO), showing broad loss or re-programming of liver rhythmic genes.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Bmal1 Knockout</td> -->
        <td>Liver</td>
        <td>Liver-KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE284601">GSE284601</a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>2, 14</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1038/s41597-025-04545-4">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>LIVER BMAL1 KNOCKOUT SCIDATA 2025 WT:</b> Temporal RNA-seq at CT2 & CT14 compares wild-type, flox/flox controls, whole-body Bmal1-/- (Global_KO) and hepatocyte-specific Bmal1-/- (Liver_KO), showing broad loss or re-programming of liver rhythmic genes.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Bmal1 Knockout</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE284601">GSE284601</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>2, 14</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1038/s41597-025-04545-4">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MATERNAL DIET AFFECT ON 16 WEEKS OFFSPRING HIGH-FAT:</b> Maternal diet's impact  on circadian rhythms of 4-week and 16-week offspring</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Diet Affect On 16 Weeks Offspring</td> -->
        <td>Liver</td>
        <td>High-Fat</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE240147">GSE240147</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>4</td> -->
        <!-- <td>RNA-Seq from mouse liver mRNA of 4-week and 16-week offspring from maternal control diet mice and maternal high-fat diet mice</td> -->
        <td><a href="https://doi.org/10.1016/j.isci.2024.110343">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MATERNAL DIET AFFECT ON 16 WEEKS OFFSPRING NORMAL-CHOW:</b> Maternal diet's impact  on circadian rhythms of 4-week and 16-week offspring</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Diet Affect On 16 Weeks Offspring</td> -->
        <td>Liver</td>
        <td>Normal-Chow</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE240147">GSE240147</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>4</td> -->
        <!-- <td>RNA-Seq from mouse liver mRNA of 4-week and 16-week offspring from maternal control diet mice and maternal high-fat diet mice</td> -->
        <td><a href="https://doi.org/10.1016/j.isci.2024.110343">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MATERNAL DIET AFFECT ON 4 WEEKS OFFSPRING HIGH-FAT:</b> Maternal diet's impact  on circadian rhythms of 4-week and 16-week offspring</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Diet Affect On 4 Weeks Offspring</td> -->
        <td>Liver</td>
        <td>High-Fat</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE240147">GSE240147</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>4</td> -->
        <!-- <td>RNA-Seq from mouse liver mRNA of 4-week and 16-week offspring from maternal control diet mice and maternal high-fat diet mice</td> -->
        <td><a href="https://doi.org/10.1016/j.isci.2024.110343">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MATERNAL DIET AFFECT ON 4 WEEKS OFFSPRING NORMAL-CHOW:</b> Maternal diet's impact  on circadian rhythms of 4-week and 16-week offspring</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Diet Affect On 4 Weeks Offspring</td> -->
        <td>Liver</td>
        <td>Normal-Chow</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE240147">GSE240147</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>4</td> -->
        <!-- <td>RNA-Seq from mouse liver mRNA of 4-week and 16-week offspring from maternal control diet mice and maternal high-fat diet mice</td> -->
        <td><a href="https://doi.org/10.1016/j.isci.2024.110343">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MONKEY ADRENAL GLAND LEMOS 2006 WT:</b> Genome-wide expression profiling to determine whether the adrenal gland of rhesus monkeys shows temporal gene expression across a 24-h period.</td>
        <td><i>Rhesus macaques</i></td>
        <!-- <td>Adrenal Gland</td> -->
        <td>Glands</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE2703">GSE2703</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>11, 15, 23, 19, 3, 7</td>
        <!-- <td>2</td> -->
        <!-- <td>Six female rhesus macaques were maintained under a 12-h light, 12-h dark photoperiod, lights on at 0700 h [ZT0].</td> -->
        <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16439464">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOSQUITO EMBRYO GOLTSEV 2009 DEVELOPMENT:</b>  Detailed temporal microarray assays of mosquito gene expression profiles revealed that the cuticular genes display biphasic expression during A. gambiae embryogenesi</td>
        <td><i>Anopheles gambiae</i></td>
        <!-- <td>Embryo</td> -->
        <td>Embryo</td>
        <td>Development</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE15001">GSE15001</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>10, 13, 22, 16, 19, 37, 31, 28, 43, 40, 34, 46, 2, 4, 7, 6, 8, 25</td>
        <!-- <td>2</td> -->
        <!-- <td>Population was reared with a 12-h light/dark cycle. Adults were maintained on a 10 percent sucrose solution and females were blood-fed on anesthetized hamsters. Eggs sampled every 2 hours.</td> -->
        <td><a href="https://www.sciencedirect.com/science/article/pii/S0012160609001584">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOSQUITO HEAD PTITSYN 2011 WT:</b> Document circadian rhythms in multiple molecular pathways essential for growth, development, immune response, detoxification/pesticide resistance.</td>
        <td><i>Aedes aegypti</i></td>
        <!-- <td>Head</td> -->
        <td>Head</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>76, 88, 72, 80, 92, 84</td>
        <!-- <td>2</td> -->
        <!-- <td>Transcripts were collected from the heads of females that were 72, 76, 80, 84, 88, 92, 96, 100 and 104 hours old.</td> -->
        <td><a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-12-153">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOSQUITO HEAD RUND 2011 DARK-DARK:</b> DNA microarray analysis of An. gambiae under light/dark cycle (LD) and constant dark (DD) conditions.</td>
        <td><i>Anopheles gambiae</i></td>
        <!-- <td>Head</td> -->
        <td>Head</td>
        <td>Dark-Dark</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 48, 16, 32, 44, 28, 36, 40, 0, 4, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>Adult mated, non blood fed female mosquitoes were collected every 4 hours for 48 hours</td> -->
        <td><a href="http://www.pnas.org/content/108/32/E421.short">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOSQUITO LARVA KOUTSOS 2007 DEVELOPMENT:</b> Genome-wide survey of mosquito gene expression profiles clustered temporally into developmental programs and spatially into adult tissue-specific patterns.</td>
        <td><i>Anopheles gambiae</i></td>
        <!-- <td>Larva</td> -->
        <td>Larva</td>
        <td>Development</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>144, 192, 240, 48, 96</td>
        <!-- <td>2</td> -->
        <!-- <td>Mosquito life cycle was sampled empirically at eight successive time periods: embryos, five larval stages, pupae, and adult females and males.</td> -->
        <td><a href="http://www.pnas.org/content/104/27/11304.short">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOSQUITO MID GUT MARINOTTI 2006 BLOOD-FED:</b> Examining sex-differential changes in gene expression after blood meal.</td>
        <td><i>Anopheles gambiae</i></td>
        <!-- <td>Mid Gut</td> -->
        <td>Digestive</td>
        <td>Blood-Fed</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 3, 72, 48, 96</td>
        <!-- <td>2</td> -->
        <!-- <td>Sampling after blood meal 3 h, 24 h, 48 h, 72 h, 96 h</td> -->
        <td><a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2583.2006.00610.x/full">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOSQUITO THORAX CHOI 2014 LIVERPOOL-INFECTED:</b> Dual RNA-seq time course analysis of Brugia Malayi parasite and host mosquito.</td>
        <td><i>Aedes aegypti</i></td>
        <!-- <td>Thorax</td> -->
        <td>Thoracic</td>
        <td>Infected</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td>Liverpool, RED</td> -->
        <td>24, 72, 48, 96</td>
        <!-- <td>2</td> -->
        <!-- <td>Thoracic tissues were collected every 24 hours from mosquitoes exposed to an infected or uninfected blood meal.</td> -->
        <td><a href="http://journals.plos.org/plosntds/article?id=10.1371/journal.pntd.0002905">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOSQUITO THORAX CHOI 2014 RED-INFECTED:</b> Dual RNA-seq time course analysis of Brugia Malayi parasite and host mosquito.</td>
        <td><i>Aedes aegypti</i></td>
        <!-- <td>Thorax</td> -->
        <td>Thoracic</td>
        <td>Infected</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td>Liverpool, RED</td> -->
        <td>24, 72, 48, 96</td>
        <!-- <td>2</td> -->
        <!-- <td>Thoracic tissues were collected every 24 hours from mosquitoes exposed to an infected or uninfected blood meal.</td> -->
        <td><a href="http://journals.plos.org/plosntds/article?id=10.1371/journal.pntd.0002905">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOSQUITOE CIRCADIAN DISRUPTION BY AECYC KO:</b> Impact of disrupting the circadian clock through a Cycle gene knockout (KO) on the transcriptome of Aedes aegypti mosquitoes.</td>
        <td><i>Aedes aegypti</i></td>
        <!-- <td>Circadian Disruption By Aecyc</td> -->
        <td>Whole Body</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE241953">GSE241953</a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>19, 11, 15, 7</td>
        <!-- <td>7</td> -->
        <!-- <td>Total RNA was isolated from each sample using miRNeasy (Qiagen) columns using the protocol provided by Qiagen. Messenger RNA was enriched from about 1 micro g of total RNA using the Poly-A enrichment method and cDNA libraries were prepared using an Illumina TruSeq RNA Library kit (Illumina).</td> -->
        <td><a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-024-10078-8">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOSQUITOE CIRCADIAN DISRUPTION BY AECYC WT:</b> Impact of disrupting the circadian clock through a Cycle gene knockout (KO) on the transcriptome of Aedes aegypti mosquitoes.</td>
        <td><i>Aedes aegypti</i></td>
        <!-- <td>Circadian Disruption By Aecyc</td> -->
        <td>Whole Body</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE241953">GSE241953</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>19, 11, 15, 7</td>
        <!-- <td>7</td> -->
        <!-- <td>Total RNA was isolated from each sample using miRNeasy (Qiagen) columns using the protocol provided by Qiagen. Messenger RNA was enriched from about 1 micro g of total RNA using the Poly-A enrichment method and cDNA libraries were prepared using an Illumina TruSeq RNA Library kit (Illumina).</td> -->
        <td><a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-024-10078-8">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE 3T3 HUGHES 2009 WT:</b> Comparison of oscillating transcription from mouse liver, NIH3T3, and U2OS cells showing 12-hour oscillatory transcripts.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>3t3</td> -->
        <td>Cells</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE11923">GSE11923</a></td>
        <td>Control</td>
        <!-- <td>3T3</td> -->
        <td>42, 48, 43, 24, 25, 26, 27, 20, 21, 22, 23, 46, 47, 44, 45, 28, 29, 40, 41, 1, 3, 2, 5, 4, 7, 6, 9, 8, 39, 38, 11, 10, 13, 12, 15, 14, 17, 16, 19, 18, 31, 30, 37, 36, 35, 34, 33, 32</td>
        <!-- <td>2</td> -->
        <!-- <td>NIH3T3 cells (ATCC) were grown to confluence and synchronized with 10 microM forskolin (Sigma).</td> -->
        <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19343201">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE ADRENAL GLAND NA 2018 WT:</b> Mouse Adrenal Gland Transcriptome</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Adrenal Gland</td> -->
        <td>Glands</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>38, 58, 46, 54, 42, 50, 62, 74, 66, 70, 82, 78</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE ADRENAL GLAND ZHANG 2014 WT:</b> Wild-type C57/BL6 mouse tissue microarray in the Circadian Atlas project.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Adrenal Gland</td> -->
        <td>Glands</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54652">GSE54652</a></td>
        <td>Control</td>
        <!-- <td>C57/BL6</td> -->
        <td>56, 42, 50, 60, 62, 64, 32, 24, 26, 20, 48, 46, 44, 28, 40, 52, 58, 38, 22, 54, 30, 36, 34, 18</td>
        <!-- <td>2</td> -->
        <!-- <td>6-wk-old male C57/BL6 mice entrained to a 12h:12h light:dark schedule for 1 week, then released into constant darkness.</td> -->
        <td><a href="http://www.pnas.org/content/111/45/16219.abstract">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE AORTA RUDIC 2005 WT:</b> Bioinformatic Analysis of Circadian Gene Oscillation in Mouse Aorta</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Aorta</td> -->
        <td>Heart</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>26, 38, 58, 22, 46, 18, 30, 42, 50, 34, 62, 54</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="http://circ.ahajournals.org/content/112/17/2716">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE AORTA ZHANG 2014 WT:</b> Wild-type C57/BL6 mouse tissue microarray in the Circadian Atlas project.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Aorta</td> -->
        <td>Heart</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54652">GSE54652</a></td>
        <td>Control</td>
        <!-- <td>C57/BL6</td> -->
        <td>56, 42, 50, 60, 62, 64, 32, 24, 26, 20, 48, 46, 44, 28, 40, 52, 58, 38, 22, 54, 30, 36, 34, 18</td>
        <!-- <td>2</td> -->
        <!-- <td>6-wk-old male C57/BL6 mice entrained to a 12h:12h light:dark schedule for 1 week, then released into constant darkness.</td> -->
        <td><a href="http://www.pnas.org/content/111/45/16219.abstract">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE BMAL1 12H PHASED CONTROL-AM:</b> 12 hours apart MSF synchronization and experimenting the knock-out of BMAL1.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Bmal1 12h Phased</td> -->
        <td>Skin</td>
        <td>Control-Am</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134333">GSE134333</a></td>
        <td>Control, Light-Dark</td>
        <!-- <td>C57BL/6</td> -->
        <td>24, 39, 12, 15, 21, 48, 33, 18, 30, 42, 36, 0, 3, 27, 6, 9, 45</td>
        <!-- <td>2</td> -->
        <!-- <td>Schematic representation of the experimental strategy used in temperature compensation analysis of the rhytmic transcriptome. MSFs(wild-type and Bmal1-/-) were synchronized by one DEX pulse and maintained at a constant temperature of 27, 32, or 37C for two complete circadian cycles and were subsequently sampled every 2 hours at these three different temperatures under constant conditions.</td> -->
        <td><a href="https://www.science.org/doi/10.1126/science.aaw7365">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE BMAL1 12H PHASED CONTROL-PM:</b> 12 hours apart MSF synchronization and experimenting the knock-out of BMAL1.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Bmal1 12h Phased</td> -->
        <td>Skin</td>
        <td>Control-Pm</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134333">GSE134333</a></td>
        <td>Control, Light-Dark</td>
        <!-- <td>C57BL/6</td> -->
        <td>24, 39, 12, 15, 21, 48, 33, 18, 30, 42, 36, 0, 3, 27, 6, 9, 45</td>
        <!-- <td>2</td> -->
        <!-- <td>Schematic representation of the experimental strategy used in temperature compensation analysis of the rhytmic transcriptome. MSFs(wild-type and Bmal1-/-) were synchronized by one DEX pulse and maintained at a constant temperature of 27, 32, or 37C for two complete circadian cycles and were subsequently sampled every 2 hours at these three different temperatures under constant conditions.</td> -->
        <td><a href="https://www.science.org/doi/10.1126/science.aaw7365">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE BMAL1 12H PHASED KO-AM:</b> 12 hours apart MSF synchronization and experimenting the knock-out of BMAL1.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Bmal1 12h Phased</td> -->
        <td>Skin</td>
        <td>KO-Am</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134333">GSE134333</a></td>
        <td>Knock-Out, Light-Dark</td>
        <!-- <td>C57BL/6</td> -->
        <td>24, 39, 12, 15, 21, 48, 33, 18, 30, 42, 36, 0, 3, 27, 6, 9, 45</td>
        <!-- <td>2</td> -->
        <!-- <td>Schematic representation of the experimental strategy used in temperature compensation analysis of the rhytmic transcriptome. MSFs(wild-type and Bmal1-/-) were synchronized by one DEX pulse and maintained at a constant temperature of 27, 32, or 37C for two complete circadian cycles and were subsequently sampled every 2 hours at these three different temperatures under constant conditions.</td> -->
        <td><a href="https://www.science.org/doi/10.1126/science.aaw7365">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE BMAL1 12H PHASED KO-PM:</b> 12 hours apart MSF synchronization and experimenting the knock-out of BMAL1.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Bmal1 12h Phased</td> -->
        <td>Skin</td>
        <td>KO-Pm</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134333">GSE134333</a></td>
        <td>Knock-Out, Light-Dark</td>
        <!-- <td>C57BL/6</td> -->
        <td>24, 39, 12, 15, 21, 48, 33, 18, 30, 42, 36, 0, 3, 27, 6, 9, 45</td>
        <!-- <td>2</td> -->
        <!-- <td>Schematic representation of the experimental strategy used in temperature compensation analysis of the rhytmic transcriptome. MSFs(wild-type and Bmal1-/-) were synchronized by one DEX pulse and maintained at a constant temperature of 27, 32, or 37C for two complete circadian cycles and were subsequently sampled every 2 hours at these three different temperatures under constant conditions.</td> -->
        <td><a href="https://www.science.org/doi/10.1126/science.aaw7365">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE BMAL1 EFFECTS ON AGING AND SURVIVAL KO:</b> Bmal1 elimination and certain aging-related phenotypes without affecting lifespan or metabolism, suggesting a nuanced role of Bmal1 beyond its traditional circadian functions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Bmal1 Effects On Aging And Survival</td> -->
        <td>Liver</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE70499">GSE70499</a></td>
        <td>Knock-Out</td>
        <!-- <td>C57BL/6</td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td>Bmal1 knockout mice were housed under 12h:12h light-dark conditions and sacrificed every 4 hours over a 20-hour period to extract liver RNA, aiming for comprehensive gene expression profiling. The total RNA was extracted using the TruSeq Stranded mRNA Sample Prep Kit Set A, followed by library construction and amplification processes that included barcoding, PCR amplification, library dilution and quality assessment, clustering on the cBot, and template DNA preparation for sequencing. This meticulous process was designed to ensure accurate gene expression analysis across different time points.</td> -->
        <td><a href="https://www.science.org/doi/10.1126/scitranslmed.aad3305">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE BMAL1 EFFECTS ON AGING AND SURVIVAL WT:</b> Bmal1 elimination and certain aging-related phenotypes without affecting lifespan or metabolism, suggesting a nuanced role of Bmal1 beyond its traditional circadian functions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Bmal1 Effects On Aging And Survival</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE70499">GSE70499</a></td>
        <td>Control</td>
        <!-- <td>C57BL/6</td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td>Bmal1 knockout mice were housed under 12h:12h light-dark conditions and sacrificed every 4 hours over a 20-hour period to extract liver RNA, aiming for comprehensive gene expression profiling. The total RNA was extracted using the TruSeq Stranded mRNA Sample Prep Kit Set A, followed by library construction and amplification processes that included barcoding, PCR amplification, library dilution and quality assessment, clustering on the cBot, and template DNA preparation for sequencing. This meticulous process was designed to ensure accurate gene expression analysis across different time points.</td> -->
        <td><a href="https://www.science.org/doi/10.1126/scitranslmed.aad3305">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE BMAL1 TEMPERATURE COMPENSATION CONTROL-27C:</b> MSFs kept in different temperatures and experimenting the knock-out of BMAL1.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Bmal1 Temperature Compensation</td> -->
        <td>Skin</td>
        <td>Control-27c</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134333">GSE134333</a></td>
        <td>Control, Temperature</td>
        <!-- <td>C57BL/6</td> -->
        <td>42, 24, 26, 20, 22, 46, 44, 28, 40, 0, 2, 4, 6, 8, 38, 10, 12, 14, 16, 18, 30, 36, 34, 32</td>
        <!-- <td>2</td> -->
        <!-- <td>Schematic representation of the experimental strategy used in temperature compensation analysis of the rhytmic transcriptome. MSFs(wild-type and Bmal1-/-) were synchronized by one DEX pulse and maintained at a constant temperature of 27, 32, or 37C for two complete circadian cycles and were subsequently sampled every 2 hours at these three different temperatures under constant conditions.</td> -->
        <td><a href="https://www.science.org/doi/10.1126/science.aaw7365">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE BMAL1 TEMPERATURE COMPENSATION CONTROL-32C:</b> MSFs kept in different temperatures and experimenting the knock-out of BMAL1.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Bmal1 Temperature Compensation</td> -->
        <td>Skin</td>
        <td>Control-32c</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134333">GSE134333</a></td>
        <td>Control, Temperature</td>
        <!-- <td>C57BL/6</td> -->
        <td>42, 24, 26, 20, 22, 46, 44, 28, 40, 0, 2, 4, 6, 8, 38, 10, 12, 14, 16, 18, 30, 36, 34, 32</td>
        <!-- <td>2</td> -->
        <!-- <td>Schematic representation of the experimental strategy used in temperature compensation analysis of the rhytmic transcriptome. MSFs(wild-type and Bmal1-/-) were synchronized by one DEX pulse and maintained at a constant temperature of 27, 32, or 37C for two complete circadian cycles and were subsequently sampled every 2 hours at these three different temperatures under constant conditions.</td> -->
        <td><a href="https://www.science.org/doi/10.1126/science.aaw7365">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE BMAL1 TEMPERATURE COMPENSATION CONTROL-37C:</b> MSFs kept in different temperatures and experimenting the knock-out of BMAL1.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Bmal1 Temperature Compensation</td> -->
        <td>Skin</td>
        <td>Control-37c</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134333">GSE134333</a></td>
        <td>Control, Temperature</td>
        <!-- <td>C57BL/6</td> -->
        <td>42, 24, 26, 20, 22, 46, 44, 28, 40, 0, 2, 4, 6, 8, 38, 10, 12, 14, 16, 18, 30, 36, 34, 32</td>
        <!-- <td>2</td> -->
        <!-- <td>Schematic representation of the experimental strategy used in temperature compensation analysis of the rhytmic transcriptome. MSFs(wild-type and Bmal1-/-) were synchronized by one DEX pulse and maintained at a constant temperature of 27, 32, or 37C for two complete circadian cycles and were subsequently sampled every 2 hours at these three different temperatures under constant conditions.</td> -->
        <td><a href="https://www.science.org/doi/10.1126/science.aaw7365">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE BMAL1 TEMPERATURE COMPENSATION KO-27C:</b> MSFs kept in different temperatures and experimenting the knock-out of BMAL1.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Bmal1 Temperature Compensation</td> -->
        <td>Skin</td>
        <td>KO-27c</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134333">GSE134333</a></td>
        <td>Knock-Out, Temperature</td>
        <!-- <td>C57BL/6</td> -->
        <td>42, 24, 26, 20, 22, 46, 44, 28, 40, 0, 2, 4, 6, 8, 38, 10, 12, 14, 16, 18, 30, 36, 34, 32</td>
        <!-- <td>2</td> -->
        <!-- <td>Schematic representation of the experimental strategy used in temperature compensation analysis of the rhytmic transcriptome. MSFs(wild-type and Bmal1-/-) were synchronized by one DEX pulse and maintained at a constant temperature of 27, 32, or 37C for two complete circadian cycles and were subsequently sampled every 2 hours at these three different temperatures under constant conditions.</td> -->
        <td><a href="https://www.science.org/doi/10.1126/science.aaw7365">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE BMAL1 TEMPERATURE COMPENSATION KO-32C:</b> MSFs kept in different temperatures and experimenting the knock-out of BMAL1.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Bmal1 Temperature Compensation</td> -->
        <td>Skin</td>
        <td>KO-32c</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134333">GSE134333</a></td>
        <td>Knock-Out, Temperature</td>
        <!-- <td>C57BL/6</td> -->
        <td>42, 24, 26, 20, 22, 46, 44, 28, 40, 0, 2, 4, 6, 8, 38, 10, 12, 14, 16, 18, 30, 36, 34, 32</td>
        <!-- <td>2</td> -->
        <!-- <td>Schematic representation of the experimental strategy used in temperature compensation analysis of the rhytmic transcriptome. MSFs(wild-type and Bmal1-/-) were synchronized by one DEX pulse and maintained at a constant temperature of 27, 32, or 37C for two complete circadian cycles and were subsequently sampled every 2 hours at these three different temperatures under constant conditions.</td> -->
        <td><a href="https://www.science.org/doi/10.1126/science.aaw7365">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE BMAL1 TEMPERATURE COMPENSATION KO-37C:</b> MSFs kept in different temperatures and experimenting the knock-out of BMAL1.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Bmal1 Temperature Compensation</td> -->
        <td>Skin</td>
        <td>KO-37c</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134333">GSE134333</a></td>
        <td>Knock-Out, Temperature</td>
        <!-- <td>C57BL/6</td> -->
        <td>42, 24, 26, 20, 22, 46, 44, 28, 40, 0, 2, 4, 6, 8, 38, 10, 12, 14, 16, 18, 30, 36, 34, 32</td>
        <!-- <td>2</td> -->
        <!-- <td>Schematic representation of the experimental strategy used in temperature compensation analysis of the rhytmic transcriptome. MSFs(wild-type and Bmal1-/-) were synchronized by one DEX pulse and maintained at a constant temperature of 27, 32, or 37C for two complete circadian cycles and were subsequently sampled every 2 hours at these three different temperatures under constant conditions.</td> -->
        <td><a href="https://www.science.org/doi/10.1126/science.aaw7365">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE BRAIN STEM ZHANG 2014 WT:</b> Wild-type C57/BL6 mouse tissue microarray in the Circadian Atlas project.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Brain Stem</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54652">GSE54652</a></td>
        <td>Control</td>
        <!-- <td>C57/BL6</td> -->
        <td>56, 42, 50, 60, 62, 64, 32, 24, 26, 20, 48, 46, 44, 28, 40, 52, 58, 38, 22, 54, 30, 36, 34, 18</td>
        <!-- <td>2</td> -->
        <!-- <td>6-wk-old male C57/BL6 mice entrained to a 12h:12h light:dark schedule for 1 week, then released into constant darkness.</td> -->
        <td><a href="http://www.pnas.org/content/111/45/16219.abstract">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE BREAST YANG 2017 WT:</b> Cellular mechano-environment regulates the mammary circadian clock.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Breast</td> -->
        <td>Breast</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>11, 39, 27, 15, 23, 19, 47, 31, 43, 35, 3, 7</td>
        <!-- <td>2</td> -->
        <!-- <td>Breast tissues were isolated at 4 hr intervals for two circadian (24 hourly) cycles, from mice kept under constant darkness to avoid any light- or dark-driven genes.</td> -->
        <td><a href="https://www.nature.com/articles/ncomms14287">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE BROWN FAT ANTERIOR DORSUM ZHANG 2014 WT:</b> Wild-type C57/BL6 mouse tissue microarray in the Circadian Atlas project.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Brown Fat Anterior Dorsum</td> -->
        <td>Adipose</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54652">GSE54652</a></td>
        <td>Control</td>
        <!-- <td>C57/BL6</td> -->
        <td>56, 42, 50, 60, 62, 64, 32, 24, 26, 20, 48, 46, 44, 28, 40, 52, 58, 38, 22, 54, 30, 36, 34, 18</td>
        <!-- <td>2</td> -->
        <!-- <td>6-wk-old male C57/BL6 mice entrained to a 12h:12h light:dark schedule for 1 week, then released into constant darkness.</td> -->
        <td><a href="http://www.pnas.org/content/111/45/16219.abstract">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE CEREBELLUM ZHANG 2014 WT:</b> Wild-type C57/BL6 mouse tissue microarray in the Circadian Atlas project.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Cerebellum</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54652">GSE54652</a></td>
        <td>Control</td>
        <!-- <td>C57/BL6</td> -->
        <td>56, 42, 50, 60, 62, 64, 32, 24, 26, 20, 48, 46, 44, 28, 40, 52, 58, 38, 22, 54, 30, 36, 34, 18</td>
        <!-- <td>2</td> -->
        <!-- <td>6-wk-old male C57/BL6 mice entrained to a 12h:12h light:dark schedule for 1 week, then released into constant darkness.</td> -->
        <td><a href="http://www.pnas.org/content/111/45/16219.abstract">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE CHOROID PLEXUS AND SCN KO:</b> Circadian regulation of cellular processes in mouse choroid plexus (ChP) and their dependence on signals from the clock in the suprachiasmatic nuclei (SCN).</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Choroid Plexus And Scn</td> -->
        <td>Brain</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243858">GSE243858</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 32, 44, 28, 36, 40, 0, 4, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>C57Bl6J mice were housed individually in a temperature-controlled facility at 21-25C with free access to food and water. The light/dark cycle was maintained with 12 h of light and 12 h of darkness (LD12:12); lights were turned on and off at 06:00. For KO mice tamoxifen was provided ad libitum through the diet (tamoxifen citrate 0.4g/kg; Envigo TD.130860) under 12:12 LD cycle (07:00 light-on and 19:00 light-off). The tamoxifen-induced flox recombination was tested in liver samples through qPCR</td> -->
        <td><a href="https://doi.org/10.1186/s12987-024-00547-3">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE CHOROID PLEXUS AND SCN WT:</b> Circadian regulation of cellular processes in mouse choroid plexus (ChP) and their dependence on signals from the clock in the suprachiasmatic nuclei (SCN).</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Choroid Plexus And Scn</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243858">GSE243858</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 32, 44, 28, 36, 40, 0, 4, 8</td>
        <!-- <td>2</td> -->
        <!-- <td>C57Bl6J mice were housed individually in a temperature-controlled facility at 21-25C with free access to food and water. The light/dark cycle was maintained with 12 h of light and 12 h of darkness (LD12:12); lights were turned on and off at 06:00. For KO mice tamoxifen was provided ad libitum through the diet (tamoxifen citrate 0.4g/kg; Envigo TD.130860) under 12:12 LD cycle (07:00 light-on and 19:00 light-off). The tamoxifen-induced flox recombination was tested in liver samples through qPCR</td> -->
        <td><a href="https://doi.org/10.1186/s12987-024-00547-3">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE CORTEX SNORD116 WT FEMALE:</b> Baseline circadian atlas of male vs female wild-type mice-reference arm for Snord116 imprinting study-showing sex-specific light x genotype gene-network effects.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Cortex Snord116 Wt</td> -->
        <td>Prefrontal cortex</td>
        <td>Female</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE297702">GSE297702</a></td>
        <td>Sex</td>
        <!-- <td></td> -->
        <td>12, 15, 21, 18, 0, 3, 6, 9</td>
        <!-- <td>7</td> -->
        <!-- <td></td> -->
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE297702">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE CORTEX SNORD116 WT MALE:</b> Baseline circadian atlas of male vs female wild-type mice-reference arm for Snord116 imprinting study-showing sex-specific light x genotype gene-network effects.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Cortex Snord116 Wt</td> -->
        <td>Prefrontal cortex</td>
        <td>Male</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE297702">GSE297702</a></td>
        <td>Sex</td>
        <!-- <td></td> -->
        <td>12, 15, 21, 18, 0, 3, 6, 9</td>
        <!-- <td>7</td> -->
        <!-- <td></td> -->
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE297702">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE CORTICAL COLLECTING DUCT ZUBER 2009 WT:</b> Circadian rhythms in the distal nephron segments, i.e., distal convoluted tubule (DCT) and connecting tubule (CNT) and the cortical collecting duct (CCD)</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Cortical Collecting Duct</td> -->
        <td>Kidney</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2752602/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE DIET MOISAN 2021 HF:</b> Mouse treated with different diet conditions</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Diet</td> -->
        <td>Hippocampus</td>
        <td>Hf</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>0, 12, 18, 6</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE DIET MOISAN 2021 HFR:</b> Mouse treated with different diet conditions</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Diet</td> -->
        <td>Hippocampus</td>
        <td>Hfr</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>0, 12, 18, 6</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE DIET MOISAN 2021 NC:</b> Mouse treated with different diet conditions</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Diet</td> -->
        <td>Hippocampus</td>
        <td>Nc</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>0, 12, 18, 6</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE DIET MOISAN 2021 NCR:</b> Mouse treated with different diet conditions</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Diet</td> -->
        <td>Hippocampus</td>
        <td>Ncr</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>0, 12, 18, 6</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE DISTAL COLON HOOGERWERF 2008 WT:</b> Microarray transcriptional profiling of mRNA expression in the mouse distal colon.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Distal Colon</td> -->
        <td>Digestive</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18848557">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE DISTAL CONVOLUTED TUBULE ZUBER 2009 WT:</b> Circadian rhythms in the distal nephron segments, i.e., distal convoluted tubule (DCT) and connecting tubule (CNT) and the cortical collecting duct (CCD)</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Distal Convoluted Tubule</td> -->
        <td>Kidney</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2752602/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE EPIDIDYMAL ADIPOSE ZHANG 2014 WT:</b> Wild-type C57/BL6 mouse tissue microarray in the Circadian Atlas project.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Epididymal Adipose</td> -->
        <td>Adipose</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54652">GSE54652</a></td>
        <td>Control</td>
        <!-- <td>C57/BL6</td> -->
        <td>56, 42, 50, 60, 62, 64, 32, 24, 26, 20, 48, 46, 44, 28, 40, 52, 58, 38, 22, 54, 30, 36, 34, 18</td>
        <!-- <td>2</td> -->
        <!-- <td>6-wk-old male C57/BL6 mice entrained to a 12h:12h light:dark schedule for 1 week, then released into constant darkness.</td> -->
        <td><a href="http://www.pnas.org/content/111/45/16219.abstract">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE FETAL KIDNEY DAN 2020 WT:</b> Examination of RNA-seq time series of developing fetal mouse kidnes from embryonic ages E18 to E20.5 (inclusive)</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Fetal Kidney</td> -->
        <td>Kidney</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 48, 16, 32, 44, 28, 36, 40, 0, 4, 8</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/32433970/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE FIBROBLAST GRECO 2018 AHCY-KO:</b> RNA-Seq Mouse Embryonic Fibroblast comparing Wild-Type to AHCY Knock-Out</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Fibroblast</td> -->
        <td>Cells</td>
        <td>Ahcy-KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 32, 28</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/33328229/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE FIBROBLAST GRECO 2018 WT:</b> RNA-Seq Mouse Embryonic Fibroblast comparing Wild-Type to AHCY Knock-Out</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Fibroblast</td> -->
        <td>Cells</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 32, 28</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/33328229/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEART BMAL1KO ESSER 2024 KO-FEMALE:</b> Cardiomyocyte-specific Bmal1 knockout vs wild-type in male and female mouse hearts (sampled over 24h). Female hearts have more rhythmic genes than males under normal conditions, and the loss of Bmal1 blunts or abolishes most sex differences in the cardiac circadian transcriptome. This indicates that Bmal1-driven clock output in heart mediates a large portion of the observed sex-specific gene expression patterns.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Heart Bmal1ko</td> -->
        <td>Heart</td>
        <td>KO-Female</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE262714">GSE262714</a></td>
        <td>Knock-Out, Sex</td>
        <!-- <td></td> -->
        <td>26, 38, 58, 22, 46, 18, 30, 42, 50, 34, 62, 54</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1093/function/zqae053">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEART BMAL1KO ESSER 2024 KO-MALE:</b> Cardiomyocyte-specific Bmal1 knockout vs wild-type in male and female mouse hearts (sampled over 24h). Female hearts have more rhythmic genes than males under normal conditions, and the loss of Bmal1 blunts or abolishes most sex differences in the cardiac circadian transcriptome. This indicates that Bmal1-driven clock output in heart mediates a large portion of the observed sex-specific gene expression patterns.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Heart Bmal1ko</td> -->
        <td>Heart</td>
        <td>KO-Male</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE262714">GSE262714</a></td>
        <td>Knock-Out, Sex</td>
        <!-- <td></td> -->
        <td>26, 38, 58, 22, 46, 18, 30, 42, 50, 34, 62, 54</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1093/function/zqae053">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEART BMAL1KO ESSER 2024 WT-FEMALE:</b> Cardiomyocyte-specific Bmal1 knockout vs wild-type in male and female mouse hearts (sampled over 24h). Female hearts have more rhythmic genes than males under normal conditions, and the loss of Bmal1 blunts or abolishes most sex differences in the cardiac circadian transcriptome. This indicates that Bmal1-driven clock output in heart mediates a large portion of the observed sex-specific gene expression patterns.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Heart Bmal1ko</td> -->
        <td>Heart</td>
        <td>Wild-Type-Female</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE262714">GSE262714</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>26, 38, 58, 22, 46, 18, 30, 42, 50, 34, 62, 54</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1093/function/zqae053">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEART BMAL1KO ESSER 2024 WT-MALE:</b> Cardiomyocyte-specific Bmal1 knockout vs wild-type in male and female mouse hearts (sampled over 24h). Female hearts have more rhythmic genes than males under normal conditions, and the loss of Bmal1 blunts or abolishes most sex differences in the cardiac circadian transcriptome. This indicates that Bmal1-driven clock output in heart mediates a large portion of the observed sex-specific gene expression patterns.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Heart Bmal1ko</td> -->
        <td>Heart</td>
        <td>Wild-Type-Male</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE262714">GSE262714</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>26, 38, 58, 22, 46, 18, 30, 42, 50, 34, 62, 54</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1093/function/zqae053">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEART GAUCHER 2022 HYPOXIC:</b> Normoxic and Hypoxic mice, experiments done in the liver, kidney, and heart</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Heart</td> -->
        <td>Heart</td>
        <td>Hypoxic</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Oxygen-Level</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEART GAUCHER 2022 NORMOXIC:</b> Normoxic and Hypoxic mice, experiments done in the liver, kidney, and heart</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Heart</td> -->
        <td>Heart</td>
        <td>Normoxic</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Oxygen-Level</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEART ZHANG 2014 WT:</b> Wild-type C57/BL6 mouse tissue microarray in the Circadian Atlas project.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Heart</td> -->
        <td>Heart</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54652">GSE54652</a></td>
        <td>Control</td>
        <!-- <td>C57/BL6</td> -->
        <td>56, 42, 50, 60, 62, 64, 32, 24, 26, 20, 48, 46, 44, 28, 40, 52, 58, 38, 22, 54, 30, 36, 34, 18</td>
        <!-- <td>2</td> -->
        <!-- <td>6-wk-old male C57/BL6 mice entrained to a 12h:12h light:dark schedule for 1 week, then released into constant darkness.</td> -->
        <td><a href="http://www.pnas.org/content/111/45/16219.abstract">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEPACTIC ALBCRE HYPOXIA:</b> Transcriptional response of HIF1a, and Bmal1 Knock-Out in hypoxia and normoxia</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Hepactic Albcre</td> -->
        <td>Liver</td>
        <td>Hypoxia</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254438">GSE254438</a></td>
        <td>Knock-Out, Oxygen-Level</td>
        <!-- <td></td> -->
        <td>4, 16</td>
        <!-- <td>4</td> -->
        <!-- <td>RNA-sequencing on livers of AlbCRE, HLKO or BLKO mice exposed to 4 hour normoxia (21% oxygen) or hypoxia (6% hypoxia) at CT4-8 or CT16-20. Lights were turned off at the transition of dark to light regiment on the day of the experiment, following regular LD regimen.</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.07.003">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEPACTIC ALBCRE NORMOXIA:</b> Transcriptional response of HIF1a, and Bmal1 Knock-Out in hypoxia and normoxia</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Hepactic Albcre</td> -->
        <td>Liver</td>
        <td>Normoxia</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254438">GSE254438</a></td>
        <td>Knock-Out, Oxygen-Level</td>
        <!-- <td></td> -->
        <td>4, 16</td>
        <!-- <td>4</td> -->
        <!-- <td>RNA-sequencing on livers of AlbCRE, HLKO or BLKO mice exposed to 4 hour normoxia (21% oxygen) or hypoxia (6% hypoxia) at CT4-8 or CT16-20. Lights were turned off at the transition of dark to light regiment on the day of the experiment, following regular LD regimen.</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.07.003">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEPACTIC BMAL1 HYPOXIA:</b> Transcriptional response of HIF1a, and Bmal1 Knock-Out in hypoxia and normoxia</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Hepactic Bmal1</td> -->
        <td>Liver</td>
        <td>Hypoxia</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254438">GSE254438</a></td>
        <td>Knock-Out, Oxygen-Level</td>
        <!-- <td></td> -->
        <td>4, 16</td>
        <!-- <td>4</td> -->
        <!-- <td>RNA-sequencing on livers of AlbCRE, HLKO or BLKO mice exposed to 4 hour normoxia (21% oxygen) or hypoxia (6% hypoxia) at CT4-8 or CT16-20. Lights were turned off at the transition of dark to light regiment on the day of the experiment, following regular LD regimen.</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.07.003">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEPACTIC BMAL1 NORMOXIA:</b> Transcriptional response of HIF1a, and Bmal1 Knock-Out in hypoxia and normoxia</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Hepactic Bmal1</td> -->
        <td>Liver</td>
        <td>Normoxia</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254438">GSE254438</a></td>
        <td>Knock-Out, Oxygen-Level</td>
        <!-- <td></td> -->
        <td>4, 16</td>
        <!-- <td>4</td> -->
        <!-- <td>RNA-sequencing on livers of AlbCRE, HLKO or BLKO mice exposed to 4 hour normoxia (21% oxygen) or hypoxia (6% hypoxia) at CT4-8 or CT16-20. Lights were turned off at the transition of dark to light regiment on the day of the experiment, following regular LD regimen.</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.07.003">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEPACTIC HIF1A HYPOXIA:</b> Transcriptional response of HIF1a, and Bmal1 Knock-Out in hypoxia and normoxia</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Hepactic Hif1a</td> -->
        <td>Liver</td>
        <td>Hypoxia</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254438">GSE254438</a></td>
        <td>Knock-Out, Oxygen-Level</td>
        <!-- <td></td> -->
        <td>4, 16</td>
        <!-- <td>4</td> -->
        <!-- <td>RNA-sequencing on livers of AlbCRE, HLKO or BLKO mice exposed to 4 hour normoxia (21% oxygen) or hypoxia (6% hypoxia) at CT4-8 or CT16-20. Lights were turned off at the transition of dark to light regiment on the day of the experiment, following regular LD regimen.</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.07.003">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEPACTIC HIF1A NORMOXIA:</b> Transcriptional response of HIF1a, and Bmal1 Knock-Out in hypoxia and normoxia</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Hepactic Hif1a</td> -->
        <td>Liver</td>
        <td>Normoxia</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254438">GSE254438</a></td>
        <td>Knock-Out, Oxygen-Level</td>
        <!-- <td></td> -->
        <td>4, 16</td>
        <!-- <td>4</td> -->
        <!-- <td>RNA-sequencing on livers of AlbCRE, HLKO or BLKO mice exposed to 4 hour normoxia (21% oxygen) or hypoxia (6% hypoxia) at CT4-8 or CT16-20. Lights were turned off at the transition of dark to light regiment on the day of the experiment, following regular LD regimen.</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.07.003">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEPATOCYTE AD LIB GUAN 2020 KO:</b> RNA sequencing in livers, isolated EC and KC of control and adult hepatocytes REV-ERB a/b double knockout cell, ATAC-seq in isolated EC and KC of control and adult hepatocytes REV-ERB a/b double knockout cell, single nuclei RNA-seq in control and adult hepatocytes REV-ERB a/b double knockout cell</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Hepatocyte Ad Lib</td> -->
        <td>Liver</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>10, 13, 22, 16, 19, 1, 4, 7</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/32732282/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEPATOCYTE AD LIB GUAN 2020 WT:</b> RNA sequencing in livers, isolated EC and KC of control and adult hepatocytes REV-ERB a/b double knockout cell, ATAC-seq in isolated EC and KC of control and adult hepatocytes REV-ERB a/b double knockout cell, single nuclei RNA-seq in control and adult hepatocytes REV-ERB a/b double knockout cell</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Hepatocyte Ad Lib</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>10, 13, 22, 16, 19, 1, 4, 7</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/32732282/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEPATOCYTE AND FEEDING KO AD-LIBITUM:</b> Control of diurnal rhythms by the REV-ERBa and REV-ERBb KO and feeding. Analysis of gene expressions in different liver cell types.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Hepatocyte And Feeding Ko</td> -->
        <td>Liver</td>
        <td>Ad-Libitum</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE143524">GSE143524</a></td>
        <td>Knock-Out, Diet</td>
        <!-- <td>C57BL/6J</td> -->
        <td>10, 13, 22, 16, 19, 1, 4, 7</td>
        <!-- <td>4</td> -->
        <!-- <td>Total RNA was extracted from liver tissue or isolated endothelial cells and kupffer cells using Trizol reagent followed by RNeasy Mini Kit (#74104).Libraries were prepared according to Illumina's instructions accompanying the TruSeq RNA Sample Preparation. The purified cDNA was captured on an Illumina flow cell for cluster generation. Libraries were sequenced on the Genome Analyzer following the manufacturer's protocols.</td> -->
        <td><a href="https://www.science.org/doi/10.1126/science.aba8984">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEPATOCYTE AND FEEDING KO REVERSE-PHASE-FEEDING:</b> Control of diurnal rhythms by the REV-ERBa and REV-ERBb KO and feeding. Analysis of gene expressions in different liver cell types.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Hepatocyte And Feeding Ko</td> -->
        <td>Liver</td>
        <td>Reverse-Phase-Feeding</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE143524">GSE143524</a></td>
        <td>Knock-Out, Diet</td>
        <!-- <td>C57BL/6J</td> -->
        <td>10, 13, 22, 16, 19, 1, 4, 7</td>
        <!-- <td>4</td> -->
        <!-- <td>Total RNA was extracted from liver tissue or isolated endothelial cells and kupffer cells using Trizol reagent followed by RNeasy Mini Kit (#74104).Libraries were prepared according to Illumina's instructions accompanying the TruSeq RNA Sample Preparation. The purified cDNA was captured on an Illumina flow cell for cluster generation. Libraries were sequenced on the Genome Analyzer following the manufacturer's protocols.</td> -->
        <td><a href="https://www.science.org/doi/10.1126/science.aba8984">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEPATOCYTE AND FEEDING WT AD-LIBITUM:</b> Control of diurnal rhythms by the REV-ERBa and REV-ERBb KO and feeding. Analysis of gene expressions in different liver cell types.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Hepatocyte And Feeding Wt</td> -->
        <td>Liver</td>
        <td>Ad-Libitum</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE143524">GSE143524</a></td>
        <td>Control, Diet</td>
        <!-- <td>C57BL/6J</td> -->
        <td>10, 13, 22, 16, 19, 1, 4, 7</td>
        <!-- <td>4</td> -->
        <!-- <td>Total RNA was extracted from liver tissue or isolated endothelial cells and kupffer cells using Trizol reagent followed by RNeasy Mini Kit (#74104).Libraries were prepared according to Illumina's instructions accompanying the TruSeq RNA Sample Preparation. The purified cDNA was captured on an Illumina flow cell for cluster generation. Libraries were sequenced on the Genome Analyzer following the manufacturer's protocols.</td> -->
        <td><a href="https://www.science.org/doi/10.1126/science.aba8984">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEPATOCYTE AND FEEDING WT REVERSE-PHASE-FEEDING:</b> Control of diurnal rhythms by the REV-ERBa and REV-ERBb KO and feeding. Analysis of gene expressions in different liver cell types.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Hepatocyte And Feeding Wt</td> -->
        <td>Liver</td>
        <td>Reverse-Phase-Feeding</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE143524">GSE143524</a></td>
        <td>Control, Diet</td>
        <!-- <td>C57BL/6J</td> -->
        <td>10, 13, 22, 16, 19, 1, 4, 7</td>
        <!-- <td>4</td> -->
        <!-- <td>Total RNA was extracted from liver tissue or isolated endothelial cells and kupffer cells using Trizol reagent followed by RNeasy Mini Kit (#74104).Libraries were prepared according to Illumina's instructions accompanying the TruSeq RNA Sample Preparation. The purified cDNA was captured on an Illumina flow cell for cluster generation. Libraries were sequenced on the Genome Analyzer following the manufacturer's protocols.</td> -->
        <td><a href="https://www.science.org/doi/10.1126/science.aba8984">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEPATOCYTE EC GUAN 2020 KO:</b> RNA sequencing in livers, isolated EC and KC of control and adult hepatocytes REV-ERB a/b double knockout cell, ATAC-seq in isolated EC and KC of control and adult hepatocytes REV-ERB a/b double knockout cell, single nuclei RNA-seq in control and adult hepatocytes REV-ERB a/b double knockout cell</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Hepatocyte Ec</td> -->
        <td>Liver</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>10, 4, 22, 16</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/32732282/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEPATOCYTE EC GUAN 2020 WT:</b> RNA sequencing in livers, isolated EC and KC of control and adult hepatocytes REV-ERB a/b double knockout cell, ATAC-seq in isolated EC and KC of control and adult hepatocytes REV-ERB a/b double knockout cell, single nuclei RNA-seq in control and adult hepatocytes REV-ERB a/b double knockout cell</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Hepatocyte Ec</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>10, 4, 22, 16</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/32732282/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEPATOCYTE IN ENDOTHELIAL CELLS KO:</b> Control of diurnal rhythms by the REV-ERBa and REV-ERBb Knock-Out and feeding. Analysis of gene expressions in different liver cell types.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Hepatocyte In Endothelial Cells</td> -->
        <td>Liver</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE143524">GSE143524</a></td>
        <td>Knock-Out</td>
        <!-- <td>C57BL/6J</td> -->
        <td>10, 4, 22, 16</td>
        <!-- <td>4</td> -->
        <!-- <td>Total RNA was extracted from liver tissue or isolated endothelial cells and kupffer cells using Trizol reagent followed by RNeasy Mini Kit (#74104).Libraries were prepared according to Illumina's instructions accompanying the TruSeq RNA Sample Preparation. The purified cDNA was captured on an Illumina flow cell for cluster generation. Libraries were sequenced on the Genome Analyzer following the manufacturer's protocols.</td> -->
        <td><a href="https://www.science.org/doi/10.1126/science.aba8984">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEPATOCYTE IN ENDOTHELIAL CELLS WT:</b> Control of diurnal rhythms by the REV-ERBa and REV-ERBb Knock-Out and feeding. Analysis of gene expressions in different liver cell types.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Hepatocyte In Endothelial Cells</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE143524">GSE143524</a></td>
        <td>Control</td>
        <!-- <td>C57BL/6J</td> -->
        <td>10, 4, 22, 16</td>
        <!-- <td>4</td> -->
        <!-- <td>Total RNA was extracted from liver tissue or isolated endothelial cells and kupffer cells using Trizol reagent followed by RNeasy Mini Kit (#74104).Libraries were prepared according to Illumina's instructions accompanying the TruSeq RNA Sample Preparation. The purified cDNA was captured on an Illumina flow cell for cluster generation. Libraries were sequenced on the Genome Analyzer following the manufacturer's protocols.</td> -->
        <td><a href="https://www.science.org/doi/10.1126/science.aba8984">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEPATOCYTE IN KUPFFER CELLS KO:</b> Control of diurnal rhythms by the REV-ERBa and REV-ERBb Knock-Out and feeding. Analysis of gene expressions in different liver cell types.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Hepatocyte In Kupffer Cells</td> -->
        <td>Liver</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE143524">GSE143524</a></td>
        <td>Knock-Out</td>
        <!-- <td>C57BL/6J</td> -->
        <td>10, 4, 22, 16</td>
        <!-- <td>4</td> -->
        <!-- <td>Total RNA was extracted from liver tissue or isolated endothelial cells and kupffer cells using Trizol reagent followed by RNeasy Mini Kit (#74104).Libraries were prepared according to Illumina's instructions accompanying the TruSeq RNA Sample Preparation. The purified cDNA was captured on an Illumina flow cell for cluster generation. Libraries were sequenced on the Genome Analyzer following the manufacturer's protocols.</td> -->
        <td><a href="https://www.science.org/doi/10.1126/science.aba8984">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEPATOCYTE IN KUPFFER CELLS WT:</b> Control of diurnal rhythms by the REV-ERBa and REV-ERBb Knock-Out and feeding. Analysis of gene expressions in different liver cell types.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Hepatocyte In Kupffer Cells</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE143524">GSE143524</a></td>
        <td>Control</td>
        <!-- <td>C57BL/6J</td> -->
        <td>10, 4, 22, 16</td>
        <!-- <td>4</td> -->
        <!-- <td>Total RNA was extracted from liver tissue or isolated endothelial cells and kupffer cells using Trizol reagent followed by RNeasy Mini Kit (#74104).Libraries were prepared according to Illumina's instructions accompanying the TruSeq RNA Sample Preparation. The purified cDNA was captured on an Illumina flow cell for cluster generation. Libraries were sequenced on the Genome Analyzer following the manufacturer's protocols.</td> -->
        <td><a href="https://www.science.org/doi/10.1126/science.aba8984">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEPATOCYTE KC GUAN 2020 KO:</b> RNA sequencing in livers, isolated EC and KC of control and adult hepatocytes REV-ERB a/b double knockout cell, ATAC-seq in isolated EC and KC of control and adult hepatocytes REV-ERB a/b double knockout cell, single nuclei RNA-seq in control and adult hepatocytes REV-ERB a/b double knockout cell</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Hepatocyte Kc</td> -->
        <td>Liver</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>10, 4, 22, 16</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/32732282/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEPATOCYTE KC GUAN 2020 WT:</b> RNA sequencing in livers, isolated EC and KC of control and adult hepatocytes REV-ERB a/b double knockout cell, ATAC-seq in isolated EC and KC of control and adult hepatocytes REV-ERB a/b double knockout cell, single nuclei RNA-seq in control and adult hepatocytes REV-ERB a/b double knockout cell</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Hepatocyte Kc</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>10, 4, 22, 16</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/32732282/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEPATOCYTE RPF GUAN 2020 KO:</b> RNA sequencing in livers, isolated EC and KC of control and adult hepatocytes REV-ERB a/b double knockout cell, ATAC-seq in isolated EC and KC of control and adult hepatocytes REV-ERB a/b double knockout cell, single nuclei RNA-seq in control and adult hepatocytes REV-ERB a/b double knockout cell</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Hepatocyte Rpf</td> -->
        <td>Liver</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>10, 13, 22, 16, 19, 1, 4, 7</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/32732282/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEPATOCYTE RPF GUAN 2020 WT:</b> RNA sequencing in livers, isolated EC and KC of control and adult hepatocytes REV-ERB a/b double knockout cell, ATAC-seq in isolated EC and KC of control and adult hepatocytes REV-ERB a/b double knockout cell, single nuclei RNA-seq in control and adult hepatocytes REV-ERB a/b double knockout cell</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Hepatocyte Rpf</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>10, 13, 22, 16, 19, 1, 4, 7</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/32732282/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HIGH FAT DIET KIDNEY 2021 HFK:</b> RNASeq samples of two groups (normal chow vs HFD), three replicates, 6 time points for RNA-seq in GHTF</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>High Fat Diet</td> -->
        <td>Kidney</td>
        <td>Hfk</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HIGH FAT DIET KIDNEY 2021 NCK:</b> RNASeq samples of two groups (normal chow vs HFD), three replicates, 6 time points for RNA-seq in GHTF</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>High Fat Diet</td> -->
        <td>Kidney</td>
        <td>Nck</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HYPOTHALAMUS ZHANG 2014 WT:</b> Wild-type C57/BL6 mouse tissue microarray in the Circadian Atlas project.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Hypothalamus</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54652">GSE54652</a></td>
        <td>Control</td>
        <!-- <td>C57/BL6</td> -->
        <td>56, 42, 50, 60, 62, 64, 32, 24, 26, 20, 48, 46, 44, 28, 40, 52, 58, 38, 22, 54, 30, 36, 34, 18</td>
        <!-- <td>2</td> -->
        <!-- <td>6-wk-old male C57/BL6 mice entrained to a 12h:12h light:dark schedule for 1 week, then released into constant darkness.</td> -->
        <td><a href="http://www.pnas.org/content/111/45/16219.abstract">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE INTESTINAL EPITHELIA TOGNINI 2017 KETOGENIC-DIET:</b> Intestinal epithelia microarray in a ketogenic diet vs. normal chow experiment.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Intestinal Epithelia</td> -->
        <td>Digestive</td>
        <td>Ketogenic-Diet</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Diet</td>
        <!-- <td>8-week-old C57BL/6</td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td>Submitted 8-week-old C57BL/6 mice to KD dietary regimen during 4 weeks and compared them to normal chow-fed mice.</td> -->
        <td><a href="http://www.cell.com/cell-metabolism/fulltext/S1550-4131(17)30500-4">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE INTESTINAL EPITHELIA TOGNINI 2017 NORMAL-CHOW:</b> Intestinal epithelia microarray in a ketogenic diet vs. normal chow experiment.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Intestinal Epithelia</td> -->
        <td>Digestive</td>
        <td>Normal-Chow</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td>8-week-old C57BL/6</td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td>Submitted 8-week-old C57BL/6 mice to KD dietary regimen during 4 weeks and compared them to normal chow-fed mice.</td> -->
        <td><a href="http://www.cell.com/cell-metabolism/fulltext/S1550-4131(17)30500-4">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE KIDNEY GAUCHER 2022 HYPOXIC:</b> Normoxic and Hypoxic mice, experiments done in the liver, kidney, and heart</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Kidney</td> -->
        <td>Kidney</td>
        <td>Hypoxic</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Oxygen-Level</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE KIDNEY GAUCHER 2022 NORMOXIC:</b> Normoxic and Hypoxic mice, experiments done in the liver, kidney, and heart</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Kidney</td> -->
        <td>Kidney</td>
        <td>Normoxic</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Oxygen-Level</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE KIDNEY ZHANG 2014 WT:</b> Wild-type C57/BL6 mouse tissue microarray in the Circadian Atlas project.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Kidney</td> -->
        <td>Kidney</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54652">GSE54652</a></td>
        <td>Control</td>
        <!-- <td>C57/BL6</td> -->
        <td>56, 42, 50, 60, 62, 64, 32, 24, 26, 20, 48, 46, 44, 28, 40, 52, 58, 38, 22, 54, 30, 36, 34, 18</td>
        <!-- <td>2</td> -->
        <!-- <td>6-wk-old male C57/BL6 mice entrained to a 12h:12h light:dark schedule for 1 week, then released into constant darkness.</td> -->
        <td><a href="http://www.pnas.org/content/111/45/16219.abstract">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LAZAR ARCUATE 2024 CONTROL:</b> Circadian gene expression in the hypothalamic arcuate nucleus of control vs liver-clock disrupted (Rev-Erba/beta double knockout in hepatocytes, HepDKO) mice. The loss of the liver clock sends altered signals via vagal afferents, leading to reprogramming of rhythmic transcripts in the arcuate nucleus that regulate feeding behavior and metabolism.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Lazar</td> -->
        <td>Brain</td>
        <td>Control</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE248462">GSE248462</a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>10, 4, 22, 16</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1126/science.adn2786">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LAZAR ARCUATE 2024 HEPDKO:</b> Circadian gene expression in the hypothalamic arcuate nucleus of control vs liver-clock disrupted (Rev-Erba/beta double knockout in hepatocytes, HepDKO) mice. The loss of the liver clock sends altered signals via vagal afferents, leading to reprogramming of rhythmic transcripts in the arcuate nucleus that regulate feeding behavior and metabolism.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Lazar</td> -->
        <td>Brain</td>
        <td>Hepdko</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE248462">GSE248462</a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>10, 4, 22, 16</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1126/science.adn2786">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LAZAR NODOSE 2024 CONTROL:</b> Circadian transcriptomic analysis of the vagal nodose ganglion in control vs HepDKO mice. Eliminating the liver's clock (Rev-Erba/beta in hepatocytes) alters rhythmic gene expression in vagal sensory neurons, suggesting that liver clock disruption feeds back to the peripheral nervous system and modulates neuroimmune and metabolic signaling rhythms.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Lazar</td> -->
        <td>Nervous</td>
        <td>Control</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE248462">GSE248462</a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>10, 4, 22, 16</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1126/science.adn2786">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LAZAR NODOSE 2024 HEPDKO:</b> Circadian transcriptomic analysis of the vagal nodose ganglion in control vs HepDKO mice. Eliminating the liver's clock (Rev-Erba/beta in hepatocytes) alters rhythmic gene expression in vagal sensory neurons, suggesting that liver clock disruption feeds back to the peripheral nervous system and modulates neuroimmune and metabolic signaling rhythms.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Lazar</td> -->
        <td>Nervous</td>
        <td>Hepdko</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE248462">GSE248462</a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>10, 4, 22, 16</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1126/science.adn2786">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER CLOCK KNOCKOUT AD LIBITUM PAOLO 2019 KO:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Clock Knockout Ad Libitum</td> -->
        <td>Liver</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER CLOCK KNOCKOUT AD LIBITUM PAOLO 2019 RE:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Clock Knockout Ad Libitum</td> -->
        <td>Liver</td>
        <td>Re</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER CLOCK KNOCKOUT AD LIBITUM PAOLO 2019 WT:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Clock Knockout Ad Libitum</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER CLOCK KNOCKOUT TRF PAOLO 2019 KO:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Clock Knockout Trf</td> -->
        <td>Liver</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER CLOCK KNOCKOUT TRF PAOLO 2019 RE:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Clock Knockout Trf</td> -->
        <td>Liver</td>
        <td>Re</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER CLOCK KNOCKOUT TRF PAOLO 2019 WT:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Clock Knockout Trf</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER ECKEL MAHAN 2022 KO:</b> Inducible insulin receptor knockout mice to look at hepatic gene expression around the circadian clock</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Eckel</td> -->
        <td>Liver</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 0, 2, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER ECKEL MAHAN 2022 WT:</b> Inducible insulin receptor knockout mice to look at hepatic gene expression around the circadian clock</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Eckel</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 0, 2, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER ECKEL-MAHAN 2013 HIGH-FAT:</b> Investigating widespread remodeling of the liver clock generated by high-fat diet.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>High-Fat</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE52333">GSE52333</a></td>
        <td>Diet</td>
        <!-- <td>8-week old, male C57BL/6J</td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td>Animals consuming normal chow or high fat diet (60 percent kcal from fat) for ten weeks were analyzed for circadian gene expression. Livers were harvested from animals every four hours throughout the circadian cycle.</td> -->
        <td><a href="http://www.cell.com/abstract/S0092-8674(13)01485-2">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER ECKEL-MAHAN 2013 NORMAL-CHOW:</b> Investigating widespread remodeling of the liver clock generated by high-fat diet.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Normal-Chow</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE52333">GSE52333</a></td>
        <td>Control</td>
        <!-- <td>8-week old, male C57BL/6J</td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td>Animals consuming normal chow or high fat diet (60 percent kcal from fat) for ten weeks were analyzed for circadian gene expression. Livers were harvested from animals every four hours throughout the circadian cycle.</td> -->
        <td><a href="http://www.cell.com/abstract/S0092-8674(13)01485-2">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER GAUCHER 2018 CHRONIC-ETOH:</b> Liver RNASeq chronically treated with ethanol</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Chronic-Etoh</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Drug Treatment</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER GAUCHER 2018 WT:</b> Liver RNASeq chronically treated with ethanol</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER GAUCHER 2022 HYPOXIC:</b> Normoxic and Hypoxic mice, experiments done in the liver, kidney, and heart</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Hypoxic</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Oxygen-Level</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER GAUCHER 2022 NORMOXIC:</b> Normoxic and Hypoxic mice, experiments done in the liver, kidney, and heart</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Normoxic</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Oxygen-Level</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER HUGHES 2009 WT:</b> Circadian microarray of mouse liver over a 48 hour time course</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>42, 48, 43, 24, 25, 26, 27, 20, 21, 22, 23, 46, 47, 44, 45, 28, 29, 40, 41, 1, 3, 2, 5, 4, 7, 6, 9, 8, 39, 38, 11, 10, 13, 12, 15, 14, 17, 16, 19, 18, 31, 30, 37, 36, 35, 34, 33, 32</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19343201">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER KINOUCHI 2018 FASTING:</b> Liver RNASeq in a fasting condition</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Fasting</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>7</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER KINOUCHI 2018 NORMAL-CHOW:</b> Liver RNASeq in a fasting condition</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Normal-Chow</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>7</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER MASRI 2014 SIRT1-KO:</b> Genomic partitioning by two independent sirtuins contributes to differential control of circadian metabolism.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Sirt1-KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE57830">GSE57830</a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td>Mice with liver-specific ablation of either Sirt1 or Sirt6 genes and their corresponding wild-type (WT) littermates.</td> -->
        <td><a href="http://www.cell.com/cell/fulltext/S0092-8674(14)00884-8">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER MASRI 2014 SIRT1-WT:</b> Genomic partitioning by two independent sirtuins contributes to differential control of circadian metabolism.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Sirt1-Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE57830">GSE57830</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td>Mice with liver-specific ablation of either Sirt1 or Sirt6 genes and their corresponding wild-type (WT) littermates.</td> -->
        <td><a href="http://www.cell.com/cell/fulltext/S0092-8674(14)00884-8">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER MASRI 2014 SIRT6-KO:</b> Genomic partitioning by two independent sirtuins contributes to differential control of circadian metabolism.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Sirt6-KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54652">GSE54652</a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td>Mice with liver-specific ablation of either Sirt1 or Sirt6 genes and their corresponding wild-type (WT) littermates.</td> -->
        <td><a href="http://www.cell.com/cell/fulltext/S0092-8674(14)00884-8">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER MASRI 2014 SIRT6-WT:</b> Genomic partitioning by two independent sirtuins contributes to differential control of circadian metabolism.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Sirt6-Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54652">GSE54652</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td>Mice with liver-specific ablation of either Sirt1 or Sirt6 genes and their corresponding wild-type (WT) littermates.</td> -->
        <td><a href="http://www.cell.com/cell/fulltext/S0092-8674(14)00884-8">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER MASRI 2016 LUNG-WT:</b> Lung adenocarcinoma operates as an endogenous reorganizer of circadian metabolism.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Lung-Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td>KrasLSL, C57BL/6J</td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td>WT and TB mice were sacrificed every four hours over the circadian cycle and livers were subjected to transcriptomics and metabolomics analyses.</td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER MASRI 2016 TUMOR-BEARING-LUNG:</b> Lung adenocarcinoma operates as an endogenous reorganizer of circadian metabolism.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Tumor-Bearing-Lung</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Cancer</td>
        <!-- <td>KrasLSL, C57BL/6J</td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td>WT and TB mice were sacrificed every four hours over the circadian cycle and livers were subjected to transcriptomics and metabolomics analyses.</td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER MILLER 2007 WT:</b> Mouse Wild Type Liver Transcriptome</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>26, 38, 58, 22, 46, 18, 30, 42, 50, 34, 62, 54</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="http://www.pnas.org/content/104/9/3342.short">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER MUSCLE DUB RE LIVER LMRE:</b> Liver muscle double KO and RE of genes</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Muscle Dub Re Liver</td> -->
        <td>Liver</td>
        <td>Lmre</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER MUSCLE DUB RE LIVER WT DUB:</b> Liver muscle double KO and RE of genes</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Muscle Dub Re Liver Wt</td> -->
        <td>Liver</td>
        <td>Dub</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER MUSCLE DUB RE MUS LMRE:</b> Liver muscle double KO and RE of genes</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Muscle Dub Re Mus</td> -->
        <td>Muscle</td>
        <td>Lmre</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER MUSCLE DUB RE MUS WT DUB:</b> Liver muscle double KO and RE of genes</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Muscle Dub Re Mus Wt</td> -->
        <td>Muscle</td>
        <td>Dub</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER NAN 2018 LIBRAMED-TREATMENT:</b> Liver RNASeq in an experiment treated with the drug Libramed</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Libramed-Treatment</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Drug Treatment</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER NAN 2018 WT:</b> Liver RNASeq in an experiment treated with the drug Libramed</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER PANDA 2002 WT:</b> Gene expression profiling to identify cycling transcripts in the SCN and in the liver</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>38, 58, 46, 54, 30, 42, 50, 34, 62, 74, 66, 70</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/12015981">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER PETRUS 2020 CONTROL-DARK:</b> Mouse treated with different diet conditions and different feeding conditions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Control-Dark</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER PETRUS 2020 CONTROL-LIGHT:</b> Mouse treated with different diet conditions and different feeding conditions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Control-Light</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER PETRUS 2020 TRYPTOPHAN-DARK:</b> Mouse treated with different diet conditions and different feeding conditions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Tryptophan-Dark</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER PETRUS 2020 TRYPTOPHAN-LIGHT:</b> Mouse treated with different diet conditions and different feeding conditions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Tryptophan-Light</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER SASSONE ILLUMINA KO:</b> Liver RNASeq with KO of all clock genes, and RE of just Liver Clock</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Sassone Illumina</td> -->
        <td>Liver</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>NA</td>
        <!-- <td>0</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER SASSONE ILLUMINA RE:</b> Liver RNASeq with KO of all clock genes, and RE of just Liver Clock</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Sassone Illumina</td> -->
        <td>Liver</td>
        <td>Re</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>NA</td>
        <!-- <td>0</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER SCN RE KO:</b> RNASeq of liver in WT, Bmal1 KO and SCN-RE mice</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Scn Re</td> -->
        <td>Liver</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER SCN RE RE:</b> RNASeq of liver in WT, Bmal1 KO and SCN-RE mice</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Scn Re</td> -->
        <td>Liver</td>
        <td>Re</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER SCN RE WT:</b> RNASeq of liver in WT, Bmal1 KO and SCN-RE mice</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Scn Re</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER TERAJIMA 2017 ADARB1-KO:</b> Identification A-to-I RNA editing as a key mechanism of post-transcriptional regulation in the circadian clockwork.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Adarb1-KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://www.nature.com/articles/ng.3731">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER TERAJIMA 2017 WT:</b> Identification A-to-I RNA editing as a key mechanism of post-transcriptional regulation in the circadian clockwork.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://www.nature.com/articles/ng.3731">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER TOGNINI 2017 KETOGENIC-DIET:</b> Liver microarray in a ketogenic diet vs. normal chow experiment.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Ketogenic-Diet</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Diet</td>
        <!-- <td>8-week-old C57BL/6</td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td>Submitted 8-week-old C57BL/6 mice to KD dietary regimen during 4 weeks and compared them to normal chow-fed mice.</td> -->
        <td><a href="http://www.cell.com/cell-metabolism/fulltext/S1550-4131(17)30500-4">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER TOGNINI 2017 NORMAL-CHOW:</b> Liver microarray in a ketogenic diet vs. normal chow experiment.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Normal-Chow</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td>8-week-old C57BL/6</td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td>Submitted 8-week-old C57BL/6 mice to KD dietary regimen during 4 weeks and compared them to normal chow-fed mice.</td> -->
        <td><a href="http://www.cell.com/cell-metabolism/fulltext/S1550-4131(17)30500-4">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER ZHANG 2014 WT:</b> Wild-type C57/BL6 mouse tissue microarray in the Circadian Atlas project.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54652">GSE54652</a></td>
        <td>Control</td>
        <!-- <td>C57/BL6</td> -->
        <td>56, 42, 50, 60, 62, 64, 32, 24, 26, 20, 48, 46, 44, 28, 40, 52, 58, 38, 22, 54, 30, 36, 34, 18</td>
        <!-- <td>2</td> -->
        <!-- <td>6-wk-old male C57/BL6 mice entrained to a 12h:12h light:dark schedule for 1 week, then released into constant darkness.</td> -->
        <td><a href="http://www.pnas.org/content/111/45/16219.abstract">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LUNG WOLFF 2023 OLD:</b> Circadian gene expression comparison in lungs of young adult vs old mice, highlighting age-related attenuation and alteration of rhythmic transcripts. Older mice show fewer and dampened circadian oscillations in lung gene expression, implicating aging as a modifier of the circadian transcriptome.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Lung</td> -->
        <td>Lung</td>
        <td>Old</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE261234">GSE261234</a></td>
        <td>Age</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2022.111982">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LUNG WOLFF 2023 YOUNG:</b> Circadian gene expression comparison in lungs of young adult vs old mice, highlighting age-related attenuation and alteration of rhythmic transcripts. Older mice show fewer and dampened circadian oscillations in lung gene expression, implicating aging as a modifier of the circadian transcriptome.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Lung</td> -->
        <td>Lung</td>
        <td>Young</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE261234">GSE261234</a></td>
        <td>Age</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2022.111982">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LUNG ZHANG 2014 WT:</b> Wild-type C57/BL6 mouse tissue microarray in the Circadian Atlas project.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Lung</td> -->
        <td>Lung</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54652">GSE54652</a></td>
        <td>Control</td>
        <!-- <td>C57/BL6</td> -->
        <td>56, 42, 50, 60, 62, 64, 32, 24, 26, 20, 48, 46, 44, 28, 40, 52, 58, 38, 22, 54, 30, 36, 34, 18</td>
        <!-- <td>2</td> -->
        <!-- <td>6-wk-old male C57/BL6 mice entrained to a 12h:12h light:dark schedule for 1 week, then released into constant darkness.</td> -->
        <td><a href="http://www.pnas.org/content/111/45/16219.abstract">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE MACROPHAGES KELLER 2009 DARK:</b> Macrophages in all dark.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Macrophages</td> -->
        <td>Cells</td>
        <td>Dark</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 32, 44, 28, 36, 40, 0, 4, 8</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19955445">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE MEF GRECO 2018 DMSO:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Mef</td> -->
        <td>Cells</td>
        <td>Dmso</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 12</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/33328229/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE MEF GRECO 2018 DZNEP:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Mef</td> -->
        <td>Cells</td>
        <td>Dznep</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 12</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/33328229/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE MPFC MCCLUNG 2025 FEMALE:</b> Sex-specific circadian transcriptomic rhythms in the mouse medial prefrontal cortex (mPFC). Approximately 12% of mPFC transcripts show daily oscillations, and the timing of core clock gene expression differs between males and females. Female mPFCs have slightly higher overlap with human PFC rhythmic genes and show some unique phase distributions compared to males, underscoring sex as a factor in cortical circadian regulation.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Mpfc</td> -->
        <td>Brain</td>
        <td>Female</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE284053">GSE284053</a></td>
        <td>Sex</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.3389/fnins.2024.1524615">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE MPFC MCCLUNG 2025 MALE:</b> Sex-specific circadian transcriptomic rhythms in the mouse medial prefrontal cortex (mPFC). Approximately 12% of mPFC transcripts show daily oscillations, and the timing of core clock gene expression differs between males and females. Female mPFCs have slightly higher overlap with human PFC rhythmic genes and show some unique phase distributions compared to males, underscoring sex as a factor in cortical circadian regulation.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Mpfc</td> -->
        <td>Brain</td>
        <td>Male</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE284053">GSE284053</a></td>
        <td>Sex</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.3389/fnins.2024.1524615">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE MUSCLE ANDREWS 2010 WT:</b> Skeletal muscle with Bmal1(-/-) condition</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Muscle</td> -->
        <td>Muscle</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>26, 38, 58, 22, 46, 18, 30, 42, 50, 34, 62, 54</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="http://www.pnas.org/content/107/44/19090.short">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE MUSCLE GASTROCNEMIUS ZHANG 2014 WT:</b> Wild-type C57/BL6 mouse tissue microarray in the Circadian Atlas project.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Muscle Gastrocnemius</td> -->
        <td>Muscle</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54652">GSE54652</a></td>
        <td>Control</td>
        <!-- <td>C57/BL6</td> -->
        <td>56, 42, 50, 60, 62, 64, 32, 24, 26, 20, 48, 46, 44, 28, 40, 52, 58, 38, 22, 54, 30, 36, 34, 18</td>
        <!-- <td>2</td> -->
        <!-- <td>6-wk-old male C57/BL6 mice entrained to a 12h:12h light:dark schedule for 1 week, then released into constant darkness.</td> -->
        <td><a href="http://www.pnas.org/content/111/45/16219.abstract">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE MUSCLE KINOUCHI 2018 FASTING:</b> Muscle RNASeq in a fasting condition</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Muscle</td> -->
        <td>Muscle</td>
        <td>Fasting</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>7</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE MUSCLE KINOUCHI 2018 NORMAL-CHOW:</b> Muscle RNASeq in a fasting condition</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Muscle</td> -->
        <td>Muscle</td>
        <td>Normal-Chow</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>7</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE NEUTROPHILDEPLETION BONEMARROW DEPLETED:</b> Circadian transcriptional profiles of bone marrow under control vs neutrophil-depleted conditions, demonstrating that neutrophil loss disrupts normal rhythmic gene expression in extracellular matrix pathways.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Neutrophildepletion Bonemarrow</td> -->
        <td>Bone Marrow</td>
        <td>Depleted</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE198770">GSE198770</a></td>
        <td>Immune</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1038/s41586-025-08741-5">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE NEUTROPHILDEPLETION BONEMARROW WT:</b> Circadian transcriptional profiles of bone marrow under control vs neutrophil-depleted conditions, demonstrating that neutrophil loss disrupts normal rhythmic gene expression in extracellular matrix pathways.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Neutrophildepletion Bonemarrow</td> -->
        <td>Bone Marrow</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE198770">GSE198770</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1038/s41586-025-08741-5">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE NEUTROPHILDEPLETION INTESTINE DEPLETED:</b> Circadian gene expression in the small intestine under neutrophil-depleted vs control conditions, showing that loss of neutrophils alters rhythmic matrix-associated transcripts and circadian patterns of gene expression.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Neutrophildepletion Intestine</td> -->
        <td>Intestine</td>
        <td>Depleted</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE198770">GSE198770</a></td>
        <td>Immune</td>
        <!-- <td></td> -->
        <td>9, 13, 5, 21, 17</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1038/s41586-025-08741-5">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE NEUTROPHILDEPLETION INTESTINE WT:</b> Circadian gene expression in the small intestine under neutrophil-depleted vs control conditions, showing that loss of neutrophils alters rhythmic matrix-associated transcripts and circadian patterns of gene expression.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Neutrophildepletion Intestine</td> -->
        <td>Intestine</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE198770">GSE198770</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>9, 13, 5, 21, 17</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1038/s41586-025-08741-5">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE NEUTROPHILDEPLETION LIVER DEPLETED:</b> Circadian transcriptome of liver tissue in control vs neutrophil-depleted mice, indicating that neutrophil elimination disrupts normal circadian expression patterns of liver genes, particularly those linked to extracellular matrix organization.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Neutrophildepletion Liver</td> -->
        <td>Liver</td>
        <td>Depleted</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE198770">GSE198770</a></td>
        <td>Immune</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1038/s41586-025-08741-5">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE NEUTROPHILDEPLETION LIVER WT:</b> Circadian transcriptome of liver tissue in control vs neutrophil-depleted mice, indicating that neutrophil elimination disrupts normal circadian expression patterns of liver genes, particularly those linked to extracellular matrix organization.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Neutrophildepletion Liver</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE198770">GSE198770</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1038/s41586-025-08741-5">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE NEUTROPHILDEPLETION LUNG DEPLETED:</b> Circadian gene expression in lung tissue with and without neutrophil depletion, illustrating that removal of neutrophils alters the temporal expression of matrix-related genes and perturbs lung circadian rhythms.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Neutrophildepletion Lung</td> -->
        <td>Lung</td>
        <td>Depleted</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE198770">GSE198770</a></td>
        <td>Immune</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1038/s41586-025-08741-5">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE NEUTROPHILDEPLETION LUNG WT:</b> Circadian gene expression in lung tissue with and without neutrophil depletion, illustrating that removal of neutrophils alters the temporal expression of matrix-related genes and perturbs lung circadian rhythms.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Neutrophildepletion Lung</td> -->
        <td>Lung</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE198770">GSE198770</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>13, 21, 17, 1, 5, 9</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1038/s41586-025-08741-5">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE NEUTROPHILDEPLETION SKIN BMAL1-KO:</b> Circadian transcriptional profile of skin under control, neutrophil-depleted, and neutrophil-specific Bmal1 knockout conditions, showing that both neutropenia and neutrophil clock disruption alter normal circadian gene expression in skin tissues.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Neutrophildepletion Skin</td> -->
        <td>Skin</td>
        <td>Bmal1-KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE198770">GSE198770</a></td>
        <td>Immune, Knock-Out</td>
        <!-- <td></td> -->
        <td>24, 10, 13, 12, 15, 21, 22, 16, 19, 18, 1, 3, 5, 4, 7, 9</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1038/s41586-025-08741-5">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE NEUTROPHILDEPLETION SKIN DEPLETED:</b> Circadian transcriptional profile of skin under control, neutrophil-depleted, and neutrophil-specific Bmal1 knockout conditions, showing that both neutropenia and neutrophil clock disruption alter normal circadian gene expression in skin tissues.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Neutrophildepletion Skin</td> -->
        <td>Skin</td>
        <td>Depleted</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE198770">GSE198770</a></td>
        <td>Immune, Knock-Out</td>
        <!-- <td></td> -->
        <td>24, 10, 13, 12, 15, 21, 22, 16, 19, 18, 1, 3, 5, 4, 7, 9</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1038/s41586-025-08741-5">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE NEUTROPHILDEPLETION SKIN WT:</b> Circadian transcriptional profile of skin under control, neutrophil-depleted, and neutrophil-specific Bmal1 knockout conditions, showing that both neutropenia and neutrophil clock disruption alter normal circadian gene expression in skin tissues.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Neutrophildepletion Skin</td> -->
        <td>Skin</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE198770">GSE198770</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 10, 13, 12, 15, 21, 22, 16, 19, 18, 1, 3, 5, 4, 7, 9</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1038/s41586-025-08741-5">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE NIGHT FEEDING KO:</b> RNASeq of liver in WT and Bmal1 KO mice under night feeding condition</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Night Feeding</td> -->
        <td>Liver</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE NIGHT FEEDING WT:</b> RNASeq of liver in WT and Bmal1 KO mice under night feeding condition</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Night Feeding</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE PITUITARY ZHANG 2014 WT:</b> Wild-type C57/BL6 mouse tissue microarray in the Circadian Atlas project.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Pituitary</td> -->
        <td>Glands</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54652">GSE54652</a></td>
        <td>Control</td>
        <!-- <td>C57/BL6</td> -->
        <td>42, 48, 43, 24, 25, 26, 27, 20, 21, 22, 23, 46, 47, 44, 45, 28, 29, 40, 41, 1, 3, 2, 5, 4, 7, 6, 9, 8, 39, 38, 11, 10, 13, 12, 15, 14, 17, 16, 19, 18, 31, 30, 37, 36, 35, 34, 33, 32</td>
        <!-- <td>2</td> -->
        <!-- <td>6-wk-old male C57/BL6 mice entrained to a 12h:12h light:dark schedule for 1 wk, then released into constant darkness.</td> -->
        <td><a href="http://www.pnas.org/content/111/45/16219.abstract">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE PREGNANCY METABOLIC FETUS CIRCADIAN-DISRUPTION:</b> Maternal circadian rhythms during pregnancy influences circadian gene regulation in the mouse liver.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Pregnancy Metabolic Fetus</td> -->
        <td>Liver</td>
        <td>Circadian-Disruption</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE236201">GSE236201</a></td>
        <td>Light-Dark</td>
        <!-- <td></td> -->
        <td>0, 12, 18, 6</td>
        <!-- <td>4</td> -->
        <!-- <td>C57BL/6J pregnant mice were randomly divided and housed under either thecontrol 12h-light/ 12h-dark condition (CON) or chronically phase-shifted condition (CPS) where phase was advanced by 8 hours every 5 days exclusively during pregnancy. 4-week-old, male offspring mice were subjected to ad libitum low-fat diet or to high-fat diet for 12 weeks. Tissues were collected every 6hours over a circadian cycle at ZT0, ZT6, ZT12, and ZT18. Total RNA was extracted from the liver, and used for RNA-seq.</td> -->
        <td><a href="https://www.cell.com/cell-metabolism/fulltext/S1550-4131(24)00484-4">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE PREGNANCY METABOLIC FETUS CONTROL:</b> Maternal circadian rhythms during pregnancy influences circadian gene regulation in the mouse liver.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Pregnancy Metabolic Fetus</td> -->
        <td>Liver</td>
        <td>Control</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE236201">GSE236201</a></td>
        <td>Light-Dark</td>
        <!-- <td></td> -->
        <td>0, 12, 18, 6</td>
        <!-- <td>4</td> -->
        <!-- <td>C57BL/6J pregnant mice were randomly divided and housed under either thecontrol 12h-light/ 12h-dark condition (CON) or chronically phase-shifted condition (CPS) where phase was advanced by 8 hours every 5 days exclusively during pregnancy. 4-week-old, male offspring mice were subjected to ad libitum low-fat diet or to high-fat diet for 12 weeks. Tissues were collected every 6hours over a circadian cycle at ZT0, ZT6, ZT12, and ZT18. Total RNA was extracted from the liver, and used for RNA-seq.</td> -->
        <td><a href="https://www.cell.com/cell-metabolism/fulltext/S1550-4131(24)00484-4">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE PREGNANCY METABOLIC METHOR CIRCADIAN-DISRUPTION:</b> Maternal circadian rhythms during pregnancy influences circadian gene regulation in the mouse liver.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Pregnancy Metabolic Methor</td> -->
        <td>Liver</td>
        <td>Circadian-Disruption</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE236201">GSE236201</a></td>
        <td>Light-Dark</td>
        <!-- <td></td> -->
        <td>0, 12, 18, 6</td>
        <!-- <td>4</td> -->
        <!-- <td>C57BL/6J pregnant mice were randomly divided and housed under either thecontrol 12h-light/ 12h-dark condition (CON) or chronically phase-shifted condition (CPS) where phase was advanced by 8 hours every 5 days exclusively during pregnancy. 4-week-old, male offspring mice were subjected to ad libitum low-fat diet or to high-fat diet for 12 weeks. Tissues were collected every 6hours over a circadian cycle at ZT0, ZT6, ZT12, and ZT18. Total RNA was extracted from the liver, and used for RNA-seq.</td> -->
        <td><a href="https://www.cell.com/cell-metabolism/fulltext/S1550-4131(24)00484-4">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE PREGNANCY METABOLIC METHOR CONTROL:</b> Maternal circadian rhythms during pregnancy influences circadian gene regulation in the mouse liver.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Pregnancy Metabolic Methor</td> -->
        <td>Liver</td>
        <td>Control</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE236201">GSE236201</a></td>
        <td>Light-Dark</td>
        <!-- <td></td> -->
        <td>0, 12, 18, 6</td>
        <!-- <td>4</td> -->
        <!-- <td>C57BL/6J pregnant mice were randomly divided and housed under either thecontrol 12h-light/ 12h-dark condition (CON) or chronically phase-shifted condition (CPS) where phase was advanced by 8 hours every 5 days exclusively during pregnancy. 4-week-old, male offspring mice were subjected to ad libitum low-fat diet or to high-fat diet for 12 weeks. Tissues were collected every 6hours over a circadian cycle at ZT0, ZT6, ZT12, and ZT18. Total RNA was extracted from the liver, and used for RNA-seq.</td> -->
        <td><a href="https://www.cell.com/cell-metabolism/fulltext/S1550-4131(24)00484-4">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE PREGNANCY METABOLIC OFFSPRING CIRCADIAN-DISRUPTION HIGH-FAT:</b> Maternal circadian rhythms during pregnancy influences circadian gene regulation in the mouse liver.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Pregnancy Metabolic Offspring Circadian-disruption</td> -->
        <td>Liver</td>
        <td>High-Fat</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE236201">GSE236201</a></td>
        <td>Light-Dark, Diet</td>
        <!-- <td></td> -->
        <td>0, 12, 18, 6</td>
        <!-- <td>4</td> -->
        <!-- <td>C57BL/6J pregnant mice were randomly divided and housed under either thecontrol 12h-light/ 12h-dark condition (CON) or chronically phase-shifted condition (CPS) where phase was advanced by 8 hours every 5 days exclusively during pregnancy. 4-week-old, male offspring mice were subjected to ad libitum low-fat diet or to high-fat diet for 12 weeks. Tissues were collected every 6hours over a circadian cycle at ZT0, ZT6, ZT12, and ZT18. Total RNA was extracted from the liver, and used for RNA-seq.</td> -->
        <td><a href="https://www.cell.com/cell-metabolism/fulltext/S1550-4131(24)00484-4">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE PREGNANCY METABOLIC OFFSPRING CIRCADIAN-DISRUPTION LOW-FAT:</b> Maternal circadian rhythms during pregnancy influences circadian gene regulation in the mouse liver.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Pregnancy Metabolic Offspring Circadian-disruption</td> -->
        <td>Liver</td>
        <td>Low-Fat</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE236201">GSE236201</a></td>
        <td>Light-Dark, Diet</td>
        <!-- <td></td> -->
        <td>0, 12, 18, 6</td>
        <!-- <td>4</td> -->
        <!-- <td>C57BL/6J pregnant mice were randomly divided and housed under either thecontrol 12h-light/ 12h-dark condition (CON) or chronically phase-shifted condition (CPS) where phase was advanced by 8 hours every 5 days exclusively during pregnancy. 4-week-old, male offspring mice were subjected to ad libitum low-fat diet or to high-fat diet for 12 weeks. Tissues were collected every 6hours over a circadian cycle at ZT0, ZT6, ZT12, and ZT18. Total RNA was extracted from the liver, and used for RNA-seq.</td> -->
        <td><a href="https://www.cell.com/cell-metabolism/fulltext/S1550-4131(24)00484-4">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE PREGNANCY METABOLIC OFFSPRING CONTROL HIGH-FAT:</b> Maternal circadian rhythms during pregnancy influences circadian gene regulation in the mouse liver.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Pregnancy Metabolic Offspring Control</td> -->
        <td>Liver</td>
        <td>High-Fat</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE236201">GSE236201</a></td>
        <td>Light-Dark, Diet</td>
        <!-- <td></td> -->
        <td>0, 12, 18, 6</td>
        <!-- <td>4</td> -->
        <!-- <td>C57BL/6J pregnant mice were randomly divided and housed under either thecontrol 12h-light/ 12h-dark condition (CON) or chronically phase-shifted condition (CPS) where phase was advanced by 8 hours every 5 days exclusively during pregnancy. 4-week-old, male offspring mice were subjected to ad libitum low-fat diet or to high-fat diet for 12 weeks. Tissues were collected every 6hours over a circadian cycle at ZT0, ZT6, ZT12, and ZT18. Total RNA was extracted from the liver, and used for RNA-seq.</td> -->
        <td><a href="https://www.cell.com/cell-metabolism/fulltext/S1550-4131(24)00484-4">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE PREGNANCY METABOLIC OFFSPRING CONTROL LOW-FAT:</b> Maternal circadian rhythms during pregnancy influences circadian gene regulation in the mouse liver.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Pregnancy Metabolic Offspring Control</td> -->
        <td>Liver</td>
        <td>Low-Fat</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE236201">GSE236201</a></td>
        <td>Light-Dark, Diet</td>
        <!-- <td></td> -->
        <td>0, 12, 18, 6</td>
        <!-- <td>4</td> -->
        <!-- <td>C57BL/6J pregnant mice were randomly divided and housed under either thecontrol 12h-light/ 12h-dark condition (CON) or chronically phase-shifted condition (CPS) where phase was advanced by 8 hours every 5 days exclusively during pregnancy. 4-week-old, male offspring mice were subjected to ad libitum low-fat diet or to high-fat diet for 12 weeks. Tissues were collected every 6hours over a circadian cycle at ZT0, ZT6, ZT12, and ZT18. Total RNA was extracted from the liver, and used for RNA-seq.</td> -->
        <td><a href="https://www.cell.com/cell-metabolism/fulltext/S1550-4131(24)00484-4">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE RESTRICTED FEEDING BAT AD-LIB:</b> Circadian gene expression in brown adipose tissue (BAT) and liver under the following five conditions: ad libitum (AL); temporally restricted during the dark phase (TR-night) or light phase (TR-day); and calorically restricted during the dark phase (CR-night) or light phase (CR-day)</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Restricted Feeding Bat</td> -->
        <td>Adipose</td>
        <td>Ad-Lib</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE266543">GSE266543</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 32, 44, 28, 36, 40, 0, 4, 8</td>
        <!-- <td>3</td> -->
        <!-- <td>To address systemic influences of daytime feeding on circadian profiles of gene expression, we generated mRNA-seq for 6 months-old C57BL/6J male mice fed under different feeding conditions. We collected liver and brown adipose tissue (BAT) samples every 4h during 2 days in constant darkness. For liver (12 timepoints x 2 biological replicates x 5 feeding conditions , total 120 samples). For BAT, (12 timepoints x 2 feeding conditions , total 24 samples).</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.114523">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE RESTRICTED FEEDING BAT CR-DAY:</b> Circadian gene expression in brown adipose tissue (BAT) and liver under the following five conditions: ad libitum (AL); temporally restricted during the dark phase (TR-night) or light phase (TR-day); and calorically restricted during the dark phase (CR-night) or light phase (CR-day)</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Restricted Feeding Bat</td> -->
        <td>Adipose</td>
        <td>Cr-Day</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE266543">GSE266543</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 32, 44, 28, 36, 40, 0, 4, 8</td>
        <!-- <td>3</td> -->
        <!-- <td>To address systemic influences of daytime feeding on circadian profiles of gene expression, we generated mRNA-seq for 6 months-old C57BL/6J male mice fed under different feeding conditions. We collected liver and brown adipose tissue (BAT) samples every 4h during 2 days in constant darkness. For liver (12 timepoints x 2 biological replicates x 5 feeding conditions , total 120 samples). For BAT, (12 timepoints x 2 feeding conditions , total 24 samples).</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.114523">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE RESTRICTED FEEDING LIVER AD-LIB:</b> Circadian gene expression in brown adipose tissue (BAT) and liver under the following five conditions: ad libitum (AL); temporally restricted during the dark phase (TR-night) or light phase (TR-day); and calorically restricted during the dark phase (CR-night) or light phase (CR-day)</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Restricted Feeding Liver</td> -->
        <td>Liver</td>
        <td>Ad-Lib</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE266543">GSE266543</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 32, 44, 28, 36, 40, 0, 4, 8</td>
        <!-- <td>3</td> -->
        <!-- <td>To address systemic influences of daytime feeding on circadian profiles of gene expression, we generated mRNA-seq for 6 months-old C57BL/6J male mice fed under different feeding conditions. We collected liver and brown adipose tissue (BAT) samples every 4h during 2 days in constant darkness. For liver (12 timepoints x 2 biological replicates x 5 feeding conditions , total 120 samples). For BAT, (12 timepoints x 2 feeding conditions , total 24 samples).</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.114523">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE RESTRICTED FEEDING LIVER CR-DAY:</b> Circadian gene expression in brown adipose tissue (BAT) and liver under the following five conditions: ad libitum (AL); temporally restricted during the dark phase (TR-night) or light phase (TR-day); and calorically restricted during the dark phase (CR-night) or light phase (CR-day)</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Restricted Feeding Liver</td> -->
        <td>Liver</td>
        <td>Cr-Day</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE266543">GSE266543</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 32, 44, 28, 36, 40, 0, 4, 8</td>
        <!-- <td>3</td> -->
        <!-- <td>To address systemic influences of daytime feeding on circadian profiles of gene expression, we generated mRNA-seq for 6 months-old C57BL/6J male mice fed under different feeding conditions. We collected liver and brown adipose tissue (BAT) samples every 4h during 2 days in constant darkness. For liver (12 timepoints x 2 biological replicates x 5 feeding conditions , total 120 samples). For BAT, (12 timepoints x 2 feeding conditions , total 24 samples).</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.114523">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE RESTRICTED FEEDING LIVER CR-NIGHT:</b> Circadian gene expression in brown adipose tissue (BAT) and liver under the following five conditions: ad libitum (AL); temporally restricted during the dark phase (TR-night) or light phase (TR-day); and calorically restricted during the dark phase (CR-night) or light phase (CR-day)</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Restricted Feeding Liver</td> -->
        <td>Liver</td>
        <td>Cr-Night</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE266543">GSE266543</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 32, 44, 28, 36, 40, 0, 4, 8</td>
        <!-- <td>3</td> -->
        <!-- <td>To address systemic influences of daytime feeding on circadian profiles of gene expression, we generated mRNA-seq for 6 months-old C57BL/6J male mice fed under different feeding conditions. We collected liver and brown adipose tissue (BAT) samples every 4h during 2 days in constant darkness. For liver (12 timepoints x 2 biological replicates x 5 feeding conditions , total 120 samples). For BAT, (12 timepoints x 2 feeding conditions , total 24 samples).</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.114523">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE RESTRICTED FEEDING LIVER TR-DAY:</b> Circadian gene expression in brown adipose tissue (BAT) and liver under the following five conditions: ad libitum (AL); temporally restricted during the dark phase (TR-night) or light phase (TR-day); and calorically restricted during the dark phase (CR-night) or light phase (CR-day)</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Restricted Feeding Liver</td> -->
        <td>Liver</td>
        <td>Tr-Day</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE266543">GSE266543</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 32, 44, 28, 36, 40, 0, 4, 8</td>
        <!-- <td>3</td> -->
        <!-- <td>To address systemic influences of daytime feeding on circadian profiles of gene expression, we generated mRNA-seq for 6 months-old C57BL/6J male mice fed under different feeding conditions. We collected liver and brown adipose tissue (BAT) samples every 4h during 2 days in constant darkness. For liver (12 timepoints x 2 biological replicates x 5 feeding conditions , total 120 samples). For BAT, (12 timepoints x 2 feeding conditions , total 24 samples).</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.114523">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE RESTRICTED FEEDING LIVER TR-NIGHT:</b> Circadian gene expression in brown adipose tissue (BAT) and liver under the following five conditions: ad libitum (AL); temporally restricted during the dark phase (TR-night) or light phase (TR-day); and calorically restricted during the dark phase (CR-night) or light phase (CR-day)</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Restricted Feeding Liver</td> -->
        <td>Liver</td>
        <td>Tr-Night</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE266543">GSE266543</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 32, 44, 28, 36, 40, 0, 4, 8</td>
        <!-- <td>3</td> -->
        <!-- <td>To address systemic influences of daytime feeding on circadian profiles of gene expression, we generated mRNA-seq for 6 months-old C57BL/6J male mice fed under different feeding conditions. We collected liver and brown adipose tissue (BAT) samples every 4h during 2 days in constant darkness. For liver (12 timepoints x 2 biological replicates x 5 feeding conditions , total 120 samples). For BAT, (12 timepoints x 2 feeding conditions , total 24 samples).</td> -->
        <td><a href="https://doi.org/10.1016/j.celrep.2024.114523">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE SCN CONTROL:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Scn</td> -->
        <td>Brain</td>
        <td>Control</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE SCN DZNEP:</b> RNASeq of SCN in a DZnep treatment experiment</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Scn</td> -->
        <td>Brain</td>
        <td>Dznep</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>9, 3, 15, 21</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/33328229/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE SCN HIGH-FAT:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Scn</td> -->
        <td>Brain</td>
        <td>High-Fat</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE SCN NOLAN 2025 CRENEG:</b> Circadian gene expression in the suprachiasmatic nucleus (SCN) of mice before and after inducible deletion of Zfhx3 (a circadian period-modulating gene) via tamoxifen, compared to Cre-negative controls. Inducible Zfhx3 knockout in adults shortens SCN clock period and causes loss of rhythms in many SCN genes, confirming that ZFHX3 is crucial for maintaining robust circadian oscillations in the adult SCN.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Scn</td> -->
        <td>Brain</td>
        <td>Creneg</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE261429">GSE261429</a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>7</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1177/0748730417722631">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE SCN NOLAN 2025 POSTTAM:</b> Circadian gene expression in the suprachiasmatic nucleus (SCN) of mice before and after inducible deletion of Zfhx3 (a circadian period-modulating gene) via tamoxifen, compared to Cre-negative controls. Inducible Zfhx3 knockout in adults shortens SCN clock period and causes loss of rhythms in many SCN genes, confirming that ZFHX3 is crucial for maintaining robust circadian oscillations in the adult SCN.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Scn</td> -->
        <td>Brain</td>
        <td>Posttam</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE261429">GSE261429</a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>7</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1177/0748730417722631">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE SCN NOLAN 2025 PRETAM:</b> Circadian gene expression in the suprachiasmatic nucleus (SCN) of mice before and after inducible deletion of Zfhx3 (a circadian period-modulating gene) via tamoxifen, compared to Cre-negative controls. Inducible Zfhx3 knockout in adults shortens SCN clock period and causes loss of rhythms in many SCN genes, confirming that ZFHX3 is crucial for maintaining robust circadian oscillations in the adult SCN.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Scn</td> -->
        <td>Brain</td>
        <td>Pretam</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE261429">GSE261429</a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>7</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1177/0748730417722631">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE SCN PETRUS 2020 CONTROL-DARK:</b> Mouse treated with different diet conditions and different feeding conditions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Scn</td> -->
        <td>Brain</td>
        <td>Control-Dark</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE SCN PETRUS 2020 CONTROL-LIGHT:</b> Mouse treated with different diet conditions and different feeding conditions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Scn</td> -->
        <td>Brain</td>
        <td>Control-Light</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE SCN PETRUS 2020 TRYPTOPHAN-DARK:</b> Mouse treated with different diet conditions and different feeding conditions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Scn</td> -->
        <td>Brain</td>
        <td>Tryptophan-Dark</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE SCN PETRUS 2020 TRYPTOPHAN-LIGHT:</b> Mouse treated with different diet conditions and different feeding conditions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Scn</td> -->
        <td>Brain</td>
        <td>Tryptophan-Light</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE SCN SALINE WT:</b> RNASeq of SCN in a saline treatment experiment</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Scn Saline</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>9, 3, 15, 21</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/33328229/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE SKELETAL MUSCLE NA 2018 SIRT1-KO:</b> Skeletal muscle RNASeq in a Sirt1KO experiment via a cre-lox system</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Skeletal Muscle</td> -->
        <td>Muscle</td>
        <td>Sirt1-KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>19, 25, 13, 7</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE SKELETAL MUSCLE NA 2018 WT:</b> Skeletal muscle RNASeq in a Sirt1KO experiment via a cre-lox system</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Skeletal Muscle</td> -->
        <td>Muscle</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>19, 25, 13, 7</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE STRIATUM NA 2020 COCAINE-D2R-KO:</b> RNASeq of striatum in a cocaine treatment experiment with D2R Knock-Out</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Striatum</td> -->
        <td>Brain</td>
        <td>Cocaine-D2R-KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE142657">GSE142657</a></td>
        <td>Drug Treatment</td>
        <!-- <td></td> -->
        <td>11, 15, 23, 19, 3, 7</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://www.nature.com/articles/s41467-020-18200-6">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE STRIATUM NA 2020 COCAINE.:</b> RNASeq of striatum in a cocaine treatment experiment with D2R Knock-Out</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Striatum</td> -->
        <td>Brain</td>
        <td>Cocaine.</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE142657">GSE142657</a></td>
        <td>Drug Treatment</td>
        <!-- <td></td> -->
        <td>11, 15, 23, 19, 3, 7</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://www.nature.com/articles/s41467-020-18200-6">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE STRIATUM NA 2020 D2R-KO.:</b> RNASeq of striatum in a cocaine treatment experiment with D2R Knock-Out</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Striatum</td> -->
        <td>Brain</td>
        <td>D2R-KO.</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE142657">GSE142657</a></td>
        <td>Drug Treatment</td>
        <!-- <td></td> -->
        <td>11, 15, 23, 19, 3, 7</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://www.nature.com/articles/s41467-020-18200-6">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE STRIATUM NA 2020 WT:</b> RNASeq of striatum in a cocaine treatment experiment with D2R Knock-Out</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Striatum</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE142657">GSE142657</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>11, 15, 23, 19, 3, 7</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://www.nature.com/articles/s41467-020-18200-6">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE SUPRACHIASMATIC NUCLEUS BALLANCE 2015 WT:</b> Circadian RNA expression profile of the mammalian biological clock, the suprachiasmatic nucleus (SCN) in C57/BL6 mice, at 2-hour resolution using microarrays.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Suprachiasmatic Nucleus</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td>C57/BL6</td> -->
        <td>56, 42, 50, 60, 62, 64, 32, 24, 26, 20, 48, 46, 44, 28, 40, 52, 58, 38, 22, 54, 30, 36, 34, 18</td>
        <!-- <td>2</td> -->
        <!-- <td>24 samples total covering 24 time points, with no replicates. SCN samples from mouse brains collected every 2 hours for 2 days</td> -->
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE70384">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE SUPRACHIASMATIC NUCLEUS PANDA 2002 WT:</b> Gene expression profiling to identify cycling transcripts in the SCN and in the liver</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Suprachiasmatic Nucleus</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td>C57BL/6J</td> -->
        <td>26, 38, 58, 22, 18, 30, 42, 50, 34, 62, 54</td>
        <!-- <td>3</td> -->
        <!-- <td>Entrained C57BL/6J mice to a 12-hour light, 12-hour dark cycle (LD12:12) under constant temperature for two weeks, and then transferred the mice to constant darkness.</td> -->
        <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/12015981">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE SYNTHESIS AND DEGRADATION EXON KO:</b> Rhythmic RNA synthesis and rhythmic degradation and their importance on 24-h and 12-h RNA rhythms. These rhythms were predominantly regulated by Bmal1 and/or the core clock mechanism</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Synthesis And Degradation Exon</td> -->
        <td>Cells</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE253826">GSE253826</a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>24, 26, 38, 46, 32, 44, 30, 28, 36, 40, 34, 42</td>
        <!-- <td>2</td> -->
        <!-- <td>Total RNA or nascent 4sU labeled RNA were submitted to Azenta for RNA sequencing. rRNA depletion was performed and library prep with 150bp reads using standard Tru-seq adapters.</td> -->
        <td><a href="https://www.pnas.org/doi/abs/10.1073/pnas.2314690121?af=R">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE SYNTHESIS AND DEGRADATION EXON WT:</b> Rhythmic RNA synthesis and rhythmic degradation and their importance on 24-h and 12-h RNA rhythms. These rhythms were predominantly regulated by Bmal1 and/or the core clock mechanism</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Synthesis And Degradation Exon</td> -->
        <td>Cells</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE253826">GSE253826</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 26, 38, 46, 32, 44, 30, 28, 36, 40, 34, 42</td>
        <!-- <td>2</td> -->
        <!-- <td>Total RNA or nascent 4sU labeled RNA were submitted to Azenta for RNA sequencing. rRNA depletion was performed and library prep with 150bp reads using standard Tru-seq adapters.</td> -->
        <td><a href="https://www.pnas.org/doi/abs/10.1073/pnas.2314690121?af=R">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE SYNTHESIS AND DEGRADATION INTRON KO:</b> Rhythmic RNA synthesis and rhythmic degradation and their importance on 24-h and 12-h RNA rhythms. These rhythms were predominantly regulated by Bmal1 and/or the core clock mechanism</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Synthesis And Degradation Intron</td> -->
        <td>Cells</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE253826">GSE253826</a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>24, 26, 38, 46, 32, 44, 30, 28, 36, 40, 34, 42</td>
        <!-- <td>2</td> -->
        <!-- <td>Total RNA or nascent 4sU labeled RNA were submitted to Azenta for RNA sequencing. rRNA depletion was performed and library prep with 150bp reads using standard Tru-seq adapters.</td> -->
        <td><a href="https://www.pnas.org/doi/abs/10.1073/pnas.2314690121?af=R">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE SYNTHESIS AND DEGRADATION INTRON WT:</b> Rhythmic RNA synthesis and rhythmic degradation and their importance on 24-h and 12-h RNA rhythms. These rhythms were predominantly regulated by Bmal1 and/or the core clock mechanism</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Synthesis And Degradation Intron</td> -->
        <td>Cells</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE253826">GSE253826</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 26, 38, 46, 32, 44, 30, 28, 36, 40, 34, 42</td>
        <!-- <td>2</td> -->
        <!-- <td>Total RNA or nascent 4sU labeled RNA were submitted to Azenta for RNA sequencing. rRNA depletion was performed and library prep with 150bp reads using standard Tru-seq adapters.</td> -->
        <td><a href="https://www.pnas.org/doi/abs/10.1073/pnas.2314690121?af=R">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE TRANSCRIPTOME EVENING-EXERCISE:</b> Transcriptomic data for mouse exercise in light dark conditions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Transcriptome</td> -->
        <td>Liver</td>
        <td>Evening-Exercise</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 4, 8</td>
        <!-- <td>7</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE TRANSCRIPTOME EVENING-SEDENTARY:</b> Transcriptomic data for mouse exercise in light dark conditions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Transcriptome</td> -->
        <td>Liver</td>
        <td>Evening-Sedentary</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 4, 8</td>
        <!-- <td>7</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE TRANSCRIPTOME MORNING-EXERCISE:</b> Transcriptomic data for mouse exercise in light dark conditions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Transcriptome</td> -->
        <td>Liver</td>
        <td>Morning-Exercise</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 4, 8</td>
        <!-- <td>7</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE TRANSCRIPTOME MORNING-SEDENTARY:</b> Transcriptomic data for mouse exercise in light dark conditions.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Transcriptome</td> -->
        <td>Liver</td>
        <td>Morning-Sedentary</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 4, 8</td>
        <!-- <td>7</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE VENTRAL HIPPOCAMPUS KONRAD 2017 TEMPORAL-LOBE-EPILEPTIC:</b> Hippocampus RNASeq in an experiment comparing epileptic vs normal brain.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Ventral Hippocampus</td> -->
        <td>Brain</td>
        <td>Temporal-Lobe-Epileptic</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54652">GSE54652</a></td>
        <td>Disease</td>
        <!-- <td></td> -->
        <td>11, 15, 23, 19, 3, 7</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="https://www.biorxiv.org/content/early/2017/10/06/199372">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE VENTRAL HIPPOCAMPUS KONRAD 2017 WT:</b> Hippocampus RNASeq in an experiment comparing epileptic vs normal brain.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Ventral Hippocampus</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54652">GSE54652</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>11, 15, 23, 19, 3, 7</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="https://www.biorxiv.org/content/early/2017/10/06/199372">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>NEUROSPORA CRASSA ANANTHASUBRAMANIAM 2018 DMSN1:</b> Sampling of liquid culture grown N. crassa every 2h over 22h in the dark from light to dark transition.</td>
        <td><i>Neurospora crassa</i></td>
        <!-- <td>Crassa</td> -->
        <td>Cells</td>
        <td>Dmsn1</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Light-Dark</td>
        <!-- <td></td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/30472532/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>NEUROSPORA CRASSA ANANTHASUBRAMANIAM 2018 WT:</b> Sampling of liquid culture grown N. crassa every 2h over 22h in the dark from light to dark transition.</td>
        <td><i>Neurospora crassa</i></td>
        <!-- <td>Crassa</td> -->
        <td>Cells</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>10, 12, 20, 14, 22, 16, 18, 0, 2, 4, 6, 8</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/30472532/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>NEUROSPORA GCN2 SIGNALING PATHWAY KO:</b> GCN2 signaling pathway in circadian clock function by regulating histone acetylation under amino acid starvation.</td>
        <td><i>Neurospora crassa</i></td>
        <!-- <td>Gcn2 Signaling Pathway</td> -->
        <td>Cells</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE220169">GSE220169</a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>0, 12</td>
        <!-- <td>4</td> -->
        <!-- <td>For RT-qPCR, the cultures of WT and cpc-1KO strains were collected at the indicated time points in constant darkness in liquid growth medium</td> -->
        <td><a href="https://elifesciences.org/articles/85241">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>NEUROSPORA GCN2 SIGNALING PATHWAY WT:</b> GCN2 signaling pathway in circadian clock function by regulating histone acetylation under amino acid starvation.</td>
        <td><i>Neurospora crassa</i></td>
        <!-- <td>Gcn2 Signaling Pathway</td> -->
        <td>Cells</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE220169">GSE220169</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>0, 12</td>
        <!-- <td>4</td> -->
        <!-- <td>For RT-qPCR, the cultures of WT and cpc-1KO strains were collected at the indicated time points in constant darkness in liquid growth medium</td> -->
        <td><a href="https://elifesciences.org/articles/85241">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>PFC PYRAMIDAL PV CIRCADIAN FEMALE-PV:</b> In mouse medial prefrontal cortex, ~35 % of pyramidal-cell transcripts and ~18 % of PV-interneuron transcripts are rhythmic; PV rhythms are largely sex-specific.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Pyramidal Pv Circadian</td> -->
        <td>Brain</td>
        <td>Female-Pv</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE287818">GSE287818</a></td>
        <td>Sex</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1101/2024.07.05.601880">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>PFC PYRAMIDAL PV CIRCADIAN FEMALE-PYR:</b> In mouse medial prefrontal cortex, ~35 % of pyramidal-cell transcripts and ~18 % of PV-interneuron transcripts are rhythmic; PV rhythms are largely sex-specific.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Pyramidal Pv Circadian</td> -->
        <td>Brain</td>
        <td>Female-Pyr</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE287818">GSE287818</a></td>
        <td>Sex</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1101/2024.07.05.601880">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>PFC PYRAMIDAL PV CIRCADIAN MALE-PV:</b> In mouse medial prefrontal cortex, ~35 % of pyramidal-cell transcripts and ~18 % of PV-interneuron transcripts are rhythmic; PV rhythms are largely sex-specific.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Pyramidal Pv Circadian</td> -->
        <td>Brain</td>
        <td>Male-Pv</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE287818">GSE287818</a></td>
        <td>Sex</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1101/2024.07.05.601880">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>PFC PYRAMIDAL PV CIRCADIAN MALE-PYR:</b> In mouse medial prefrontal cortex, ~35 % of pyramidal-cell transcripts and ~18 % of PV-interneuron transcripts are rhythmic; PV rhythms are largely sex-specific.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Pyramidal Pv Circadian</td> -->
        <td>Brain</td>
        <td>Male-Pyr</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE287818">GSE287818</a></td>
        <td>Sex</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1101/2024.07.05.601880">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RAT BRAIN STAEHLE 2020 CEA-WT:</b> High-throughput qRT-PCR profiling of 145 genes using Fluidigm's 96.96 array. Bilateral samples from a single animal were considered as one sample. There was no inter-animal pooling of samples</td>
        <td><i>Rattus norvegicus</i></td>
        <!-- <td>Brain</td> -->
        <td>Brain</td>
        <td>Cea-Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>9, 3, 5</td>
        <!-- <td>18</td> -->
        <!-- <td></td> -->
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/32477043/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RAT BRAIN STAEHLE 2020 DVC-WT:</b> High-throughput qRT-PCR profiling of 145 genes using Fluidigm's 96.96 array. Bilateral samples from a single animal were considered as one sample. There was no inter-animal pooling of samples</td>
        <td><i>Rattus norvegicus</i></td>
        <!-- <td>Brain</td> -->
        <td>Brain</td>
        <td>Dvc-Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>9, 3, 5</td>
        <!-- <td>18</td> -->
        <!-- <td></td> -->
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/32477043/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RAT LIVER ALMON 2008 WT:</b> Global gene expression analysis in the identification of circadian-regulated genes involved in drug action.</td>
        <td><i>Rattus norvegicus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE8988">GSE8988</a></td>
        <td>Control</td>
        <!-- <td>Male Winstar</td> -->
        <td>11, 10, 13, 12, 20, 14, 22, 16, 24, 18, 23, 1, 2, 4, 6, 8</td>
        <!-- <td>7</td> -->
        <!-- <td>Twenty-seven rats were acclimatized for 2 weeks prior to study to a normal light/dark cycle.</td> -->
        <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18562560">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RAT SLEEP AND BRAIN DISORDERS SLEEP-DEPRIVED:</b> Sleep deprivation in Long Evans rats were subjects being sleep deprived for 6 hours and then samples being collected every 2h during the recovery.</td>
        <td><i>Rattus norvegicus</i></td>
        <!-- <td>Sleep And Brain Disorders</td> -->
        <td>Brain</td>
        <td>Sleep-Deprived</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE250324">GSE250324</a></td>
        <td>Sleep</td>
        <!-- <td></td> -->
        <td>0, 2, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td>In a series of experiments involving Long Evans rats, thirty were divided into six groups to study the effects of sleep deprivation (SD) by keeping them awake for 6 hours using an automated bedding stir bar. The rats were housed under controlled conditions and acclimated to these conditions before undergoing SD. Post-SD, they were euthanized at various times for analysis. The forebrains were extracted, frozen, and stored for RNA extraction, which was performed under specific light conditions. The RNA was then isolated using a standard protocol involving homogenization, purification with Trizol Reagent, and subsequent analysis to study the impact of SD on gene expression.</td> -->
        <td><a href="https://www.researchsquare.com/article/rs-3867797/v1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RAT SLEEP AND BRAIN DISORDERS WT:</b> Sleep deprivation in Long Evans rats were subjects being sleep deprived for 6 hours and then samples being collected every 2h during the recovery.</td>
        <td><i>Rattus norvegicus</i></td>
        <!-- <td>Sleep And Brain Disorders</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE250324">GSE250324</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>0, 2, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td>In a series of experiments involving Long Evans rats, thirty were divided into six groups to study the effects of sleep deprivation (SD) by keeping them awake for 6 hours using an automated bedding stir bar. The rats were housed under controlled conditions and acclimated to these conditions before undergoing SD. Post-SD, they were euthanized at various times for analysis. The forebrains were extracted, frozen, and stored for RNA extraction, which was performed under specific light conditions. The RNA was then isolated using a standard protocol involving homogenization, purification with Trizol Reagent, and subsequent analysis to study the impact of SD on gene expression.</td> -->
        <td><a href="https://www.researchsquare.com/article/rs-3867797/v1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY ADRENAL GLAND FEMALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Adrenal Gland Female</td> -->
        <td>Glands</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY ADRENAL GLAND FEMALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Adrenal Gland Female</td> -->
        <td>Glands</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY ADRENAL GLAND MALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Adrenal Gland Male</td> -->
        <td>Glands</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY ADRENAL GLAND MALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Adrenal Gland Male</td> -->
        <td>Glands</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY BROWN ADIPOSE FEMALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Brown Adipose Female</td> -->
        <td>Adipose</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY BROWN ADIPOSE FEMALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Brown Adipose Female</td> -->
        <td>Adipose</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY BROWN ADIPOSE MALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Brown Adipose Male</td> -->
        <td>Adipose</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY BROWN ADIPOSE MALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Brown Adipose Male</td> -->
        <td>Adipose</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY EXTENSOR DIGITORUM LONGUS FEMALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Extensor Digitorum Longus Female</td> -->
        <td>Muscle</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY EXTENSOR DIGITORUM LONGUS FEMALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Extensor Digitorum Longus Female</td> -->
        <td>Muscle</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY EXTENSOR DIGITORUM LONGUS MALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Extensor Digitorum Longus Male</td> -->
        <td>Muscle</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY EXTENSOR DIGITORUM LONGUS MALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Extensor Digitorum Longus Male</td> -->
        <td>Muscle</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY EXTENSOR DIGITORUM LONGUS TENDON FEMALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Extensor Digitorum Longus Tendon Female</td> -->
        <td>Cells</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY EXTENSOR DIGITORUM LONGUS TENDON FEMALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Extensor Digitorum Longus Tendon Female</td> -->
        <td>Cells</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY EXTENSOR DIGITORUM LONGUS TENDON MALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Extensor Digitorum Longus Tendon Male</td> -->
        <td>Cells</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY EXTENSOR DIGITORUM LONGUS TENDON MALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Extensor Digitorum Longus Tendon Male</td> -->
        <td>Cells</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY GASTROCNEMIUS FEMALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Gastrocnemius Female</td> -->
        <td>Muscle</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY GASTROCNEMIUS FEMALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Gastrocnemius Female</td> -->
        <td>Muscle</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY GASTROCNEMIUS MUSCLE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Gastrocnemius Muscle</td> -->
        <td>Muscle</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY GASTROCNEMIUS MUSCLE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Gastrocnemius Muscle</td> -->
        <td>Muscle</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY GASTROCNEMIUS TENDON MALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Gastrocnemius Tendon Male</td> -->
        <td>Cells</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY GASTROCNEMIUS TENDON MALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Gastrocnemius Tendon Male</td> -->
        <td>Cells</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY GASTROCNEMIUSTENDON FEMALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Gastrocnemiustendon Female</td> -->
        <td>Cells</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY GASTROCNEMIUSTENDON FEMALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Gastrocnemiustendon Female</td> -->
        <td>Cells</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY HEART FEMALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Heart Female</td> -->
        <td>Heart</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY HEART FEMALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Heart Female</td> -->
        <td>Heart</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY HEART MALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Heart Male</td> -->
        <td>Heart</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY HEART MALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Heart Male</td> -->
        <td>Heart</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY KIDNEY FEMALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Kidney Female</td> -->
        <td>Kidney</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY KIDNEY FEMALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Kidney Female</td> -->
        <td>Kidney</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY KIDNEY MALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Kidney Male</td> -->
        <td>Kidney</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY KIDNEY MALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Kidney Male</td> -->
        <td>Kidney</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY LIVER FEMALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Liver Female</td> -->
        <td>Liver</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY LIVER FEMALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Liver Female</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY LIVER MALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Liver Male</td> -->
        <td>Liver</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY LIVER MALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Liver Male</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY LUNG FEMALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Lung Female</td> -->
        <td>Lung</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY LUNG FEMALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Lung Female</td> -->
        <td>Lung</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY LUNG MALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Lung Male</td> -->
        <td>Lung</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY LUNG MALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Lung Male</td> -->
        <td>Lung</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY OVARY FEMALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Ovary Female</td> -->
        <td>Reproductive</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY OVARY FEMALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Ovary Female</td> -->
        <td>Reproductive</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY PITUITARY FEMALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Pituitary Female</td> -->
        <td>Glands</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY PITUITARY FEMALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Pituitary Female</td> -->
        <td>Glands</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY PITUITARY MALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Pituitary Male</td> -->
        <td>Glands</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY PITUITARY MALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Pituitary Male</td> -->
        <td>Glands</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY QUADRICEPS FEMALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Quadriceps Female</td> -->
        <td>Muscle</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY QUADRICEPS FEMALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Quadriceps Female</td> -->
        <td>Muscle</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY QUADRICEPS MALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Quadriceps Male</td> -->
        <td>Muscle</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY QUADRICEPS MALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Quadriceps Male</td> -->
        <td>Muscle</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY SOLEUS FEMALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Soleus Female</td> -->
        <td>Muscle</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY SOLEUS FEMALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Soleus Female</td> -->
        <td>Muscle</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY SOLEUS MALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Soleus Male</td> -->
        <td>Muscle</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY SOLEUS MALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Soleus Male</td> -->
        <td>Muscle</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY SPLEEN FEMALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Spleen Female</td> -->
        <td>Spleen</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY SPLEEN FEMALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Spleen Female</td> -->
        <td>Spleen</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY SPLEEN MALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Spleen Male</td> -->
        <td>Spleen</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY SPLEEN MALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Spleen Male</td> -->
        <td>Spleen</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY TESTIS MALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Testis Male</td> -->
        <td>Reproductive</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY TESTIS MALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Testis Male</td> -->
        <td>Reproductive</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY TIBIALIS ANTERIOR FEMALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Tibialis Anterior Female</td> -->
        <td>Muscle</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY TIBIALIS ANTERIOR FEMALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Tibialis Anterior Female</td> -->
        <td>Muscle</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY TIBIALIS ANTERIOR MALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Tibialis Anterior Male</td> -->
        <td>Muscle</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY TIBIALIS ANTERIOR MALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Tibialis Anterior Male</td> -->
        <td>Muscle</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY TIBIALIS ANTERIOR TENDON FEMALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Tibialis Anterior Tendon Female</td> -->
        <td>Cells</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY TIBIALIS ANTERIOR TENDON FEMALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Tibialis Anterior Tendon Female</td> -->
        <td>Cells</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY TIBIALIS ANTERIOR TENDON MALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Tibialis Anterior Tendon Male</td> -->
        <td>Cells</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>3</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY TIBIALIS ANTERIOR TENDON MALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Tibialis Anterior Tendon Male</td> -->
        <td>Cells</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>3</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY UTERUS FEMALE DEFICIENT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Uterus Female</td> -->
        <td>Reproductive</td>
        <td>Deficient</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RELATIVE ENERGY DEFICIENCY UTERUS FEMALE WT:</b> Placing mice under energy-deficient and -sufficient conditions through an exercise-for-food paradigm to explore the impact of energy deficiency.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Energy Deficiency Uterus Female</td> -->
        <td>Reproductive</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE243060">GSE243060</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>5, 17</td>
        <!-- <td>4</td> -->
        <!-- <td>Conducted an extensive analysis of the transcriptome of 19 different metabolic tissues from energy-sufficient (Ctrl) and energy-deficient (Deficient) female and male CBA/CaJ mice (419 samples of bulk RNA-seq passed quality control).</td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2024.08.001">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RIJOFERREIRA AEDES SALIVARYGLAND 2025 DD:</b> Daily gene expression rhythms in Aedes aegypti mosquito salivary glands under normal light-dark (LD) and constant darkness (DD) conditions, revealing that ~50% of salivary gland genes (including key blood-feeding factors) oscillate circadianly, aligning with the mosquito's nocturnal feeding behavior.</td>
        <td><i>Anopheles stephensi</i></td>
        <!-- <td>Aedes</td> -->
        <td>Salivary Gland</td>
        <td>Dd</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE238168">GSE238168</a></td>
        <td>Light-Dark</td>
        <!-- <td></td> -->
        <td>24, 18, 12, 6</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1038/s41564-025-01949-1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RIJOFERREIRA AEDES SALIVARYGLAND 2025 LD:</b> Daily gene expression rhythms in Aedes aegypti mosquito salivary glands under normal light-dark (LD) and constant darkness (DD) conditions, revealing that ~50% of salivary gland genes (including key blood-feeding factors) oscillate circadianly, aligning with the mosquito's nocturnal feeding behavior.</td>
        <td><i>Anopheles stephensi</i></td>
        <!-- <td>Aedes</td> -->
        <td>Salivary Gland</td>
        <td>Ld</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE238168">GSE238168</a></td>
        <td>Light-Dark</td>
        <!-- <td></td> -->
        <td>24, 18, 12, 6</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1038/s41564-025-01949-1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RIJOFERREIRA ANOPHELES SALIVARYGLAND 2025 DD:</b> Circadian RNA-seq of mosquito salivary glands collected under a 12 h light-dark cycle (LD) or constant darkness (DD). Roughly half of gland transcripts oscillate with ~24 h period- including genes encoding anti-hemostatic factors and odorant/chemosensory proteins- priming the vector for its nocturnal blood meal and enhancing malaria transmission efficiency.</td>
        <td><i>Anopheles stephensi</i></td>
        <!-- <td>Anopheles</td> -->
        <td>Salivary gland</td>
        <td>Dd</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE284425">GSE284425</a></td>
        <td>Light-Dark</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 48, 16, 32, 44, 56, 28, 36, 40, 52, 60, 0, 64, 4, 68, 8, 72</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1038/s41564-025-01949-1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RIJOFERREIRA ANOPHELES SALIVARYGLAND 2025 LD:</b> Circadian RNA-seq of mosquito salivary glands collected under a 12 h light-dark cycle (LD) or constant darkness (DD). Roughly half of gland transcripts oscillate with ~24 h period- including genes encoding anti-hemostatic factors and odorant/chemosensory proteins- priming the vector for its nocturnal blood meal and enhancing malaria transmission efficiency.</td>
        <td><i>Anopheles stephensi</i></td>
        <!-- <td>Anopheles</td> -->
        <td>Salivary gland</td>
        <td>Ld</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE284425">GSE284425</a></td>
        <td>Light-Dark</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 48, 16, 32, 44, 56, 28, 36, 40, 52, 60, 0, 64, 4, 68, 8, 72</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1038/s41564-025-01949-1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RIJOFERREIRA PLASMODIUM SPOROZOITE 2025 DD:</b> Circadian transcriptome of malaria-parasite sporozoites isolated from the same mosquito glands under normal light-dark (LD) and constant darkness (DD) conditions. A defined subset of sporozoite genes- many involved in motility and host invasion- display ~24 h oscillations, indicating that the parasite clock synchronises with the mosquito's to maximise nighttime infectivity.</td>
        <td><i>Plasmodium berghei</i></td>
        <!-- <td>Plasmodium</td> -->
        <td>Parasite</td>
        <td>Dd</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE284425">GSE284425</a></td>
        <td>Light-Dark</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 48, 16, 32, 44, 56, 28, 36, 40, 52, 60, 0, 64, 4, 68, 8, 72</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1038/s41564-025-01949-1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RIJOFERREIRA PLASMODIUM SPOROZOITE 2025 DD:</b> Circadian transcriptome of malaria-parasite sporozoites isolated from the same mosquito glands under normal light-dark (LD) and constant darkness (DD) conditions. A defined subset of sporozoite genes- many involved in motility and host invasion- display ~24 h oscillations, indicating that the parasite clock synchronises with the mosquito's to maximise nighttime infectivity.</td>
        <td><i>Plasmodium berghei</i></td>
        <!-- <td>Plasmodium</td> -->
        <td>Parasite</td>
        <td>Dd</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE284425">GSE284425</a></td>
        <td>Light-Dark</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 48, 16, 32, 44, 56, 28, 36, 40, 52, 60, 0, 64, 4, 68, 8, 72</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1038/s41564-025-01949-1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RIJOFERREIRA PLASMODIUM SPOROZOITE 2025 LD:</b> Circadian transcriptome of malaria-parasite sporozoites isolated from the same mosquito glands under normal light-dark (LD) and constant darkness (DD) conditions. A defined subset of sporozoite genes- many involved in motility and host invasion- display ~24 h oscillations, indicating that the parasite clock synchronises with the mosquito's to maximise nighttime infectivity.</td>
        <td><i>Plasmodium berghei</i></td>
        <!-- <td>Plasmodium</td> -->
        <td>Parasite</td>
        <td>Ld</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE284425">GSE284425</a></td>
        <td>Light-Dark</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 48, 16, 32, 44, 56, 28, 36, 40, 52, 60, 0, 64, 4, 68, 8, 72</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1038/s41564-025-01949-1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>RIJOFERREIRA PLASMODIUM SPOROZOITE 2025 LD:</b> Circadian transcriptome of malaria-parasite sporozoites isolated from the same mosquito glands under normal light-dark (LD) and constant darkness (DD) conditions. A defined subset of sporozoite genes- many involved in motility and host invasion- display ~24 h oscillations, indicating that the parasite clock synchronises with the mosquito's to maximise nighttime infectivity.</td>
        <td><i>Plasmodium berghei</i></td>
        <!-- <td>Plasmodium</td> -->
        <td>Parasite</td>
        <td>Ld</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE284425">GSE284425</a></td>
        <td>Light-Dark</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 48, 16, 32, 44, 56, 28, 36, 40, 52, 60, 0, 64, 4, 68, 8, 72</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1038/s41564-025-01949-1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>SHEEP RUMEN VFA DIET GRAIN:</b> High-VFA (grain) versus low-VFA (hay) media reset epithelial clock genes and transporters, linking nutrient load to the rumen circadian programme.</td>
        <td><i>Ovis aries</i></td>
        <!-- <td>Rumen Vfa Diet</td> -->
        <td>Rumen epithelium</td>
        <td>Grain</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE286932">GSE286932</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.3168/jds.2020-19622">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>SHEEP RUMEN VFA DIET HAY:</b> High-VFA (grain) versus low-VFA (hay) media reset epithelial clock genes and transporters, linking nutrient load to the rumen circadian programme.</td>
        <td><i>Ovis aries</i></td>
        <!-- <td>Rumen Vfa Diet</td> -->
        <td>Rumen epithelium</td>
        <td>Hay</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE286932">GSE286932</a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.3168/jds.2020-19622">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>SIFUENTES ZEBRAFISH 2016 INJURY:</b> Transcriptional profiles of 0, 8, and 16 hour post-lesion zebrafish Muller glia (in triplicate) were generated by high-throughput sequencing in an Illumina GAIIx</td>
        <td><i>Danio rerio</i></td>
        <!-- <td></td> -->
        <td>Eye</td>
        <td>Injury</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Injury</td>
        <!-- <td></td> -->
        <td>0, 16, 8</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/27699411/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>SLEEP DISRUPTION AND AGING FOREBRAIN SLEEP-DEPRIVED:</b> Property of circadian-affected tissue-specific genes in cerebellum, forebrain, liver, kidney, and their link to aging and longevity.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Disruption And Aging Forebrain</td> -->
        <td>Brain</td>
        <td>Sleep-Deprived</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE240693">GSE240693</a></td>
        <td>Sleep</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>3</td> -->
        <!-- <td>We extracted total RNA from tissue samples using Trizol (Invitrogen). After separating the RNA, DNA, and protein layers with chloroform, we precipitated the RNA using a standard isopropanol/ethanol procedure. The resulting RNA pellet was washed, and then resuspended in RNase-free water.</td> -->
        <td><a href="https://www.nature.com/articles/s42003-024-06421-0">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>SLEEP DISRUPTION AND AGING FOREBRAIN WT:</b> Property of circadian-affected tissue-specific genes in cerebellum, forebrain, liver, kidney, and their link to aging and longevity.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Disruption And Aging Forebrain</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE240693">GSE240693</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>3</td> -->
        <!-- <td>We extracted total RNA from tissue samples using Trizol (Invitrogen). After separating the RNA, DNA, and protein layers with chloroform, we precipitated the RNA using a standard isopropanol/ethanol procedure. The resulting RNA pellet was washed, and then resuspended in RNase-free water.</td> -->
        <td><a href="https://www.nature.com/articles/s42003-024-06421-0">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>SLEEP DISRUPTION AND AGING IN CEREBELLUM SLEEP-DEPRIVED:</b> Property of circadian-affected tissue-specific genes in cerebellum, forebrain, liver, kidney, and their link to aging and longevity.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Disruption And Aging In Cerebellum</td> -->
        <td>Brain</td>
        <td>Sleep-Deprived</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE240693">GSE240693</a></td>
        <td>Sleep</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>3</td> -->
        <!-- <td>We extracted total RNA from tissue samples using Trizol (Invitrogen). After separating the RNA, DNA, and protein layers with chloroform, we precipitated the RNA using a standard isopropanol/ethanol procedure. The resulting RNA pellet was washed, and then resuspended in RNase-free water.</td> -->
        <td><a href="https://www.nature.com/articles/s42003-024-06421-0">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>SLEEP DISRUPTION AND AGING IN CEREBELLUM WT:</b> Property of circadian-affected tissue-specific genes in cerebellum, forebrain, liver, kidney, and their link to aging and longevity.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Disruption And Aging In Cerebellum</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE240693">GSE240693</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>3</td> -->
        <!-- <td>We extracted total RNA from tissue samples using Trizol (Invitrogen). After separating the RNA, DNA, and protein layers with chloroform, we precipitated the RNA using a standard isopropanol/ethanol procedure. The resulting RNA pellet was washed, and then resuspended in RNase-free water.</td> -->
        <td><a href="https://www.nature.com/articles/s42003-024-06421-0">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>SLEEP DISRUPTION AND AGING IN KIDNEY SLEEP-DEPRIVED:</b> Property of circadian-affected tissue-specific genes in cerebellum, forebrain, liver, kidney, and their link to aging and longevity.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Disruption And Aging In Kidney</td> -->
        <td>Kidney</td>
        <td>Sleep-Deprived</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE240693">GSE240693</a></td>
        <td>Sleep</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>3</td> -->
        <!-- <td>We extracted total RNA from tissue samples using Trizol (Invitrogen). After separating the RNA, DNA, and protein layers with chloroform, we precipitated the RNA using a standard isopropanol/ethanol procedure. The resulting RNA pellet was washed, and then resuspended in RNase-free water.</td> -->
        <td><a href="https://www.nature.com/articles/s42003-024-06421-0">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>SLEEP DISRUPTION AND AGING IN KIDNEY WT:</b> Property of circadian-affected tissue-specific genes in cerebellum, forebrain, liver, kidney, and their link to aging and longevity.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Disruption And Aging In Kidney</td> -->
        <td>Kidney</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE240693">GSE240693</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>3</td> -->
        <!-- <td>We extracted total RNA from tissue samples using Trizol (Invitrogen). After separating the RNA, DNA, and protein layers with chloroform, we precipitated the RNA using a standard isopropanol/ethanol procedure. The resulting RNA pellet was washed, and then resuspended in RNase-free water.</td> -->
        <td><a href="https://www.nature.com/articles/s42003-024-06421-0">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>SLEEP DISRUPTION AND AGING IN LIVER SLEEP-DEPRIVED:</b> Property of circadian-affected tissue-specific genes in cerebellum, forebrain, liver, kidney, and their link to aging and longevity.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Disruption And Aging In Liver</td> -->
        <td>Liver</td>
        <td>Sleep-Deprived</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE240693">GSE240693</a></td>
        <td>Sleep</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>3</td> -->
        <!-- <td>We extracted total RNA from tissue samples using Trizol (Invitrogen). After separating the RNA, DNA, and protein layers with chloroform, we precipitated the RNA using a standard isopropanol/ethanol procedure. The resulting RNA pellet was washed, and then resuspended in RNase-free water.</td> -->
        <td><a href="https://www.nature.com/articles/s42003-024-06421-0">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>SLEEP DISRUPTION AND AGING IN LIVER WT:</b> Property of circadian-affected tissue-specific genes in cerebellum, forebrain, liver, kidney, and their link to aging and longevity.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Disruption And Aging In Liver</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE240693">GSE240693</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>3</td> -->
        <!-- <td>We extracted total RNA from tissue samples using Trizol (Invitrogen). After separating the RNA, DNA, and protein layers with chloroform, we precipitated the RNA using a standard isopropanol/ethanol procedure. The resulting RNA pellet was washed, and then resuspended in RNase-free water.</td> -->
        <td><a href="https://www.nature.com/articles/s42003-024-06421-0">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>TEMPERATURE CHANGE AND CIRC RHYTHM 18C:</b> mRNA expression level of the fat body from Drosophila melanogaster following a step change from 25C to 18C</td>
        <td><i>Drosophila melanogaster</i></td>
        <!-- <td>Change And Circ Rhythm</td> -->
        <td>Fat Body</td>
        <td>18c</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE241002">GSE241002</a></td>
        <td>Temperature</td>
        <!-- <td></td> -->
        <td>114, 88, 110, 112, 82, 80, 86, 84, 24, 26, 20, 22, 28, 0, 2, 4, 6, 8, 78, 98, 108, 102, 90, 100, 92, 106, 94, 104, 96, 10, 12, 14, 16, 32, 30, 36, 34</td>
        <!-- <td>3</td> -->
        <!-- <td>The experimental group was shifted to 18 degrees C at seven days old while the control group was kept at 25 degrees C.</td> -->
        <td><a href="https://www.biorxiv.org/content/10.1101/2023.10.27.563979v1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>TEMPERATURE CHANGE AND CIRC RHYTHM 18C,HIGH-CALORIE:</b> mRNA expression level of the fat body from Drosophila melanogaster following a step change from 25C to 18C</td>
        <td><i>Drosophila melanogaster</i></td>
        <!-- <td>Change And Circ Rhythm</td> -->
        <td>Fat Body</td>
        <td>18c,high-Calorie</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE241003">GSE241003</a></td>
        <td>Diet, Temperature</td>
        <!-- <td></td> -->
        <td>24, 10, 12, 20, 14, 22, 16, 18, 2, 4, 6, 8</td>
        <!-- <td>3</td> -->
        <!-- <td>All the flies used for experiments were first raised on a standard yeast-cornmeal-molasses based diet under a light-dark (LD) 12:12h cycle. Young mated female (~3 days old) flies (w1118: Bloomington stock number 5905, Clk-Jrk: backcrossed 7 times to w1118) of ~24 hours cohorts were entrained at either 25C or 18C for 5 days in 12L:12D cycles on a standard or diluted diet (Control: 15% [w/v] Sucrose and Yeast, 15SY; Dietary Restriction: 5% [w/v] Sucrose and Yeast, 5SY).</td> -->
        <td><a href="https://www.biorxiv.org/content/10.1101/2023.10.27.563979v1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>TEMPERATURE CHANGE AND CIRC RHYTHM 18C,LOW-CALORIE:</b> mRNA expression level of the fat body from Drosophila melanogaster following a step change from 25C to 18C</td>
        <td><i>Drosophila melanogaster</i></td>
        <!-- <td>Change And Circ Rhythm</td> -->
        <td>Fat Body</td>
        <td>18c,low-Calorie</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE241003">GSE241003</a></td>
        <td>Diet, Temperature</td>
        <!-- <td></td> -->
        <td>24, 10, 12, 20, 14, 22, 16, 18, 2, 4, 6, 8</td>
        <!-- <td>3</td> -->
        <!-- <td>All the flies used for experiments were first raised on a standard yeast-cornmeal-molasses based diet under a light-dark (LD) 12:12h cycle. Young mated female (~3 days old) flies (w1118: Bloomington stock number 5905, Clk-Jrk: backcrossed 7 times to w1118) of ~24 hours cohorts were entrained at either 25C or 18C for 5 days in 12L:12D cycles on a standard or diluted diet (Control: 15% [w/v] Sucrose and Yeast, 15SY; Dietary Restriction: 5% [w/v] Sucrose and Yeast, 5SY).</td> -->
        <td><a href="https://www.biorxiv.org/content/10.1101/2023.10.27.563979v1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>TEMPERATURE CHANGE AND CIRC RHYTHM 25C:</b> mRNA expression level of the fat body from Drosophila melanogaster following a step change from 25C to 18C</td>
        <td><i>Drosophila melanogaster</i></td>
        <!-- <td>Change And Circ Rhythm</td> -->
        <td>Fat Body</td>
        <td>25c</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE241002">GSE241002</a></td>
        <td>Temperature</td>
        <!-- <td></td> -->
        <td>114, 88, 110, 112, 82, 80, 86, 84, 24, 26, 20, 22, 28, 0, 2, 4, 6, 8, 78, 98, 108, 102, 90, 100, 92, 106, 94, 104, 96, 10, 12, 14, 16, 32, 30, 36, 34</td>
        <!-- <td>3</td> -->
        <!-- <td>The experimental group was shifted to 18 degrees C at seven days old while the control group was kept at 25 degrees C.</td> -->
        <td><a href="https://www.biorxiv.org/content/10.1101/2023.10.27.563979v1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>TEMPERATURE CHANGE AND CIRC RHYTHM 25C,HIGH-CALORIE:</b> mRNA expression level of the fat body from Drosophila melanogaster following a step change from 25C to 18C</td>
        <td><i>Drosophila melanogaster</i></td>
        <!-- <td>Change And Circ Rhythm</td> -->
        <td>Fat Body</td>
        <td>25c,high-Calorie</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE241003">GSE241003</a></td>
        <td>Diet, Temperature</td>
        <!-- <td></td> -->
        <td>24, 10, 12, 20, 14, 22, 16, 18, 2, 4, 6, 8</td>
        <!-- <td>3</td> -->
        <!-- <td>All the flies used for experiments were first raised on a standard yeast-cornmeal-molasses based diet under a light-dark (LD) 12:12h cycle. Young mated female (~3 days old) flies (w1118: Bloomington stock number 5905, Clk-Jrk: backcrossed 7 times to w1118) of ~24 hours cohorts were entrained at either 25C or 18C for 5 days in 12L:12D cycles on a standard or diluted diet (Control: 15% [w/v] Sucrose and Yeast, 15SY; Dietary Restriction: 5% [w/v] Sucrose and Yeast, 5SY).</td> -->
        <td><a href="https://www.biorxiv.org/content/10.1101/2023.10.27.563979v1">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>TOVIN ZEBRAFISH 2012 KO:</b> Adult (0.5-1.5 years old) transgenic zebrafish, Tg(aanat2:EGFP)Y8, which express enhanced green fluorescent protein (EGFP) in the pineal gland under the control of the aanat2 regulatory regions, were used. Fish were raised under 12-hr light:12-hr dark (LD) cycles, in a temperature controlled room, and transferred to constant darkness (DD) for tissue collection.</td>
        <td><i>Danio rerio</i></td>
        <!-- <td></td> -->
        <td>Glands</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>10, 14, 22, 18, 2, 6</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="https://pubmed.ncbi.nlm.nih.gov/23284293/">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>VISION AND CIRCADIAN BIOLOGY CHOROIDAL MYOPIC:</b> Examining the role of circadian biology in myopia by analyzing gene expression in the retina and choroid of chicks, comparing the myopic (occluded) eye to the contralateral control (open) eye</td>
        <td><i>Gallus gallus</i></td>
        <!-- <td>And Circadian Biology Choroidal</td> -->
        <td>Eye</td>
        <td>Myopic</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE261232">GSE261232</a></td>
        <td>Disease</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>7</td> -->
        <!-- <td>After applying RNA-Seq to each tissue extract, we used various bioinformatics and gene expression profiling analyses to assess the retinas and choroids at each time point. We extended these findings with Gene Set Enrichment Analysis and with comparison to candidate genes for human myopia.</td> -->
        <td><a href="https://doi.org/10.1371/journal.pone.0307091">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>VISION AND CIRCADIAN BIOLOGY CHOROIDAL WT:</b> Examining the role of circadian biology in myopia by analyzing gene expression in the retina and choroid of chicks, comparing the myopic (occluded) eye to the contralateral control (open) eye</td>
        <td><i>Gallus gallus</i></td>
        <!-- <td>And Circadian Biology Choroidal</td> -->
        <td>Eye</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE261232">GSE261232</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>7</td> -->
        <!-- <td>After applying RNA-Seq to each tissue extract, we used various bioinformatics and gene expression profiling analyses to assess the retinas and choroids at each time point. We extended these findings with Gene Set Enrichment Analysis and with comparison to candidate genes for human myopia.</td> -->
        <td><a href="https://doi.org/10.1371/journal.pone.0307091">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>VISION AND CIRCADIAN BIOLOGY RETINAL MYOPIC:</b> Examining the role of circadian biology in myopia by analyzing gene expression in the retina and choroid of chicks, comparing the myopic (occluded) eye to the contralateral control (open) eye</td>
        <td><i>Gallus gallus</i></td>
        <!-- <td>And Circadian Biology Retinal</td> -->
        <td>Eye</td>
        <td>Myopic</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE261232">GSE261232</a></td>
        <td>Disease</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>7</td> -->
        <!-- <td>After applying RNA-Seq to each tissue extract, we used various bioinformatics and gene expression profiling analyses to assess the retinas and choroids at each time point. We extended these findings with Gene Set Enrichment Analysis and with comparison to candidate genes for human myopia.</td> -->
        <td><a href="https://doi.org/10.1371/journal.pone.0307091">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>VISION AND CIRCADIAN BIOLOGY RETINAL WT:</b> Examining the role of circadian biology in myopia by analyzing gene expression in the retina and choroid of chicks, comparing the myopic (occluded) eye to the contralateral control (open) eye</td>
        <td><i>Gallus gallus</i></td>
        <!-- <td>And Circadian Biology Retinal</td> -->
        <td>Eye</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE261232">GSE261232</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>7</td> -->
        <!-- <td>After applying RNA-Seq to each tissue extract, we used various bioinformatics and gene expression profiling analyses to assess the retinas and choroids at each time point. We extended these findings with Gene Set Enrichment Analysis and with comparison to candidate genes for human myopia.</td> -->
        <td><a href="https://doi.org/10.1371/journal.pone.0307091">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
    </tbody>
   </table>
  </div>
  <div class="tab-pane" id="metabs" role="tabpanel">
    <table id="metabolome_table" class="table table-striped">
    <thead class="thead-dark">
      <tr>
        <th scope="col">Dataset Description</th>
        <th scope="col">Species</th>
        <!-- <th scope="col">Tissue</th> -->
        <th scope="col">Tissue Class</th>
        <th scope="col">Condition</th>
        <th scope="col">GEO Accession</th>
        <th scope="col">Experiment Type</th>
        <!-- <th scope="col">Animal</th> -->
        <th scope="col">Timepoints</th>
        <!-- <th scope="col">Replicates</th> -->
        <!-- <th scope="col">Details</th> -->
        <th scope="col">Link</th>
      </tr>
    </thead>
    <tbody>
     
      <tr class="dataset_row">
        <td><b>HUMAN U2 OS KRISHNAIAH 2017 OSTEOSARCOMA:</b> High temporal resolution metabolite profiling to explore clock regulation of mouse liver and cell-autonomous metabolism.</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>U2 Os</td> -->
        <td>Cells</td>
        <td>Osteosarcoma</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>42, 24, 26, 20, 22, 46, 44, 48, 28, 40, 0, 2, 4, 6, 8, 38, 10, 12, 14, 16, 18, 30, 36, 34, 32</td>
        <!-- <td>3</td> -->
        <!-- <td>Sampled U2 OS cells every 2 hr for 48 hr.</td> -->
        <td><a href="http://www.cell.com/cell-metabolism/fulltext/S1550-4131(17)30171-7">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE BRAIN LIVER MUSCLE RE PETRUS BRAIN RE:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Brain Liver Muscle Re Petrus Brain</td> -->
        <td>Brain</td>
        <td>Re</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE BRAIN LIVER MUSCLE RE PETRUS KO:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Brain Liver Muscle Re Petrus</td> -->
        <td>Liver</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE BRAIN LIVER MUSCLE RE PETRUS LIVER RE:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Brain Liver Muscle Re Petrus Liver</td> -->
        <td>Liver</td>
        <td>Re</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE BRAIN LIVER MUSCLE RE PETRUS MUSCLE RE:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Brain Liver Muscle Re Petrus Muscle</td> -->
        <td>Muscle</td>
        <td>Re</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE BRAIN LIVER MUSCLE RE PETRUS WT:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Brain Liver Muscle Re Petrus</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE BROWN ADIPOSE DYAR 2018 HIGH-FAT:</b> Comprehensive in vivo temporal atlas of circadian metabolism across a variety of tissues and serum.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Brown Adipose</td> -->
        <td>Adipose</td>
        <td>High-Fat</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 0, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE BROWN ADIPOSE DYAR 2018 NORMAL-CHOW:</b> Comprehensive in vivo temporal atlas of circadian metabolism across a variety of tissues and serum.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Brown Adipose</td> -->
        <td>Adipose</td>
        <td>Normal-Chow</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 0, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE EXERCISE METABOLOME EVENING-EXERCISE:</b> Metabolomic data for mouse exercise</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Exercise Metabolome</td> -->
        <td>Liver</td>
        <td>Evening-Exercise</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 4, 8</td>
        <!-- <td>7</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE EXERCISE METABOLOME EVENING-SEDENTARY:</b> Metabolomic data for mouse exercise</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Exercise Metabolome</td> -->
        <td>Liver</td>
        <td>Evening-Sedentary</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 4, 8</td>
        <!-- <td>7</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE EXERCISE METABOLOME MORNING-EXERCISE:</b> Metabolomic data for mouse exercise</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Exercise Metabolome</td> -->
        <td>Liver</td>
        <td>Morning-Exercise</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 4, 8</td>
        <!-- <td>7</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE EXERCISE METABOLOME MORNING-SEDENTARY:</b> Metabolomic data for mouse exercise</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Exercise Metabolome</td> -->
        <td>Liver</td>
        <td>Morning-Sedentary</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 4, 8</td>
        <!-- <td>7</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEPATIC EVENING-EXERCISE:</b> Metabolomic data for mouse exercise</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Hepatic</td> -->
        <td>Liver</td>
        <td>Evening-Exercise</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>7</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEPATIC EVENING-SEDENTARY:</b> Metabolomic data for mouse exercise</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Hepatic</td> -->
        <td>Liver</td>
        <td>Evening-Sedentary</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>7</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEPATIC MORNING-EXERCISE:</b> Metabolomic data for mouse exercise</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Hepatic</td> -->
        <td>Liver</td>
        <td>Morning-Exercise</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>7</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEPATIC MORNING-SEDENTARY:</b> Metabolomic data for mouse exercise</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Hepatic</td> -->
        <td>Liver</td>
        <td>Morning-Sedentary</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Exercise</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>7</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE HEPATOCYTE KRISHNAIAH 2017 WT:</b> High temporal resolution metabolite profiling to explore clock regulation of mouse liver and cell-autonomous metabolism.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Hepatocyte</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 0, 4, 8</td>
        <!-- <td>13</td> -->
        <!-- <td>Due to known dampening of molecular clock in sampling was done every 4 hours over 24 hours</td> -->
        <td><a href="http://www.cell.com/cell-metabolism/fulltext/S1550-4131(17)30171-7">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER ECKEL-MAHAN 2011 CLOCK-KO:</b> Liver metabolome of a Clock-KO experiment</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Clock-KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>9, 3, 15, 21</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="http://www.pnas.org/content/109/14/5541.short">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER ECKEL-MAHAN 2011 WT:</b> Liver metabolome of a Clock-KO experiment</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>9, 3, 15, 21</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="http://www.pnas.org/content/109/14/5541.short">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER KRISHNAIAH 2017 BMAL1-KD:</b> High temporal resolution metabolite profiling to explore clock regulation of mouse liver and cell-autonomous metabolism.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Bmal1-Kd</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>42, 43, 24, 25, 26, 27, 20, 21, 22, 23, 46, 47, 44, 45, 28, 29, 40, 41, 1, 0, 3, 2, 5, 4, 7, 6, 9, 8, 39, 38, 11, 10, 13, 12, 15, 14, 17, 16, 19, 18, 31, 30, 37, 36, 35, 34, 33, 32</td>
        <!-- <td>5</td> -->
        <!-- <td>Sampled mouse in replicate every 1 hr for 48 hr in DD</td> -->
        <td><a href="http://www.cell.com/cell-metabolism/fulltext/S1550-4131(17)30171-7">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER KRISHNAIAH 2017 CRY1-KD:</b> High temporal resolution metabolite profiling to explore clock regulation of mouse liver and cell-autonomous metabolism.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Cry1-Kd</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>42, 43, 24, 25, 26, 27, 20, 21, 22, 23, 46, 47, 44, 45, 28, 29, 40, 41, 1, 0, 3, 2, 5, 4, 7, 6, 9, 8, 39, 38, 11, 10, 13, 12, 15, 14, 17, 16, 19, 18, 31, 30, 37, 36, 35, 34, 33, 32</td>
        <!-- <td>5</td> -->
        <!-- <td>Sampled mouse in replicate every 1 hr for 48 hr in DD</td> -->
        <td><a href="http://www.cell.com/cell-metabolism/fulltext/S1550-4131(17)30171-7">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER KRISHNAIAH 2017 CRY2-KD:</b> High temporal resolution metabolite profiling to explore clock regulation of mouse liver and cell-autonomous metabolism.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Cry2-Kd</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>42, 43, 24, 25, 26, 27, 20, 21, 22, 23, 46, 47, 44, 45, 28, 29, 40, 41, 1, 0, 3, 2, 5, 4, 7, 6, 9, 8, 39, 38, 11, 10, 13, 12, 15, 14, 17, 16, 19, 18, 31, 30, 37, 36, 35, 34, 33, 32</td>
        <!-- <td>5</td> -->
        <!-- <td>Sampled mouse in replicate every 1 hr for 48 hr in DD</td> -->
        <td><a href="http://www.cell.com/cell-metabolism/fulltext/S1550-4131(17)30171-7">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER KRISHNAIAH 2017 WT:</b> High temporal resolution metabolite profiling to explore clock regulation of mouse liver and cell-autonomous metabolism.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>42, 43, 24, 25, 26, 27, 20, 21, 22, 23, 46, 47, 44, 45, 28, 29, 40, 41, 1, 0, 3, 2, 5, 4, 7, 6, 9, 8, 39, 38, 11, 10, 13, 12, 15, 14, 17, 16, 19, 18, 31, 30, 37, 36, 35, 34, 33, 32</td>
        <!-- <td>5</td> -->
        <!-- <td>Sampled mouse in replicate every 1 hr for 48 hr in DD</td> -->
        <td><a href="http://www.cell.com/cell-metabolism/fulltext/S1550-4131(17)30171-7">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER METABOLOME ECKEL-MAHAN 2013 HIGH-FAT:</b> Investigating widespread remodeling of the liver clock generated by high-fat diet.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Metabolome</td> -->
        <td>Liver</td>
        <td>High-Fat</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE52333">GSE52333</a></td>
        <td>Diet</td>
        <!-- <td>8-week old, male C57BL/6J</td> -->
        <td>24, 12, 20, 16, 0, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td>Animals consuming normal chow or high fat diet (60 percent kcal from fat) for ten weeks were analyzed for circadian gene expression. Livers were harvested from animals every four hours throughout the circadian cycle.</td> -->
        <td><a href="http://www.cell.com/abstract/S0092-8674(13)01485-2">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER METABOLOME ECKEL-MAHAN 2013 NORMAL-CHOW:</b> Investigating widespread remodeling of the liver clock generated by high-fat diet.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Metabolome</td> -->
        <td>Liver</td>
        <td>Normal-Chow</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE52333">GSE52333</a></td>
        <td>Control</td>
        <!-- <td>8-week old, male C57BL/6J</td> -->
        <td>24, 12, 20, 16, 0, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td>Animals consuming normal chow or high fat diet (60 percent kcal from fat) for ten weeks were analyzed for circadian gene expression. Livers were harvested from animals every four hours throughout the circadian cycle.</td> -->
        <td><a href="http://www.cell.com/abstract/S0092-8674(13)01485-2">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER METABOLOME MASRI 2014 SIRT1-KO:</b> Liver metabolome of a Sirt1 KO experiment.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Metabolome</td> -->
        <td>Liver</td>
        <td>Sirt1-KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE57830">GSE57830</a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td>Mice with liver-specific ablation of either Sirt1 or Sirt6 genes and their corresponding wild-type (WT) littermates. Livers were harvested every 4 hr over the circadian cycle</td> -->
        <td><a href="http://www.cell.com/cell/fulltext/S0092-8674(14)00884-8">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER METABOLOME MASRI 2014 SIRT1-WT:</b> Liver metabolome of a Sirt1 KO experiment.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Metabolome</td> -->
        <td>Liver</td>
        <td>Sirt1-Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE57830">GSE57830</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td>Mice with liver-specific ablation of either Sirt1 or Sirt6 genes and their corresponding wild-type (WT) littermates. Livers were harvested every 4 hr over the circadian cycle</td> -->
        <td><a href="http://www.cell.com/cell/fulltext/S0092-8674(14)00884-8">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER METABOLOME MASRI 2014 SIRT6-KO:</b> Metabolome of a liver in a Sirt6 KO experiment</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Metabolome</td> -->
        <td>Liver</td>
        <td>Sirt6-KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE57830">GSE57830</a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td>Mice with liver-specific ablation of either Sirt1 or Sirt6 genes and their corresponding wild-type (WT) littermates. Livers were harvested every 4 hr over the circadian cycle</td> -->
        <td><a href="http://www.cell.com/cell/fulltext/S0092-8674(14)00884-8">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER METABOLOME MASRI 2014 SIRT6-WT:</b> Metabolome of a liver in a Sirt6 KO experiment</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Metabolome</td> -->
        <td>Liver</td>
        <td>Sirt6-Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE57830">GSE57830</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td>Mice with liver-specific ablation of either Sirt1 or Sirt6 genes and their corresponding wild-type (WT) littermates. Livers were harvested every 4 hr over the circadian cycle</td> -->
        <td><a href="http://www.cell.com/cell/fulltext/S0092-8674(14)00884-8">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER METABOLOME MASRI 2016 LUNG-WT:</b> Liver metabolome of a mouse with a distal tumor bearing lung as an experimental condition</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Metabolome</td> -->
        <td>Liver</td>
        <td>Lung-Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73222">GSE73222</a></td>
        <td>Control</td>
        <!-- <td>Kras, C57BL/6J</td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td>WT and TB mice were sacrificed every 4 hr over the circadian cycle (zeitgeber time [ZT] 0, 4, 8, 12, 16 and 20) and livers were subjected to transcriptomics and metabolomics analyses</td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER METABOLOME MASRI 2016 TUMOR-BEARING-LUNG:</b> Liver metabolome of a mouse with a distal tumor bearing lung as an experimental condition</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Metabolome</td> -->
        <td>Liver</td>
        <td>Tumor-Bearing-Lung</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73222">GSE73222</a></td>
        <td>Cancer</td>
        <!-- <td>Kras, C57BL/6J</td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td>WT and TB mice were sacrificed every 4 hr over the circadian cycle (zeitgeber time [ZT] 0, 4, 8, 12, 16 and 20) and livers were subjected to transcriptomics and metabolomics analyses</td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER SASSONE METABOLOME KO:</b> Liver RNASeq with KO of all clock genes, and RE of just Liver Clock</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Sassone Metabolome</td> -->
        <td>Liver</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER SASSONE METABOLOME RE:</b> Liver RNASeq with KO of all clock genes, and RE of just Liver Clock</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Sassone Metabolome</td> -->
        <td>Liver</td>
        <td>Re</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER SASSONE METABOLOME WT:</b> Liver RNASeq with KO of all clock genes, and RE of just Liver Clock</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Sassone Metabolome</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE MUSCLE DYAR 2018 HIGH-FAT:</b> Comprehensive in vivo temporal atlas of circadian metabolism across a variety of tissues and seru.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Muscle</td> -->
        <td>Muscle</td>
        <td>High-Fat</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE MUSCLE DYAR 2018 NORMAL-CHOW:</b> Comprehensive in vivo temporal atlas of circadian metabolism across a variety of tissues and seru.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Muscle</td> -->
        <td>Muscle</td>
        <td>Normal-Chow</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE MUSCLE NA 2018 KO:</b> Mouse with Knock-Out condition</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Muscle</td> -->
        <td>Muscle</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE MUSCLE NA 2018 WT:</b> Mouse with Knock-Out condition</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Muscle</td> -->
        <td>Muscle</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE NF RE PETRUS KO:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Nf Re Petrus</td> -->
        <td>Liver</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE NF RE PETRUS WT:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Nf Re Petrus</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>5</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE PREFRONTAL CORTEX NA 2018 HIGH-FAT:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Prefrontal Cortex</td> -->
        <td>Brain</td>
        <td>High-Fat</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE PREFRONTAL CORTEX NA 2018 NORMAL-CHOW:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Prefrontal Cortex</td> -->
        <td>Brain</td>
        <td>Normal-Chow</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE SERUM ABBONDANTE 2016 HIGH-FAT:</b> Comparative circadian metabolomic analysis on serum and liver in mice under high fat diet.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Serum</td> -->
        <td>Blood</td>
        <td>High-Fat</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Diet</td>
        <!-- <td>C57BL/6J</td> -->
        <td>24, 12, 20, 16, 0, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td>Age-matched, male C57BL/6J mice (JAX, 00064) were maintained on a 12-h light/12-h dark cycle. At 6 weeks of age, animals were placed on a normal chow diet (Prolab RMH 2500) or an HFD (60 percent of calories from fat; Research Diets, D12492) for 10 weeks. Liver and serum were harvested across the circadian cycle every 4 h. Serum was prepared from an abdominal/thoracic blood sample.</td> -->
        <td><a href="http://www.cell.com/cell-metabolism/fulltext/S1550-4131(17)30171-7">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE SERUM ABBONDANTE 2016 WT:</b> Comparative circadian metabolomic analysis on serum and liver in mice under high fat diet.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Serum</td> -->
        <td>Blood</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td>C57BL/6J</td> -->
        <td>24, 12, 20, 16, 0, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td>Age-matched, male C57BL/6J mice (JAX, 00064) were maintained on a 12-h light/12-h dark cycle. At 6 weeks of age, animals were placed on a normal chow diet (Prolab RMH 2500) or an HFD (60 percent of calories from fat; Research Diets, D12492) for 10 weeks. Liver and serum were harvested across the circadian cycle every 4 h. Serum was prepared from an abdominal/thoracic blood sample.</td> -->
        <td><a href="http://www.cell.com/cell-metabolism/fulltext/S1550-4131(17)30171-7">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE SPERM DYAR 2018 HIGH-FAT:</b> Comprehensive in vivo temporal atlas of circadian metabolism across a variety of tissues and serum.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Sperm</td> -->
        <td>Reproductive</td>
        <td>High-Fat</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE SPERM DYAR 2018 NORMAL-CHOW:</b> Comprehensive in vivo temporal atlas of circadian metabolism across a variety of tissues and serum.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Sperm</td> -->
        <td>Reproductive</td>
        <td>Normal-Chow</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE SUPRACHIASMATIC NUCLEUS NA 2018 HIGH-FAT:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Suprachiasmatic Nucleus</td> -->
        <td>Brain</td>
        <td>High-Fat</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Diet</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE SUPRACHIASMATIC NUCLEUS NA 2018 NORMAL-CHOW:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Suprachiasmatic Nucleus</td> -->
        <td>Brain</td>
        <td>Normal-Chow</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE WHITE ADIPOSE DYAR 2018 HIGH-FAT:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>White Adipose</td> -->
        <td>Adipose</td>
        <td>High-Fat</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 0, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE WHITE ADIPOSE DYAR 2018 NORMAL-CHOW:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>White Adipose</td> -->
        <td>Adipose</td>
        <td>Normal-Chow</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 12, 20, 16, 0, 4, 8</td>
        <!-- <td>6</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
    </tbody>
   </table>
  </div>
  <div class="tab-pane" id="acyls" role="tabpanel">
    <table id="acetylome_table"  class="table table-striped">
    <thead class="thead-dark">
      <tr>
        <th scope="col">Dataset Description</th>
        <th scope="col">Species</th>
        <!-- <th scope="col">Tissue</th> -->
        <th scope="col">Tissue Class</th>
        <th scope="col">Condition</th>
        <th scope="col">GEO Accession</th>
        <th scope="col">Experiment Type</th>
        <!-- <th scope="col">Animal</th> -->
        <th scope="col">Timepoints</th>
        <!-- <th scope="col">Replicates</th> -->
        <!-- <th scope="col">Details</th> -->
        <th scope="col">Link</th>
      </tr>
    </thead>
    <tbody>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER MASRI 2013 CLOCK-KO:</b> Liver acetylome in a high fat diet experiment</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Clock-KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td>Age-matched Adult Male</td> -->
        <td>9, 3, 15, 21</td>
        <!-- <td>4</td> -->
        <!-- <td>MS-based approach to analyze lysine acetylation sites in mouse liver in WT and Clock KO littermates. Mice were killed at ZT 3, 9, 15, and 21, and three independent livers from each ZT were used for acetylome analysis</td> -->
        <td><a href="http://www.pnas.org/content/110/9/3339">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER MASRI 2013 WT:</b> Liver acetylome in a high fat diet experiment</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td>Age-matched Adult Male</td> -->
        <td>9, 3, 15, 21</td>
        <!-- <td>4</td> -->
        <!-- <td>MS-based approach to analyze lysine acetylation sites in mouse liver in WT and Clock KO littermates. Mice were killed at ZT 3, 9, 15, and 21, and three independent livers from each ZT were used for acetylome analysis</td> -->
        <td><a href="http://www.pnas.org/content/110/9/3339">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER MAUVOISIN 2017 BMAL-KO:</b> Comprehensive and rhythmic acetylome map of the mouse liver for total and nuclear proteins.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Bmal-KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td>C57BL/6</td> -->
        <td>0, 12, 18, 6</td>
        <!-- <td>2</td> -->
        <!-- <td>Optimized sample preparation including antibody-based enrichment protocols combined with sensitive nano-reverse-phase liquid chromatography (RP-LC) tandem MS (MS/MS) applied to total (TE) and nuclear (NE) mouse liver protein extracts harvested during two diurnal cycles</td> -->
        <td><a href="https://www.ncbi.nlm.nih.gov/pubmed?term=28813682">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER MAUVOISIN 2017 BMAL-KO-NUCLEAR:</b> Comprehensive and rhythmic acetylome map of the mouse liver for total and nuclear proteins.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Bmal-KO-Nuclear</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td>C57BL/6</td> -->
        <td>0, 12, 18, 6</td>
        <!-- <td>2</td> -->
        <!-- <td>Optimized sample preparation including antibody-based enrichment protocols combined with sensitive nano-reverse-phase liquid chromatography (RP-LC) tandem MS (MS/MS) applied to total (TE) and nuclear (NE) mouse liver protein extracts harvested during two diurnal cycles</td> -->
        <td><a href="https://www.ncbi.nlm.nih.gov/pubmed?term=28813682">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER MAUVOISIN 2017 CRY-KO:</b> Comprehensive and rhythmic acetylome map of the mouse liver for total and nuclear proteins.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Cry-KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td>C57BL/6</td> -->
        <td>0, 12, 18, 6</td>
        <!-- <td>2</td> -->
        <!-- <td>Optimized sample preparation including antibody-based enrichment protocols combined with sensitive nano-reverse-phase liquid chromatography (RP-LC) tandem MS (MS/MS) applied to total (TE) and nuclear (NE) mouse liver protein extracts harvested during two diurnal cycles</td> -->
        <td><a href="https://www.ncbi.nlm.nih.gov/pubmed?term=28813682">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER MAUVOISIN 2017 CRY-KO-NUCLEAR:</b> Comprehensive and rhythmic acetylome map of the mouse liver for total and nuclear proteins.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Cry-KO-Nuclear</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Knock-Out</td>
        <!-- <td>C57BL/6</td> -->
        <td>0, 12, 18, 6</td>
        <!-- <td>2</td> -->
        <!-- <td>Optimized sample preparation including antibody-based enrichment protocols combined with sensitive nano-reverse-phase liquid chromatography (RP-LC) tandem MS (MS/MS) applied to total (TE) and nuclear (NE) mouse liver protein extracts harvested during two diurnal cycles</td> -->
        <td><a href="https://www.ncbi.nlm.nih.gov/pubmed?term=28813682">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER MAUVOISIN 2017 WT:</b> Comprehensive and rhythmic acetylome map of the mouse liver for total and nuclear proteins.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td>C57BL/6</td> -->
        <td>0, 12, 18, 6</td>
        <!-- <td>2</td> -->
        <!-- <td>Optimized sample preparation including antibody-based enrichment protocols combined with sensitive nano-reverse-phase liquid chromatography (RP-LC) tandem MS (MS/MS) applied to total (TE) and nuclear (NE) mouse liver protein extracts harvested during two diurnal cycles</td> -->
        <td><a href="https://www.ncbi.nlm.nih.gov/pubmed?term=28813682">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER MAUVOISIN 2017 WT-NUCLEAR:</b> Comprehensive and rhythmic acetylome map of the mouse liver for total and nuclear proteins.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Wild-Type-Nuclear</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td>C57BL/6</td> -->
        <td>0, 12, 18, 6</td>
        <!-- <td>2</td> -->
        <!-- <td>Optimized sample preparation including antibody-based enrichment protocols combined with sensitive nano-reverse-phase liquid chromatography (RP-LC) tandem MS (MS/MS) applied to total (TE) and nuclear (NE) mouse liver protein extracts harvested during two diurnal cycles</td> -->
        <td><a href="https://www.ncbi.nlm.nih.gov/pubmed?term=28813682">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
    </tbody>
   </table>
  </div>
  <div class="tab-pane" id="prots" role="tabpanel">
    <table id="proteome_table" class="table table-striped">
    <thead class="thead-dark">
      <tr>
        <th scope="col">Dataset Description</th>
        <th scope="col">Species</th>
        <!-- <th scope="col">Tissue</th> -->
        <th scope="col">Tissue Class</th>
        <th scope="col">Condition</th>
        <th scope="col">GEO Accession</th>
        <th scope="col">Experiment Type</th>
        <!-- <th scope="col">Animal</th> -->
        <th scope="col">Timepoints</th>
        <!-- <th scope="col">Replicates</th> -->
        <!-- <th scope="col">Details</th> -->
        <th scope="col">Link</th>
      </tr>
    </thead>
    <tbody>
     
      <tr class="dataset_row">
        <td><b>HUMAN PLASMA DEPNER 2018 MISALIGNED:</b> 24 hour plasma proteome profiles from human subjects under normal vs misaligned (simulated night shift) sleep/feeding schedules. Demonstrates that circadian misalignment (nighttime eating and daytime sleeping) rapidly disrupts normal 24h protein rhythms altering levels and timing of ~127 plasma proteins linked to metabolic and immune pathways.</td>
        <td><i>Homo sapiens</i></td>
        <!-- <td>Plasma</td> -->
        <td>Blood</td>
        <td>Misaligned</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>11, 13, 15, 21, 17, 23, 19, 22, 1, 3, 5, 7, 9</td>
        <!-- <td>13</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.6084/m9.figshare.5752650">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE CARTILAGE DUDEK 2021 WT:</b> Mouse hip articular cartilage collected every 4h over 48h for proteomic time-series analysis, identifying ~145 proteins with circadian oscillations in abundance (revealing robust daily rhythmicity in cartilage physiology).</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Cartilage</td> -->
        <td>Cartilage</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=PXD019431">PXD019431</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>10, 26, 38, 14, 22, 46, 18, 30, 42, 34, 2, 6</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1016/j.joca.2021.02.008">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE CEREBRAL CORTEX GERSTNER 2016 WT:</b> Comprehensive analysis of the effects of sleep deprivation and subsequent recovery sleep on gene expression in the mouse cortex.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Cerebral Cortex</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE78215">GSE78215</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>11, 0, 8, 7, 6</td>
        <!-- <td>8</td> -->
        <!-- <td>Subjected mice to 5 to 6 h of sleep deprivation and different amounts of recovery sleep were sacrificed at matching circadian times.</td> -->
        <td><a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-016-3065-8">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER MAUVOISIN 2014 KO:</b> Circadian liver proteome profiling (in vivo SILAC) in wild-type vs Cry1/Cry2 double-knockout mice under rhythmic feeding. Identifies hundreds of oscillating hepatic proteins and shows that many protein rhythms persist even without a functional clock (driven by feeding cues).</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=PXD001211">PXD001211</a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td></td>
        <!-- <td>0</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1073/pnas.1314066111">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER MAUVOISIN 2014 WT:</b> Circadian liver proteome profiling (in vivo SILAC) in wild-type vs Cry1/Cry2 double-knockout mice under rhythmic feeding. Identifies hundreds of oscillating hepatic proteins and shows that many protein rhythms persist even without a functional clock (driven by feeding cues).</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=PXD001211">PXD001211</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td></td>
        <!-- <td>0</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1073/pnas.1314066111">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER NA 2018 WT:</b> Proteome of mouse liver.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>12, 20, 16, 0, 4, 8</td>
        <!-- <td>3</td> -->
        <!-- <td></td> -->
        <td><a href="">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER NUCLEAR PHOSPHOPROTEOME WANG 2017 BMALKO:</b> Time-series nuclear phosphoproteomics of mouse liver (phosphopeptide enrichment every 3h for 45h) in wild-type vs Cry1/2 and Bmal1 knockout mice, providing insight into how circadian clock disruption affects rhythmic protein phosphorylation and function in the liver nucleus.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Nuclear Phosphoproteome</td> -->
        <td>Liver</td>
        <td>Bmalko</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=PXD004191">PXD004191</a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>24, 39, 12, 15, 21, 33, 18, 30, 42, 36, 0, 3, 27, 6, 9, 45</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2016.10.003">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER NUCLEAR PHOSPHOPROTEOME WANG 2017 BMALWT:</b> Time-series nuclear phosphoproteomics of mouse liver (phosphopeptide enrichment every 3h for 45h) in wild-type vs Cry1/2 and Bmal1 knockout mice, providing insight into how circadian clock disruption affects rhythmic protein phosphorylation and function in the liver nucleus.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Nuclear Phosphoproteome</td> -->
        <td>Liver</td>
        <td>Bmalwild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=PXD004191">PXD004191</a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>24, 39, 12, 15, 21, 33, 18, 30, 42, 36, 0, 3, 27, 6, 9, 45</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2016.10.003">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER NUCLEAR PHOSPHOPROTEOME WANG 2017 CRYKO:</b> Time-series nuclear phosphoproteomics of mouse liver (phosphopeptide enrichment every 3h for 45h) in wild-type vs Cry1/2 and Bmal1 knockout mice, providing insight into how circadian clock disruption affects rhythmic protein phosphorylation and function in the liver nucleus.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Nuclear Phosphoproteome</td> -->
        <td>Liver</td>
        <td>Cryko</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=PXD004191">PXD004191</a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>24, 39, 12, 15, 21, 33, 18, 30, 42, 36, 0, 3, 27, 6, 9, 45</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2016.10.003">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER NUCLEAR PHOSPHOPROTEOME WANG 2017 WT:</b> Time-series nuclear phosphoproteomics of mouse liver (phosphopeptide enrichment every 3h for 45h) in wild-type vs Cry1/2 and Bmal1 knockout mice, providing insight into how circadian clock disruption affects rhythmic protein phosphorylation and function in the liver nucleus.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Nuclear Phosphoproteome</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=PXD004191">PXD004191</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 39, 12, 15, 21, 33, 18, 30, 42, 36, 0, 3, 27, 6, 9, 45</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2016.10.003">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER NUCLEAR PROTEOME WANG 2017 BMAL++:</b> Time-series nuclear proteomics of mouse liver (total nuclear protein levels measured every 3h for 45h) in wild-type vs clock-deficient mice (Cry1/2 double knockout and Bmal1 knockout), uncovering diurnal regulation of nuclear protein accumulation by the circadian clock.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Nuclear Proteome</td> -->
        <td>Liver</td>
        <td>Bmal++</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=PXD003818">PXD003818</a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>24, 39, 12, 15, 21, 33, 18, 30, 42, 36, 0, 3, 27, 6, 9, 45</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2016.10.003">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER NUCLEAR PROTEOME WANG 2017 BMALKO:</b> Time-series nuclear proteomics of mouse liver (total nuclear protein levels measured every 3h for 45h) in wild-type vs clock-deficient mice (Cry1/2 double knockout and Bmal1 knockout), uncovering diurnal regulation of nuclear protein accumulation by the circadian clock.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Nuclear Proteome</td> -->
        <td>Liver</td>
        <td>Bmalko</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=PXD003818">PXD003818</a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>24, 39, 12, 15, 21, 33, 18, 30, 42, 36, 0, 3, 27, 6, 9, 45</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2016.10.003">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER NUCLEAR PROTEOME WANG 2017 CRYKO:</b> Time-series nuclear proteomics of mouse liver (total nuclear protein levels measured every 3h for 45h) in wild-type vs clock-deficient mice (Cry1/2 double knockout and Bmal1 knockout), uncovering diurnal regulation of nuclear protein accumulation by the circadian clock.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Nuclear Proteome</td> -->
        <td>Liver</td>
        <td>Cryko</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=PXD003818">PXD003818</a></td>
        <td>Knock-Out</td>
        <!-- <td></td> -->
        <td>24, 39, 12, 15, 21, 33, 18, 30, 42, 36, 0, 3, 27, 6, 9, 45</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2016.10.003">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER NUCLEAR PROTEOME WANG 2017 WT:</b> Time-series nuclear proteomics of mouse liver (total nuclear protein levels measured every 3h for 45h) in wild-type vs clock-deficient mice (Cry1/2 double knockout and Bmal1 knockout), uncovering diurnal regulation of nuclear protein accumulation by the circadian clock.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver Nuclear Proteome</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=PXD003818">PXD003818</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 39, 12, 15, 21, 33, 18, 30, 42, 36, 0, 3, 27, 6, 9, 45</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1016/j.cmet.2016.10.003">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER ROBLES 2014 WT:</b> Global circadian proteome of mouse liver: ~6 percent of liver proteins exhibit 24h oscillations, often with phases differing from their mRNA rhythms highlighting post transcriptional mechanisms in circadian regulation of liver metabolism.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=PXD000601">PXD000601</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 39, 12, 15, 21, 33, 18, 30, 42, 36, 0, 3, 27, 6, 9, 45</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1371/journal.pgen.1004047">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER ROBLES 2014 WT:</b> Global circadian proteome of mouse liver: ~6 percent of liver proteins exhibit 24h oscillations, often with phases differing from their mRNA rhythms highlighting post transcriptional mechanisms in circadian regulation of liver metabolism.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=PXD000601">PXD000601</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 39, 12, 15, 21, 33, 18, 30, 42, 36, 0, 3, 27, 6, 9, 45</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1371/journal.pgen.1004047">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE LIVER ROBLES 2014 WT:</b> Global circadian proteome of mouse liver: ~6 percent of liver proteins exhibit 24h oscillations, often with phases differing from their mRNA rhythms highlighting post transcriptional mechanisms in circadian regulation of liver metabolism.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Liver</td> -->
        <td>Liver</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=PXD000601">PXD000601</a></td>
        <td>Control</td>
        <!-- <td></td> -->
        <td>24, 39, 12, 15, 21, 33, 18, 30, 42, 36, 0, 3, 27, 6, 9, 45</td>
        <!-- <td>4</td> -->
        <!-- <td></td> -->
        <td><a href="https://doi.org/10.1371/journal.pgen.1004047">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE SUPRACHIASMATIC NUCLEUS MENDOZA 2017 MIR132-KO:</b> SCN neurons from miR-132/212-deficient mice.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Suprachiasmatic Nucleus</td> -->
        <td>Brain</td>
        <td>Mir132-KO</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td>C57BL/6J</td> -->
        <td>9, 3, 12, 21</td>
        <!-- <td>5</td> -->
        <!-- <td>Male and female mice, aged 4 to 8 weeks, were killed by cervical dislocation and decapitation under white light or dim red light as appropriate. Coronal sections were obtained with a vibratome or brain matrix.</td> -->
        <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28423315">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE SUPRACHIASMATIC NUCLEUS MENDOZA 2017 WT:</b> SCN neurons from miR-132/212-deficient mice.</td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Suprachiasmatic Nucleus</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td>C57BL/6J</td> -->
        <td>9, 3, 12, 21</td>
        <!-- <td>5</td> -->
        <!-- <td>Male and female mice, aged 4 to 8 weeks, were killed by cervical dislocation and decapitation under white light or dim red light as appropriate. Coronal sections were obtained with a vibratome or brain matrix.</td> -->
        <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28423315">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE VENTRAL HIPPOCAMPUS PROTEOME KONRAD 2017 TEMPORAL-LOBE-EPILEPTIC:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Ventral Hippocampus Proteome</td> -->
        <td>Brain</td>
        <td>Temporal-Lobe-Epileptic</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Disease</td>
        <!-- <td>Age Matched Adult</td> -->
        <td>11, 15, 23, 19, 1, 3, 7</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="https://www.biorxiv.org/content/early/2017/10/06/199372">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
      <tr class="dataset_row">
        <td><b>MOUSE VENTRAL HIPPOCAMPUS PROTEOME KONRAD 2017 WT:</b> </td>
        <td><i>Mus musculus</i></td>
        <!-- <td>Ventral Hippocampus Proteome</td> -->
        <td>Brain</td>
        <td>Wild-Type</td>
        <td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc="></a></td>
        <td>Control</td>
        <!-- <td>Age Matched Adult</td> -->
        <td>11, 15, 23, 19, 1, 3, 7</td>
        <!-- <td>2</td> -->
        <!-- <td></td> -->
        <td><a href="https://www.biorxiv.org/content/early/2017/10/06/199372">Link</a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp</td>
      </tr>
     
    </tbody>
   </table>
  </div>
</div>
</div>
</div>
</div>

<footer><center>&copy; 2018 Institute for Genomics and Bioinformatics - University of California, Irvine</center></footer>
<br>
<br>



    
<script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
<script src="/static/oscillation.js"></script>

    <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
    <script src="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js"></script>
<script type="text/javascript">
  google.charts.load("current", {packages:["corechart"]});
  google.charts.setOnLoadCallback(drawPie);
  function drawPie() {
    var data1 = google.visualization.arrayToDataTable([["Species", "Datasets"], ["Mus musculus", 489], ["Papio anubis", 64], ["Homo sapiens", 29], ["Drosophila melanogaster", 19], ["Aedes aegypti", 5], ["Rattus norvegicus", 5], ["Gallus gallus", 5], ["Anopheles gambiae", 4], ["Anopheles stephensi", 4], ["Neurospora crassa", 4], ["Plasmodium berghei", 4], ["Arabidopsis thaliana", 3], ["Danio rerio", 2], ["Ovis aries", 2], ["Caenorhabditis elegans", 2], ["Rhesus macaques", 1]]);
    var data2 = google.visualization.arrayToDataTable([["Tissue Class", "Datasets"], ["Liver", 180], ["Brain", 128], ["Muscle", 45], ["Cells", 35], ["Skin", 28], ["Kidney", 23], ["Heart", 23], ["Digestive", 22], ["Glands", 22], ["Lung", 20], ["Adipose", 20], ["Blood", 15], ["Reproductive", 12], ["Eye", 10], ["Spleen", 5], ["Fat Body", 5], ["Tumor", 4], ["Parasite", 4], ["Whole Body", 4], ["Hippocampus", 4], ["Salivary Gland", 2], ["Bone Marrow", 2], ["Salivary gland", 2], ["iWAT", 2], ["Bone", 2], ["Intestine", 2], ["Head", 2], ["Prefrontal cortex", 2], ["Seedling", 2], ["Arcuate nucleus", 2], ["Rumen epithelium", 2], ["Nervous", 2], ["Thoracic", 2], ["Larva", 1], ["Embryo", 1], ["Cartilage", 1], ["Organoid", 1], ["Bladder", 1], ["Plant", 1], ["Breast", 1]]);
    var data3 = google.visualization.arrayToDataTable([["Condition", "Datasets"], ["Control", 297], ["Diet", 100], ["Knock-Out", 83], ["Exercise", 50], ["Light-Dark", 38], ["Disease", 32], ["Oxygen-Level", 24], ["Age", 20], ["Sex", 14], ["Temperature", 14], ["Immune", 6], ["Sleep", 5], ["Drug Treatment", 5], ["Microbiome", 5], ["Cancer", 3], ["Inflammation", 2], ["Drug", 2], ["Knock-Down", 2], ["Injury", 1]]);
    var data4 = google.visualization.arrayToDataTable([["Omic Type", "Datasets"], ["Transcriptome", 563], ["Metabolome", 50], ["Proteome", 21], ["Acetylome", 8]]);
    var options1 = {
      pieSliceText: 'none',
      title: '16 Species',
      titleTextStyle: { fontSize: 16 },
      pieHole: 0.2,
      slices: {
        0: { color: "#457AB2" },
        // 1: { color: "#577CA4" },
        // 2: { color: "#6A7E97" },
        // 1: { color: "#7C818A" },
        // 2: { color: "#8F837D" },
        1: { color: "#A28670" },
        2: { color: "#B48862" },
        3: { color: "#C78A55" },
        4: { color: "#D98D58" },
        5: { color: "#EC8F3B" },
        6: { color: "#FF922E" },
        7: { color: "#E58329"},
        8: { color: "#CC7424"},
      },
    };
    var chart1 = new google.visualization.PieChart(document.getElementById('piechart1'));
    chart1.draw(data1, options1);

    var options2 = {
      pieSliceText: 'none',
      title: '40 Tissue Categories',
      titleTextStyle: { fontSize: 16 },
      pieHole: 0.1,
      sliceVisibilityThreshold: 0.015,
      slices: {
        0: { color: "#457AB2" },
        1: { color: "#577CA4" },
        2: { color: "#6A7E97" },
        3: { color: "#7C818A" },
        4: { color: "#8F837D" },
        5: { color: "#A28670" },
        6: { color: "#B48862" },
        7: { color: "#C78A55" },
        8: { color: "#D98D58" },
        9: { color: "#EC8F3B" },
        10: { color: "#FF922E" },
        11: { color: "#E58329"},
        12: { color: "#CC7424"},
      },
    };

    var chart2 = new google.visualization.PieChart(document.getElementById('piechart2'));
    chart2.draw(data2, options2);

    var options3 = {
      pieSliceText: 'none',
      title: '19 Condition Types',
      titleTextStyle: { fontSize: 16 },
      pieHole: 0.2,
      slices: {
        0: { color: "#457AB2" },
        // 1: { color: "#577CA4" },
        // 2: { color: "#6A7E97" },
        1: { color: "#7C818A" },
        2: { color: "#8F837D" },
        3: { color: "#A28670" },
        4: { color: "#B48862" },
        5: { color: "#C78A55" },
        6: { color: "#D98D58" },
        7: { color: "#EC8F3B" },
        8: { color: "#FF922E" },
      },
    };

    var chart3 = new google.visualization.PieChart(document.getElementById('piechart3'));
    chart3.draw(data3, options3);

    var options4 = {
      pieSliceText: 'none',
      title: '642 Datasets',
      titleTextStyle: { fontSize: 16 },
      pieHole: 0.2,
      slices: {
        0: { color: "#457AB2" },
        1: { color: "#8F837D" },
        2: { color: "#C78A55" },
        3: { color: "#FF922E" },
      },
    };

    var chart4 = new google.visualization.PieChart(document.getElementById('piechart4'));
    chart4.draw(data4, options4);

  }
</script>
<script>
function filterRows() {
  var input, filter, table, tr, td, i, description, species, tissue, condition, exp, details;
  input = document.getElementById("datasetFilter");
  filter = input.value.toUpperCase();
  table = document.getElementById("transcriptome_table");
  tr = table.getElementsByClassName("dataset_row");
  for (i = 0; i < tr.length; i++) {
    description = tr[i].getElementsByTagName("td")[0];
    species = tr[i].getElementsByTagName("td")[1];
    tissue = tr[i].getElementsByTagName("td")[3];
    condition = tr[i].getElementsByTagName("td")[4];
    exp = tr[i].getElementsByTagName("td")[6];
    details = tr[i].getElementsByTagName("td")[7];
    if (description && species && tissue && condition && exp && details) {
      var row_text = "";
      row_text += description.innerHTML.toUpperCase();
      row_text += species.innerHTML.toUpperCase();
      row_text += tissue.innerHTML.toUpperCase();
      row_text += condition.innerHTML.toUpperCase();
      row_text += exp.innerHTML.toUpperCase();
      row_text += details.innerHTML.toUpperCase();
      if (row_text.indexOf(filter) > -1) {
        tr[i].style.display = "";
      } else {
        tr[i].style.display = "none";
      }
    }
  }

  table = document.getElementById("metabolome_table");
  tr = table.getElementsByClassName("dataset_row");
  for (i = 0; i < tr.length; i++) {
    td = tr[i].getElementsByTagName("td")[0];
    if (td) {
      if (td.innerHTML.toUpperCase().indexOf(filter) > -1) {
        tr[i].style.display = "";
      } else {
        tr[i].style.display = "none";
      }
    }
  }
  table = document.getElementById("acetylome_table");
  tr = table.getElementsByClassName("dataset_row");
  for (i = 0; i < tr.length; i++) {
    td = tr[i].getElementsByTagName("td")[0];
    if (td) {
      if (td.innerHTML.toUpperCase().indexOf(filter) > -1) {
        tr[i].style.display = "";
      } else {
        tr[i].style.display = "none";
      }
    }
  }
  table = document.getElementById("proteome_table");
  tr = table.getElementsByClassName("dataset_row");
  for (i = 0; i < tr.length; i++) {
    td = tr[i].getElementsByTagName("td")[0];
    if (td) {
      if (td.innerHTML.toUpperCase().indexOf(filter) > -1) {
        tr[i].style.display = "";
      } else {
        tr[i].style.display = "none";
      }
    }
  }
}
</script>
<script>
$(document).on('click', "a.nav-link", function() {
  var current_id = $(this).attr('id');
  var omic_type = current_id.split("-")[1];
  console.log(current_id);
  var old_id = document.getElementsByClassName("nav-link active")[0].id;
  var previous_omic_type = old_id.split("-")[1];
  document.getElementById(old_id).innerHTML = "<font color='black'><b>" + previous_omic_type + "</b></font>";
  document.getElementById(current_id).innerHTML = "<font color='#457ab2'><b>" + omic_type + "</b></font>";
  $("#"+old_id).removeClass('active');
  $("#"+current_id).addClass('active');
});
</script>

  </body>
</html>
